// lucide-react/dist/cjs/lucide-react.js
/**
 * lucide-react v0.14.0 - ISC
 */

'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var react = require('react');
var PropTypes = require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var PropTypes__default = /*#__PURE__*/_interopDefaultLegacy(PropTypes);

function _extends() {
  _extends = Object.assign || function (target) {
    for (var i = 1; i < arguments.length; i++) {
      var source = arguments[i];

      for (var key in source) {
        if (Object.prototype.hasOwnProperty.call(source, key)) {
          target[key] = source[key];
        }
      }
    }

    return target;
  };

  return _extends.apply(this, arguments);
}

function _objectWithoutPropertiesLoose(source, excluded) {
  if (source == null) return {};
  var target = {};
  var sourceKeys = Object.keys(source);
  var key, i;

  for (i = 0; i < sourceKeys.length; i++) {
    key = sourceKeys[i];
    if (excluded.indexOf(key) >= 0) continue;
    target[key] = source[key];
  }

  return target;
}

var createReactComponent = (function (iconName, _ref) {
  var tag = _ref[0],
      attrs = _ref[1],
      children = _ref[2];
  var Component = react.forwardRef(function (_ref2, ref) {
    var _ref2$color = _ref2.color,
        color = _ref2$color === void 0 ? 'currentColor' : _ref2$color,
        _ref2$size = _ref2.size,
        size = _ref2$size === void 0 ? 24 : _ref2$size,
        _ref2$strokeWidth = _ref2.strokeWidth,
        strokeWidth = _ref2$strokeWidth === void 0 ? 2 : _ref2$strokeWidth,
        rest = _objectWithoutPropertiesLoose(_ref2, ["color", "size", "strokeWidth"]);

    return react.createElement(tag, _extends({
      ref: ref
    }, attrs, {
      width: size,
      height: size,
      stroke: color,
      strokeWidth: strokeWidth
    }, rest), children.map(function (_ref3) {
      var childTag = _ref3[0],
          childAttrs = _ref3[1];
      return react.createElement(childTag, childAttrs);
    }));
  });
  Component.propTypes = {
    color: PropTypes__default['default'].string,
    size: PropTypes__default['default'].oneOfType([PropTypes__default['default'].string, PropTypes__default['default'].number]),
    strokeWidth: PropTypes__default['default'].oneOfType([PropTypes__default['default'].string, PropTypes__default['default'].number])
  };
  Component.displayName = "" + iconName;
  return Component;
});

var defaultAttributes = {
  xmlns: 'http://www.w3.org/2000/svg',
  width: 24,
  height: 24,
  viewBox: '0 0 24 24',
  fill: 'none',
  stroke: 'currentColor',
  strokeWidth: 2,
  strokeLinecap: 'round',
  strokeLinejoin: 'round'
};

var Activity = createReactComponent("Activity", ["svg", defaultAttributes, [["polyline", {
  points: "22 12 18 12 15 21 9 3 6 12 2 12",
  key: "gu3rjn"
}]]]);

var Airplay = createReactComponent("Airplay", ["svg", defaultAttributes, [["path", {
  d: "M5 17H4a2 2 0 01-2-2V5a2 2 0 012-2h16a2 2 0 012 2v10a2 2 0 01-2 2h-1",
  key: "11m5yb"
}], ["polygon", {
  points: "12 15 17 21 7 21 12 15",
  key: "192zxh"
}]]]);

var AlarmCheck = createReactComponent("AlarmCheck", ["svg", defaultAttributes, [["path", {
  d: "M12 21a8 8 0 100-16 8 8 0 000 16z",
  key: "ubbk1n"
}], ["path", {
  d: "M5 3L2 6",
  key: "yo78bm"
}], ["path", {
  d: "M22 6l-3-3",
  key: "ry34ie"
}], ["path", {
  d: "M6 19l-2 2",
  key: "1dpmc8"
}], ["path", {
  d: "M18 19l2 2",
  key: "1v10ht"
}], ["path", {
  d: "M9 13l2 2 4-4",
  key: "brbdw6"
}]]]);

var AlarmClock = createReactComponent("AlarmClock", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "13",
  r: "8",
  key: "8t7gpc"
}], ["path", {
  d: "M12 9v4l2 2",
  key: "ap5om0"
}], ["path", {
  d: "M5 3L2 6",
  key: "yo78bm"
}], ["path", {
  d: "M22 6l-3-3",
  key: "ry34ie"
}], ["path", {
  d: "M6 19l-2 2",
  key: "1dpmc8"
}], ["path", {
  d: "M18 19l2 2",
  key: "1v10ht"
}]]]);

var AlarmMinus = createReactComponent("AlarmMinus", ["svg", defaultAttributes, [["path", {
  d: "M12 21a8 8 0 100-16 8 8 0 000 16z",
  key: "ubbk1n"
}], ["path", {
  d: "M5 3L2 6",
  key: "yo78bm"
}], ["path", {
  d: "M22 6l-3-3",
  key: "ry34ie"
}], ["path", {
  d: "M6 19l-2 2",
  key: "1dpmc8"
}], ["path", {
  d: "M18 19l2 2",
  key: "1v10ht"
}], ["path", {
  d: "M8 13h8",
  key: "1ti47a"
}]]]);

var AlarmPlus = createReactComponent("AlarmPlus", ["svg", defaultAttributes, [["path", {
  d: "M12 21a8 8 0 100-16 8 8 0 000 16z",
  key: "ubbk1n"
}], ["path", {
  d: "M5 3L2 6",
  key: "yo78bm"
}], ["path", {
  d: "M22 6l-3-3",
  key: "ry34ie"
}], ["path", {
  d: "M6 19l-2 2",
  key: "1dpmc8"
}], ["path", {
  d: "M18 19l2 2",
  key: "1v10ht"
}], ["path", {
  d: "M12 9v8",
  key: "1qin3f"
}], ["path", {
  d: "M8 13h8",
  key: "1ti47a"
}]]]);

var Album = createReactComponent("Album", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["polyline", {
  points: "11 3 11 11 14 8 17 11 17 3",
  key: "5fl13i"
}]]]);

var AlertCircle = createReactComponent("AlertCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "12",
  y1: "8",
  x2: "12",
  y2: "12",
  key: "1y3920"
}], ["line", {
  x1: "12",
  y1: "16",
  x2: "12.01",
  y2: "16",
  key: "14g55c"
}]]]);

var AlertOctagon = createReactComponent("AlertOctagon", ["svg", defaultAttributes, [["polygon", {
  points: "7.86 2 16.14 2 22 7.86 22 16.14 16.14 22 7.86 22 2 16.14 2 7.86 7.86 2",
  key: "1qo9ur"
}], ["line", {
  x1: "12",
  y1: "8",
  x2: "12",
  y2: "12",
  key: "1y3920"
}], ["line", {
  x1: "12",
  y1: "16",
  x2: "12.01",
  y2: "16",
  key: "14g55c"
}]]]);

var AlertTriangle = createReactComponent("AlertTriangle", ["svg", defaultAttributes, [["path", {
  d: "M10.29 3.86L1.82 18a2 2 0 001.71 3h16.94a2 2 0 001.71-3L13.71 3.86a2 2 0 00-3.42 0z",
  key: "16e7ti"
}], ["line", {
  x1: "12",
  y1: "9",
  x2: "12",
  y2: "13",
  key: "avsm3v"
}], ["line", {
  x1: "12",
  y1: "17",
  x2: "12.01",
  y2: "17",
  key: "1fwdlo"
}]]]);

var AlignCenter = createReactComponent("AlignCenter", ["svg", defaultAttributes, [["line", {
  x1: "21",
  y1: "6",
  x2: "3",
  y2: "6",
  key: "1kn1du"
}], ["line", {
  x1: "17",
  y1: "12",
  x2: "7",
  y2: "12",
  key: "go884h"
}], ["line", {
  x1: "19",
  y1: "18",
  x2: "5",
  y2: "18",
  key: "19guv1"
}]]]);

var AlignJustify = createReactComponent("AlignJustify", ["svg", defaultAttributes, [["line", {
  x1: "3",
  y1: "6",
  x2: "21",
  y2: "6",
  key: "1jdu5d"
}], ["line", {
  x1: "3",
  y1: "12",
  x2: "21",
  y2: "12",
  key: "thangg"
}], ["line", {
  x1: "3",
  y1: "18",
  x2: "21",
  y2: "18",
  key: "ktql6o"
}]]]);

var AlignLeft = createReactComponent("AlignLeft", ["svg", defaultAttributes, [["line", {
  x1: "21",
  y1: "6",
  x2: "3",
  y2: "6",
  key: "1kn1du"
}], ["line", {
  x1: "15",
  y1: "12",
  x2: "3",
  y2: "12",
  key: "wvjmef"
}], ["line", {
  x1: "17",
  y1: "18",
  x2: "3",
  y2: "18",
  key: "24plut"
}]]]);

var AlignRight = createReactComponent("AlignRight", ["svg", defaultAttributes, [["line", {
  x1: "21",
  y1: "6",
  x2: "3",
  y2: "6",
  key: "1kn1du"
}], ["line", {
  x1: "21",
  y1: "12",
  x2: "9",
  y2: "12",
  key: "1g0nry"
}], ["line", {
  x1: "21",
  y1: "18",
  x2: "7",
  y2: "18",
  key: "1u9cht"
}]]]);

var Anchor = createReactComponent("Anchor", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "5",
  r: "3",
  key: "19a85e"
}], ["line", {
  x1: "12",
  y1: "22",
  x2: "12",
  y2: "8",
  key: "rq7cvc"
}], ["path", {
  d: "M5 12H2a10 10 0 0020 0h-3",
  key: "yc0v05"
}]]]);

var Aperture = createReactComponent("Aperture", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "14.31",
  y1: "8",
  x2: "20.05",
  y2: "17.94",
  key: "16b32f"
}], ["line", {
  x1: "9.69",
  y1: "8",
  x2: "21.17",
  y2: "8",
  key: "qls56r"
}], ["line", {
  x1: "7.38",
  y1: "12",
  x2: "13.12",
  y2: "2.06",
  key: "qz700k"
}], ["line", {
  x1: "9.69",
  y1: "16",
  x2: "3.95",
  y2: "6.06",
  key: "18rt0g"
}], ["line", {
  x1: "14.31",
  y1: "16",
  x2: "2.83",
  y2: "16",
  key: "2om0by"
}], ["line", {
  x1: "16.62",
  y1: "12",
  x2: "10.88",
  y2: "21.94",
  key: "p9xp5d"
}]]]);

var Archive = createReactComponent("Archive", ["svg", defaultAttributes, [["polyline", {
  points: "21 8 21 21 3 21 3 8",
  key: "o37y8j"
}], ["rect", {
  x: "1",
  y: "3",
  width: "22",
  height: "5",
  key: "6p2xdd"
}], ["line", {
  x1: "10",
  y1: "12",
  x2: "14",
  y2: "12",
  key: "i3jslw"
}]]]);

var ArrowBigDown = createReactComponent("ArrowBigDown", ["svg", defaultAttributes, [["path", {
  d: "M9 3h6v11h4l-7 7-7-7h4z",
  key: "9v4vit"
}]]]);

var ArrowBigLeft = createReactComponent("ArrowBigLeft", ["svg", defaultAttributes, [["path", {
  d: "M3 12l7-7v4h11v6H10v4z",
  key: "1w83wz"
}]]]);

var ArrowBigRight = createReactComponent("ArrowBigRight", ["svg", defaultAttributes, [["path", {
  d: "M21 12l-7-7v4H3v6h11v4z",
  key: "uzio89"
}]]]);

var ArrowBigUp = createReactComponent("ArrowBigUp", ["svg", defaultAttributes, [["path", {
  d: "M9 21V10H5l7-7 7 7h-4v11z",
  key: "1hr2dw"
}]]]);

var ArrowDownCircle = createReactComponent("ArrowDownCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["polyline", {
  points: "8 12 12 16 16 12",
  key: "1fxsbv"
}], ["line", {
  x1: "12",
  y1: "8",
  x2: "12",
  y2: "16",
  key: "74zyzz"
}]]]);

var ArrowDownLeft = createReactComponent("ArrowDownLeft", ["svg", defaultAttributes, [["line", {
  x1: "17",
  y1: "7",
  x2: "7",
  y2: "17",
  key: "xiukkw"
}], ["polyline", {
  points: "17 17 7 17 7 7",
  key: "gh1zeq"
}]]]);

var ArrowDownRight = createReactComponent("ArrowDownRight", ["svg", defaultAttributes, [["line", {
  x1: "7",
  y1: "7",
  x2: "17",
  y2: "17",
  key: "s6w834"
}], ["polyline", {
  points: "17 7 17 17 7 17",
  key: "9dq2ur"
}]]]);

var ArrowDown = createReactComponent("ArrowDown", ["svg", defaultAttributes, [["line", {
  x1: "12",
  y1: "5",
  x2: "12",
  y2: "19",
  key: "1ebv7v"
}], ["polyline", {
  points: "19 12 12 19 5 12",
  key: "rbfin9"
}]]]);

var ArrowLeftCircle = createReactComponent("ArrowLeftCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["polyline", {
  points: "12 8 8 12 12 16",
  key: "xj9ozb"
}], ["line", {
  x1: "16",
  y1: "12",
  x2: "8",
  y2: "12",
  key: "nlup73"
}]]]);

var ArrowLeft = createReactComponent("ArrowLeft", ["svg", defaultAttributes, [["line", {
  x1: "19",
  y1: "12",
  x2: "5",
  y2: "12",
  key: "1p9a8d"
}], ["polyline", {
  points: "12 19 5 12 12 5",
  key: "fstcns"
}]]]);

var ArrowRightCircle = createReactComponent("ArrowRightCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["polyline", {
  points: "12 16 16 12 12 8",
  key: "y82s6g"
}], ["line", {
  x1: "8",
  y1: "12",
  x2: "16",
  y2: "12",
  key: "6hnc9b"
}]]]);

var ArrowRight = createReactComponent("ArrowRight", ["svg", defaultAttributes, [["line", {
  x1: "5",
  y1: "12",
  x2: "19",
  y2: "12",
  key: "iqslzh"
}], ["polyline", {
  points: "12 5 19 12 12 19",
  key: "1rvi6t"
}]]]);

var ArrowUpCircle = createReactComponent("ArrowUpCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["polyline", {
  points: "16 12 12 8 8 12",
  key: "11k32n"
}], ["line", {
  x1: "12",
  y1: "16",
  x2: "12",
  y2: "8",
  key: "1gthwp"
}]]]);

var ArrowUpLeft = createReactComponent("ArrowUpLeft", ["svg", defaultAttributes, [["line", {
  x1: "17",
  y1: "17",
  x2: "7",
  y2: "7",
  key: "mfqg40"
}], ["polyline", {
  points: "7 17 7 7 17 7",
  key: "hpw4z7"
}]]]);

var ArrowUpRight = createReactComponent("ArrowUpRight", ["svg", defaultAttributes, [["line", {
  x1: "7",
  y1: "17",
  x2: "17",
  y2: "7",
  key: "uesjio"
}], ["polyline", {
  points: "7 7 17 7 17 17",
  key: "p99yn6"
}]]]);

var ArrowUp = createReactComponent("ArrowUp", ["svg", defaultAttributes, [["line", {
  x1: "12",
  y1: "19",
  x2: "12",
  y2: "5",
  key: "7jz27h"
}], ["polyline", {
  points: "5 12 12 5 19 12",
  key: "15x67i"
}]]]);

var Asterisk = createReactComponent("Asterisk", ["svg", defaultAttributes, [["path", {
  d: "M12 6v12",
  key: "lqrgx8"
}], ["path", {
  d: "M17.196 9L6.804 15",
  key: "wu5l3j"
}], ["path", {
  d: "M6.804 9l10.392 6",
  key: "i3kzks"
}]]]);

var AtSign = createReactComponent("AtSign", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "4",
  key: "7pwehp"
}], ["path", {
  d: "M16 8v5a3 3 0 006 0v-1a10 10 0 10-3.92 7.94",
  key: "13mccx"
}]]]);

var Award = createReactComponent("Award", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "8",
  r: "7",
  key: "1a1y5i"
}], ["polyline", {
  points: "8.21 13.89 7 23 12 20 17 23 15.79 13.88",
  key: "1lk7zz"
}]]]);

var Axe = createReactComponent("Axe", ["svg", defaultAttributes, [["path", {
  d: "M14 12l-8.501 8.501a2.12 2.12 0 01-2.998 0h-.002a2.12 2.12 0 010-2.998L11 9.002",
  key: "1dp2c9"
}], ["path", {
  d: "M9 7l4-4 6 6h3l-.13.648a7.648 7.648 0 01-5.081 5.756L15 16v-3z",
  key: "9hiefg"
}]]]);

var BarChart2 = createReactComponent("BarChart2", ["svg", defaultAttributes, [["line", {
  x1: "18",
  y1: "20",
  x2: "18",
  y2: "10",
  key: "aot9pv"
}], ["line", {
  x1: "12",
  y1: "20",
  x2: "12",
  y2: "4",
  key: "5iquee"
}], ["line", {
  x1: "6",
  y1: "20",
  x2: "6",
  y2: "14",
  key: "1mwajh"
}]]]);

var BarChart = createReactComponent("BarChart", ["svg", defaultAttributes, [["line", {
  x1: "12",
  y1: "20",
  x2: "12",
  y2: "10",
  key: "b64o6r"
}], ["line", {
  x1: "18",
  y1: "20",
  x2: "18",
  y2: "4",
  key: "51ffxi"
}], ["line", {
  x1: "6",
  y1: "20",
  x2: "6",
  y2: "16",
  key: "4eimgl"
}]]]);

var BatteryCharging = createReactComponent("BatteryCharging", ["svg", defaultAttributes, [["path", {
  d: "M7 7H3.78A1.73 1.73 0 002 8.67v6.66A1.73 1.73 0 003.78 17H6m8-10h2.22A1.73 1.73 0 0118 8.67v6.66A1.73 1.73 0 0116.22 17H13",
  key: "hdcb3"
}], ["polyline", {
  points: "11 7 8 12 12 12 9 17",
  key: "1vjezw"
}], ["line", {
  x1: "22",
  x2: "22",
  y1: "11",
  y2: "13",
  key: "13axwy"
}]]]);

var BatteryFull = createReactComponent("BatteryFull", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "7",
  width: "16",
  height: "10",
  rx: "2",
  ry: "2",
  key: "174y1r"
}], ["line", {
  x1: "22",
  x2: "22",
  y1: "11",
  y2: "13",
  key: "13axwy"
}], ["line", {
  x1: "6",
  x2: "6",
  y1: "10",
  y2: "14",
  key: "1mnnv9"
}], ["line", {
  x1: "10",
  x2: "10",
  y1: "10",
  y2: "14",
  key: "1e2289"
}], ["line", {
  x1: "14",
  x2: "14",
  y1: "10",
  y2: "14",
  key: "1comla"
}]]]);

var BatteryLow = createReactComponent("BatteryLow", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "7",
  width: "16",
  height: "10",
  rx: "2",
  ry: "2",
  key: "174y1r"
}], ["line", {
  x1: "22",
  x2: "22",
  y1: "11",
  y2: "13",
  key: "13axwy"
}], ["line", {
  x1: "6",
  x2: "6",
  y1: "10",
  y2: "14",
  key: "1mnnv9"
}]]]);

var BatteryMedium = createReactComponent("BatteryMedium", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "7",
  width: "16",
  height: "10",
  rx: "2",
  ry: "2",
  key: "174y1r"
}], ["line", {
  x1: "22",
  x2: "22",
  y1: "11",
  y2: "13",
  key: "13axwy"
}], ["line", {
  x1: "6",
  x2: "6",
  y1: "10",
  y2: "14",
  key: "1mnnv9"
}], ["line", {
  x1: "10",
  x2: "10",
  y1: "10",
  y2: "14",
  key: "1e2289"
}]]]);

var Battery = createReactComponent("Battery", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "7",
  width: "16",
  height: "10",
  rx: "2",
  ry: "2",
  key: "174y1r"
}], ["line", {
  x1: "22",
  x2: "22",
  y1: "11",
  y2: "13",
  key: "13axwy"
}]]]);

var Beaker = createReactComponent("Beaker", ["svg", defaultAttributes, [["path", {
  d: "M4.5 3h15",
  key: "1qrsw4"
}], ["path", {
  d: "M6 3v16a2 2 0 002 2h8a2 2 0 002-2V3",
  key: "z0qw0"
}], ["path", {
  d: "M6 14h12",
  key: "79jpmc"
}]]]);

var BellMinus = createReactComponent("BellMinus", ["svg", defaultAttributes, [["path", {
  d: "M13.73 21a2 2 0 01-3.46 0",
  key: "41ys2d"
}], ["path", {
  d: "M21 5h-6",
  key: "15uff2"
}], ["path", {
  d: "M18.021 9C18.29 15.193 21 17 21 17H3s3-2 3-9a6 6 0 017-5.916",
  key: "18fh8w"
}]]]);

var BellOff = createReactComponent("BellOff", ["svg", defaultAttributes, [["path", {
  d: "M13.73 21a2 2 0 01-3.46 0",
  key: "41ys2d"
}], ["path", {
  d: "M18.63 13A17.888 17.888 0 0118 8",
  key: "182lqt"
}], ["path", {
  d: "M6.26 6.26A5.86 5.86 0 006 8c0 7-3 9-3 9h14",
  key: "1xmazw"
}], ["path", {
  d: "M18 8a6 6 0 00-9.33-5",
  key: "1b66x3"
}], ["path", {
  d: "M2 2l20 20",
  key: "131xdx"
}]]]);

var BellPlus = createReactComponent("BellPlus", ["svg", defaultAttributes, [["path", {
  d: "M18.387 12C19.198 15.799 21 17 21 17H3s3-2 3-9a6 6 0 017-5.916",
  key: "fun4fk"
}], ["path", {
  d: "M13.73 21a2 2 0 01-3.46 0",
  key: "41ys2d"
}], ["path", {
  d: "M18 2v6",
  key: "191g95"
}], ["path", {
  d: "M21 5h-6",
  key: "15uff2"
}]]]);

var Bell = createReactComponent("Bell", ["svg", defaultAttributes, [["path", {
  d: "M18 8A6 6 0 006 8c0 7-3 9-3 9h18s-3-2-3-9",
  key: "1wwper"
}], ["path", {
  d: "M13.73 21a2 2 0 01-3.46 0",
  key: "41ys2d"
}]]]);

var Bike = createReactComponent("Bike", ["svg", defaultAttributes, [["circle", {
  cx: "5.5",
  cy: "17.5",
  r: "3.5",
  key: "13rwq7"
}], ["circle", {
  cx: "18.5",
  cy: "17.5",
  r: "3.5",
  key: "1da4lf"
}], ["path", {
  d: "M15 6a1 1 0 100-2 1 1 0 000 2zm-3 11.5V14l-3-3 4-3 2 3h2",
  key: "fitlvz"
}]]]);

var Binary = createReactComponent("Binary", ["svg", defaultAttributes, [["path", {
  d: "M10 4H6v6h4V4z",
  key: "mdqhcz"
}], ["path", {
  d: "M18 14h-4v6h4v-6z",
  key: "y4gjga"
}], ["path", {
  d: "M14 4h2v6m-2 0h4",
  key: "495vtl"
}], ["path", {
  d: "M6 14h2v6m-2 0h4",
  key: "1eiql5"
}]]]);

var BluetoothConnected = createReactComponent("BluetoothConnected", ["svg", defaultAttributes, [["path", {
  d: "M7 7l10 10-5 5V2l5 5L7 17",
  key: "1vkd5p"
}], ["line", {
  x1: "18",
  y1: "12",
  y2: "12",
  x2: "21",
  key: "p4cere"
}], ["line", {
  x1: "3",
  y1: "12",
  y2: "12",
  x2: "6",
  key: "1bw2jp"
}]]]);

var BluetoothOff = createReactComponent("BluetoothOff", ["svg", defaultAttributes, [["path", {
  d: "M17 17l-5 5V12l-5 5",
  key: "17kqnf"
}], ["path", {
  d: "M2 2l20 20",
  key: "131xdx"
}], ["path", {
  d: "M14.5 9.5L17 7l-5-5v4.5",
  key: "1qgx6g"
}]]]);

var BluetoothSearching = createReactComponent("BluetoothSearching", ["svg", defaultAttributes, [["path", {
  d: "M7 7l10 10-5 5V2l5 5L7 17",
  key: "1vkd5p"
}], ["path", {
  d: "M20.83 14.83a4 4 0 000-5.66",
  key: "dfdwxo"
}], ["path", {
  d: "M18 12h.01",
  key: "1txspt"
}]]]);

var Bluetooth = createReactComponent("Bluetooth", ["svg", defaultAttributes, [["path", {
  d: "M7 7l10 10-5 5V2l5 5L7 17",
  key: "1vkd5p"
}]]]);

var Bold = createReactComponent("Bold", ["svg", defaultAttributes, [["path", {
  d: "M6 4h8a4 4 0 014 4 4 4 0 01-4 4H6z",
  key: "y8kye"
}], ["path", {
  d: "M6 12h9a4 4 0 014 4 4 4 0 01-4 4H6z",
  key: "e5wyww"
}]]]);

var BookOpen = createReactComponent("BookOpen", ["svg", defaultAttributes, [["path", {
  d: "M2 3h6a4 4 0 014 4v14a3 3 0 00-3-3H2z",
  key: "feqv6p"
}], ["path", {
  d: "M22 3h-6a4 4 0 00-4 4v14a3 3 0 013-3h7z",
  key: "14x2xk"
}]]]);

var Book = createReactComponent("Book", ["svg", defaultAttributes, [["path", {
  d: "M4 19.5A2.5 2.5 0 016.5 17H20",
  key: "1bmfnz"
}], ["path", {
  d: "M6.5 2H20v20H6.5A2.5 2.5 0 014 19.5v-15A2.5 2.5 0 016.5 2z",
  key: "1qo54g"
}]]]);

var Bookmark = createReactComponent("Bookmark", ["svg", defaultAttributes, [["path", {
  d: "M19 21l-7-5-7 5V5a2 2 0 012-2h10a2 2 0 012 2z",
  key: "qip9cr"
}]]]);

var Bot = createReactComponent("Bot", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "11",
  width: "18",
  height: "10",
  rx: "2",
  key: "rk0hcj"
}], ["circle", {
  cx: "12",
  cy: "5",
  r: "2",
  key: "1i986l"
}], ["path", {
  d: "M12 7v4",
  key: "1fq99g"
}], ["line", {
  x1: "8",
  y1: "16",
  x2: "8",
  y2: "16",
  key: "uw7els"
}], ["line", {
  x1: "16",
  y1: "16",
  x2: "16",
  y2: "16",
  key: "6xq6mo"
}]]]);

var BoxSelect = createReactComponent("BoxSelect", ["svg", defaultAttributes, [["path", {
  d: "M4 2a2 2 0 00-2 2",
  key: "1eheh9"
}], ["line", {
  x1: "8",
  y1: "2",
  x2: "10",
  y2: "2",
  key: "ecd3k9"
}], ["line", {
  x1: "14",
  y1: "2",
  x2: "16",
  y2: "2",
  key: "ply5xu"
}], ["path", {
  d: "M4 22a2 2 0 01-2-2",
  key: "wwvhe0"
}], ["line", {
  x1: "22",
  y1: "8",
  x2: "22",
  y2: "10",
  key: "1vwrgu"
}], ["line", {
  x1: "22",
  y1: "14",
  x2: "22",
  y2: "16",
  key: "2fkjma"
}], ["path", {
  d: "M22 20a2 2 0 01-2 2",
  key: "1lm9s5"
}], ["line", {
  x1: "14",
  y1: "22",
  x2: "16",
  y2: "22",
  key: "185ln8"
}], ["line", {
  x1: "8",
  y1: "22",
  x2: "10",
  y2: "22",
  key: "1501e8"
}], ["path", {
  d: "M20 2a2 2 0 012 2",
  key: "8qmqb0"
}], ["line", {
  x1: "2",
  y1: "14",
  x2: "2",
  y2: "16",
  key: "bcn2gi"
}], ["line", {
  x1: "2",
  y1: "8",
  x2: "2",
  y2: "10",
  key: "16xi07"
}]]]);

var Box = createReactComponent("Box", ["svg", defaultAttributes, [["path", {
  d: "M21 16V8a2 2 0 00-1-1.73l-7-4a2 2 0 00-2 0l-7 4A2 2 0 003 8v8a2 2 0 001 1.73l7 4a2 2 0 002 0l7-4A2 2 0 0021 16z",
  key: "eor234"
}], ["polyline", {
  points: "3.27 6.96 12 12.01 20.73 6.96",
  key: "5vdlqk"
}], ["line", {
  x1: "12",
  y1: "22.08",
  x2: "12",
  y2: "12",
  key: "n2060l"
}]]]);

var Briefcase = createReactComponent("Briefcase", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "7",
  width: "20",
  height: "14",
  rx: "2",
  ry: "2",
  key: "1yeia0"
}], ["path", {
  d: "M16 21V5a2 2 0 00-2-2h-4a2 2 0 00-2 2v16",
  key: "1eja0h"
}]]]);

var Brush = createReactComponent("Brush", ["svg", defaultAttributes, [["path", {
  d: "M9.06 11.9l8.07-8.06a2.85 2.85 0 114.03 4.03l-8.06 8.08",
  key: "1h5c0a"
}], ["path", {
  d: "M7.07 14.94c-1.66 0-3 1.35-3 3.02 0 1.33-2.5 1.52-2 2.02 1.08 1.1 2.49 2.02 4 2.02 2.2 0 4-1.8 4-4.04a3.01 3.01 0 00-3-3.02z",
  key: "1w5cap"
}]]]);

var Building = createReactComponent("Building", ["svg", defaultAttributes, [["rect", {
  x: "4",
  y: "2",
  width: "16",
  height: "20",
  rx: "2",
  ry: "2",
  key: "1p0e78"
}], ["path", {
  d: "M9 22v-4h6v4",
  key: "ujjh2e"
}], ["path", {
  d: "M8 6h.01",
  key: "1wzyqp"
}], ["path", {
  d: "M16 6h.01",
  key: "1lnkkb"
}], ["path", {
  d: "M12 6h.01",
  key: "j7ydwe"
}], ["path", {
  d: "M12 10h.01",
  key: "11wgo0"
}], ["path", {
  d: "M12 14h.01",
  key: "rao6x9"
}], ["path", {
  d: "M16 10h.01",
  key: "1yhyjk"
}], ["path", {
  d: "M16 14h.01",
  key: "1tqadc"
}], ["path", {
  d: "M8 10h.01",
  key: "9s5hzm"
}], ["path", {
  d: "M8 14h.01",
  key: "2jug5y"
}]]]);

var Bus = createReactComponent("Bus", ["svg", defaultAttributes, [["path", {
  d: "M19 17h2l.64-2.54a6 6 0 000-2.92l-1.07-4.27A3 3 0 0017.66 5H4a2 2 0 00-2 2v10h2m10 0h-4",
  key: "joxivo"
}], ["circle", {
  cx: "6.5",
  cy: "17.5",
  r: "2.5",
  key: "1qcjyb"
}], ["circle", {
  cx: "16.5",
  cy: "17.5",
  r: "2.5",
  key: "1da6r5"
}]]]);

var Calculator = createReactComponent("Calculator", ["svg", defaultAttributes, [["rect", {
  x: "4",
  y: "2",
  width: "16",
  height: "20",
  rx: "2",
  key: "1ft04i"
}], ["line", {
  x1: "8",
  x2: "16",
  y1: "6",
  y2: "6",
  key: "19xyj3"
}], ["line", {
  x1: "16",
  x2: "16",
  y1: "14",
  y2: "18",
  key: "1hm6pj"
}], ["path", {
  d: "M16 10h.01",
  key: "1yhyjk"
}], ["path", {
  d: "M12 10h.01",
  key: "11wgo0"
}], ["path", {
  d: "M8 10h.01",
  key: "9s5hzm"
}], ["path", {
  d: "M12 14h.01",
  key: "rao6x9"
}], ["path", {
  d: "M8 14h.01",
  key: "2jug5y"
}], ["path", {
  d: "M12 18h.01",
  key: "5lmu0x"
}], ["path", {
  d: "M8 18h.01",
  key: "1sjb0i"
}]]]);

var Calendar = createReactComponent("Calendar", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "4",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "j76o"
}], ["line", {
  x1: "16",
  y1: "2",
  x2: "16",
  y2: "6",
  key: "xs39ck"
}], ["line", {
  x1: "8",
  y1: "2",
  x2: "8",
  y2: "6",
  key: "1t0wan"
}], ["line", {
  x1: "3",
  y1: "10",
  x2: "21",
  y2: "10",
  key: "9li9q8"
}]]]);

var CameraOff = createReactComponent("CameraOff", ["svg", defaultAttributes, [["line", {
  x1: "2",
  y1: "2",
  x2: "22",
  y2: "22",
  key: "1ehfaz"
}], ["path", {
  d: "M9.5 4h5L17 7h3a2 2 0 012 2v7.5M7 7H4a2 2 0 00-2 2v9a2 2 0 002 2h16",
  key: "11g4us"
}], ["path", {
  d: "M14.121 15.121A3 3 0 119.88 10.88",
  key: "ijgh50"
}]]]);

var Camera = createReactComponent("Camera", ["svg", defaultAttributes, [["path", {
  d: "M14.5 4h-5L7 7H4a2 2 0 00-2 2v9a2 2 0 002 2h16a2 2 0 002-2V9a2 2 0 00-2-2h-3l-2.5-3z",
  key: "13xylk"
}], ["circle", {
  cx: "12",
  cy: "13",
  r: "3",
  key: "5yg47v"
}]]]);

var Car = createReactComponent("Car", ["svg", defaultAttributes, [["path", {
  d: "M14 16H9m10 0h3v-3.15a1 1 0 00-.84-.99L16 11l-2.7-3.6a1 1 0 00-.8-.4H5.24a2 2 0 00-1.8 1.1l-.8 1.63A6 6 0 002 12.42V16h2",
  key: "6uwmab"
}], ["circle", {
  cx: "6.5",
  cy: "16.5",
  r: "2.5",
  key: "qnl2dt"
}], ["circle", {
  cx: "16.5",
  cy: "16.5",
  r: "2.5",
  key: "1t12lf"
}]]]);

var Cast = createReactComponent("Cast", ["svg", defaultAttributes, [["path", {
  d: "M2 16.1A5 5 0 015.9 20M2 12.05A9 9 0 019.95 20M2 8V6a2 2 0 012-2h16a2 2 0 012 2v12a2 2 0 01-2 2h-6",
  key: "1v255n"
}], ["line", {
  x1: "2",
  y1: "20",
  x2: "2.01",
  y2: "20",
  key: "1pj7gx"
}]]]);

var CheckCircle = createReactComponent("CheckCircle", ["svg", defaultAttributes, [["path", {
  d: "M22 11.08V12a10 10 0 11-5.93-9.14",
  key: "110yx7"
}], ["polyline", {
  points: "22 4 12 14.01 9 11.01",
  key: "ny83mg"
}]]]);

var CheckSquare = createReactComponent("CheckSquare", ["svg", defaultAttributes, [["polyline", {
  points: "9 11 12 14 22 4",
  key: "1hoiwr"
}], ["path", {
  d: "M21 12v7a2 2 0 01-2 2H5a2 2 0 01-2-2V5a2 2 0 012-2h11",
  key: "fq6zhn"
}]]]);

var Check = createReactComponent("Check", ["svg", defaultAttributes, [["polyline", {
  points: "20 6 9 17 4 12",
  key: "19n91j"
}]]]);

var ChevronDown = createReactComponent("ChevronDown", ["svg", defaultAttributes, [["polyline", {
  points: "6 9 12 15 18 9",
  key: "113ckw"
}]]]);

var ChevronLeft = createReactComponent("ChevronLeft", ["svg", defaultAttributes, [["polyline", {
  points: "15 18 9 12 15 6",
  key: "1pf9z0"
}]]]);

var ChevronRight = createReactComponent("ChevronRight", ["svg", defaultAttributes, [["polyline", {
  points: "9 18 15 12 9 6",
  key: "wrg7ij"
}]]]);

var ChevronUp = createReactComponent("ChevronUp", ["svg", defaultAttributes, [["polyline", {
  points: "18 15 12 9 6 15",
  key: "1ipzt1"
}]]]);

var ChevronsDown = createReactComponent("ChevronsDown", ["svg", defaultAttributes, [["polyline", {
  points: "7 13 12 18 17 13",
  key: "1aqf8r"
}], ["polyline", {
  points: "7 6 12 11 17 6",
  key: "lz5bb5"
}]]]);

var ChevronsLeft = createReactComponent("ChevronsLeft", ["svg", defaultAttributes, [["polyline", {
  points: "11 17 6 12 11 7",
  key: "1yp1jo"
}], ["polyline", {
  points: "18 17 13 12 18 7",
  key: "1mbq08"
}]]]);

var ChevronsRight = createReactComponent("ChevronsRight", ["svg", defaultAttributes, [["polyline", {
  points: "13 17 18 12 13 7",
  key: "8cl2ko"
}], ["polyline", {
  points: "6 17 11 12 6 7",
  key: "6q5tup"
}]]]);

var ChevronsUp = createReactComponent("ChevronsUp", ["svg", defaultAttributes, [["polyline", {
  points: "17 11 12 6 7 11",
  key: "pbh56f"
}], ["polyline", {
  points: "17 18 12 13 7 18",
  key: "4kr3s3"
}]]]);

var Chrome = createReactComponent("Chrome", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "4",
  key: "7pwehp"
}], ["line", {
  x1: "21.17",
  y1: "8",
  x2: "12",
  y2: "8",
  key: "1mge5x"
}], ["line", {
  x1: "3.95",
  y1: "6.06",
  x2: "8.54",
  y2: "14",
  key: "1td8ah"
}], ["line", {
  x1: "10.88",
  y1: "21.94",
  x2: "15.46",
  y2: "14",
  key: "k3fms2"
}]]]);

var Circle = createReactComponent("Circle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}]]]);

var Clipboard = createReactComponent("Clipboard", ["svg", defaultAttributes, [["path", {
  d: "M16 4h2a2 2 0 012 2v14a2 2 0 01-2 2H6a2 2 0 01-2-2V6a2 2 0 012-2h2",
  key: "6nzyoi"
}], ["rect", {
  x: "8",
  y: "2",
  width: "8",
  height: "4",
  rx: "1",
  ry: "1",
  key: "2zmm01"
}]]]);

var Clock = createReactComponent("Clock", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["polyline", {
  points: "12 6 12 12 16 14",
  key: "1nl836"
}]]]);

var CloudDrizzle = createReactComponent("CloudDrizzle", ["svg", defaultAttributes, [["line", {
  x1: "8",
  y1: "19",
  x2: "8",
  y2: "21",
  key: "ac69bv"
}], ["line", {
  x1: "8",
  y1: "13",
  x2: "8",
  y2: "15",
  key: "3io7q"
}], ["line", {
  x1: "16",
  y1: "19",
  x2: "16",
  y2: "21",
  key: "1b5vz3"
}], ["line", {
  x1: "16",
  y1: "13",
  x2: "16",
  y2: "15",
  key: "1ybewi"
}], ["line", {
  x1: "12",
  y1: "21",
  x2: "12",
  y2: "23",
  key: "t04rde"
}], ["line", {
  x1: "12",
  y1: "15",
  x2: "12",
  y2: "17",
  key: "1r59ua"
}], ["path", {
  d: "M20 16.58A5 5 0 0018 7h-1.26A8 8 0 104 15.25",
  key: "1qxhfx"
}]]]);

var CloudLightning = createReactComponent("CloudLightning", ["svg", defaultAttributes, [["path", {
  d: "M19 16.9A5 5 0 0018 7h-1.26a8 8 0 10-11.62 9",
  key: "1yaqsn"
}], ["polyline", {
  points: "13 11 9 17 15 17 11 23",
  key: "pzjnu5"
}]]]);

var CloudOff = createReactComponent("CloudOff", ["svg", defaultAttributes, [["path", {
  d: "M22.61 16.95A5 5 0 0018 10h-1.26a8 8 0 00-7.05-6M5 5a8 8 0 004 15h9a5 5 0 001.7-.3",
  key: "1k46wa"
}], ["line", {
  x1: "1",
  y1: "1",
  x2: "23",
  y2: "23",
  key: "czowfk"
}]]]);

var CloudRain = createReactComponent("CloudRain", ["svg", defaultAttributes, [["line", {
  x1: "16",
  y1: "13",
  x2: "16",
  y2: "21",
  key: "1j5n7q"
}], ["line", {
  x1: "8",
  y1: "13",
  x2: "8",
  y2: "21",
  key: "1i6fuc"
}], ["line", {
  x1: "12",
  y1: "15",
  x2: "12",
  y2: "23",
  key: "1ydugz"
}], ["path", {
  d: "M20 16.58A5 5 0 0018 7h-1.26A8 8 0 104 15.25",
  key: "1qxhfx"
}]]]);

var CloudSnow = createReactComponent("CloudSnow", ["svg", defaultAttributes, [["path", {
  d: "M20 17.58A5 5 0 0018 8h-1.26A8 8 0 104 16.25",
  key: "1kj545"
}], ["line", {
  x1: "8",
  y1: "16",
  x2: "8.01",
  y2: "16",
  key: "1m975f"
}], ["line", {
  x1: "8",
  y1: "20",
  x2: "8.01",
  y2: "20",
  key: "aukqwv"
}], ["line", {
  x1: "12",
  y1: "18",
  x2: "12.01",
  y2: "18",
  key: "rbtten"
}], ["line", {
  x1: "12",
  y1: "22",
  x2: "12.01",
  y2: "22",
  key: "1qhv3z"
}], ["line", {
  x1: "16",
  y1: "16",
  x2: "16.01",
  y2: "16",
  key: "doshnj"
}], ["line", {
  x1: "16",
  y1: "20",
  x2: "16.01",
  y2: "20",
  key: "th9hkv"
}]]]);

var Cloud = createReactComponent("Cloud", ["svg", defaultAttributes, [["path", {
  d: "M18 10h-1.26A8 8 0 109 20h9a5 5 0 000-10z",
  key: "1xwwv6"
}]]]);

var Clover = createReactComponent("Clover", ["svg", defaultAttributes, [["path", {
  d: "M16.2 3.8a2.7 2.7 0 00-3.81 0l-.4.38-.4-.4a2.7 2.7 0 00-3.82 0C6.73 4.85 6.67 6.64 8 8l4 4 4-4c1.33-1.36 1.27-3.15.2-4.2z",
  key: "nt5bkz"
}], ["path", {
  d: "M8 8c-1.36-1.33-3.15-1.27-4.2-.2a2.7 2.7 0 000 3.81l.38.4-.4.4a2.7 2.7 0 000 3.82C4.85 17.27 6.64 17.33 8 16",
  key: "1hrxst"
}], ["path", {
  d: "M16 16c1.36 1.33 3.15 1.27 4.2.2a2.7 2.7 0 000-3.81l-.38-.4.4-.4a2.7 2.7 0 000-3.82C19.15 6.73 17.36 6.67 16 8",
  key: "1mcxp0"
}], ["path", {
  d: "M7.8 20.2a2.7 2.7 0 003.81 0l.4-.38.4.4a2.7 2.7 0 003.82 0c1.06-1.06 1.12-2.85-.21-4.21l-4-4-4 4c-1.33 1.36-1.27 3.15-.2 4.2z",
  key: "1ub18x"
}], ["path", {
  d: "M7 17l-5 5",
  key: "7up3ks"
}]]]);

var Code = createReactComponent("Code", ["svg", defaultAttributes, [["polyline", {
  points: "16 18 22 12 16 6",
  key: "t8aahb"
}], ["polyline", {
  points: "8 6 2 12 8 18",
  key: "15ak9b"
}]]]);

var Codepen = createReactComponent("Codepen", ["svg", defaultAttributes, [["polygon", {
  points: "12 2 22 8.5 22 15.5 12 22 2 15.5 2 8.5 12 2",
  key: "1512qd"
}], ["line", {
  x1: "12",
  y1: "22",
  x2: "12",
  y2: "15.5",
  key: "hz0be7"
}], ["polyline", {
  points: "22 8.5 12 15.5 2 8.5",
  key: "ta10jh"
}], ["polyline", {
  points: "2 15.5 12 8.5 22 15.5",
  key: "1rwkas"
}], ["line", {
  x1: "12",
  y1: "2",
  x2: "12",
  y2: "8.5",
  key: "1y8rn7"
}]]]);

var Codesandbox = createReactComponent("Codesandbox", ["svg", defaultAttributes, [["path", {
  d: "M21 16V8a2 2 0 00-1-1.73l-7-4a2 2 0 00-2 0l-7 4A2 2 0 003 8v8a2 2 0 001 1.73l7 4a2 2 0 002 0l7-4A2 2 0 0021 16z",
  key: "eor234"
}], ["polyline", {
  points: "7.5 4.21 12 6.81 16.5 4.21",
  key: "dglsmp"
}], ["polyline", {
  points: "7.5 19.79 7.5 14.6 3 12",
  key: "6t1eom"
}], ["polyline", {
  points: "21 12 16.5 14.6 16.5 19.79",
  key: "15hlzw"
}], ["polyline", {
  points: "3.27 6.96 12 12.01 20.73 6.96",
  key: "5vdlqk"
}], ["line", {
  x1: "12",
  y1: "22.08",
  x2: "12",
  y2: "12",
  key: "n2060l"
}]]]);

var Coffee = createReactComponent("Coffee", ["svg", defaultAttributes, [["path", {
  d: "M18 8h1a4 4 0 010 8h-1",
  key: "g6jb7s"
}], ["path", {
  d: "M2 8h16v9a4 4 0 01-4 4H6a4 4 0 01-4-4V8z",
  key: "1t67t2"
}], ["line", {
  x1: "6",
  y1: "1",
  x2: "6",
  y2: "4",
  key: "1w7hat"
}], ["line", {
  x1: "10",
  y1: "1",
  x2: "10",
  y2: "4",
  key: "zjoenp"
}], ["line", {
  x1: "14",
  y1: "1",
  x2: "14",
  y2: "4",
  key: "1s8gwc"
}]]]);

var Coins = createReactComponent("Coins", ["svg", defaultAttributes, [["circle", {
  cx: "8",
  cy: "8",
  r: "7",
  key: "j6dzdq"
}], ["path", {
  d: "M19.5 9.94a7 7 0 11-9.56 9.56",
  key: "1crc7a"
}], ["path", {
  d: "M7 6h1v4",
  key: "190gst"
}], ["path", {
  d: "M17.3 14.3l.7.7-2.8 2.8",
  key: "1je1e1"
}]]]);

var Columns = createReactComponent("Columns", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["line", {
  x1: "12",
  y1: "3",
  x2: "12",
  y2: "21",
  key: "1cuoio"
}]]]);

var Command = createReactComponent("Command", ["svg", defaultAttributes, [["path", {
  d: "M18 3a3 3 0 00-3 3v12a3 3 0 003 3 3 3 0 003-3 3 3 0 00-3-3H6a3 3 0 00-3 3 3 3 0 003 3 3 3 0 003-3V6a3 3 0 00-3-3 3 3 0 00-3 3 3 3 0 003 3h12a3 3 0 003-3 3 3 0 00-3-3z",
  key: "1eeb3l"
}]]]);

var Compass = createReactComponent("Compass", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["polygon", {
  points: "16.24 7.76 14.12 14.12 7.76 16.24 9.88 9.88 16.24 7.76",
  key: "aph1zg"
}]]]);

var Contact = createReactComponent("Contact", ["svg", defaultAttributes, [["path", {
  d: "M19 22H5c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2h14c1.1 0 2 .9 2 2v14c0 1.1-.9 2-2 2z",
  key: "3h9131"
}], ["line", {
  x1: "16",
  y1: "2",
  x2: "16",
  y2: "4",
  key: "rfgxli"
}], ["line", {
  x1: "8",
  y1: "2",
  x2: "8",
  y2: "4",
  key: "1sshf5"
}], ["circle", {
  cx: "12",
  cy: "11",
  r: "3",
  key: "uqgcwp"
}], ["path", {
  d: "M17 18.5c-1.4-1-3.1-1.5-5-1.5s-3.6.6-5 1.5",
  key: "1c7dp2"
}]]]);

var Contrast = createReactComponent("Contrast", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["path", {
  d: "M12 18a6 6 0 000-12v12z",
  key: "6cp19i"
}]]]);

var Copy = createReactComponent("Copy", ["svg", defaultAttributes, [["rect", {
  x: "9",
  y: "9",
  width: "13",
  height: "13",
  rx: "2",
  ry: "2",
  key: "1ebfjm"
}], ["path", {
  d: "M5 15H4a2 2 0 01-2-2V4a2 2 0 012-2h9a2 2 0 012 2v1",
  key: "8srf0o"
}]]]);

var Copyleft = createReactComponent("Copyleft", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["path", {
  d: "M9 9.35a4 4 0 110 5.3",
  key: "1jjrhe"
}]]]);

var Copyright = createReactComponent("Copyright", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["path", {
  d: "M15 9.354a4 4 0 100 5.292",
  key: "1eezgv"
}]]]);

var CornerDownLeft = createReactComponent("CornerDownLeft", ["svg", defaultAttributes, [["polyline", {
  points: "9 10 4 15 9 20",
  key: "qm0z7k"
}], ["path", {
  d: "M20 4v7a4 4 0 01-4 4H4",
  key: "12q9n7"
}]]]);

var CornerDownRight = createReactComponent("CornerDownRight", ["svg", defaultAttributes, [["polyline", {
  points: "15 10 20 15 15 20",
  key: "1ffsyz"
}], ["path", {
  d: "M4 4v7a4 4 0 004 4h12",
  key: "1cpydb"
}]]]);

var CornerLeftDown = createReactComponent("CornerLeftDown", ["svg", defaultAttributes, [["polyline", {
  points: "14 15 9 20 4 15",
  key: "n54egp"
}], ["path", {
  d: "M20 4h-7a4 4 0 00-4 4v12",
  key: "lli475"
}]]]);

var CornerLeftUp = createReactComponent("CornerLeftUp", ["svg", defaultAttributes, [["polyline", {
  points: "14 9 9 4 4 9",
  key: "1fwp9g"
}], ["path", {
  d: "M20 20h-7a4 4 0 01-4-4V4",
  key: "12dfkc"
}]]]);

var CornerRightDown = createReactComponent("CornerRightDown", ["svg", defaultAttributes, [["polyline", {
  points: "10 15 15 20 20 15",
  key: "cidjhy"
}], ["path", {
  d: "M4 4h7a4 4 0 014 4v12",
  key: "fw55me"
}]]]);

var CornerRightUp = createReactComponent("CornerRightUp", ["svg", defaultAttributes, [["polyline", {
  points: "10 9 15 4 20 9",
  key: "j9ek3k"
}], ["path", {
  d: "M4 20h7a4 4 0 004-4V4",
  key: "6vjwt7"
}]]]);

var CornerUpLeft = createReactComponent("CornerUpLeft", ["svg", defaultAttributes, [["polyline", {
  points: "9 14 4 9 9 4",
  key: "17q6t5"
}], ["path", {
  d: "M20 20v-7a4 4 0 00-4-4H4",
  key: "kxfyy5"
}]]]);

var CornerUpRight = createReactComponent("CornerUpRight", ["svg", defaultAttributes, [["polyline", {
  points: "15 14 20 9 15 4",
  key: "10r5bc"
}], ["path", {
  d: "M4 20v-7a4 4 0 014-4h12",
  key: "15eub8"
}]]]);

var Cpu = createReactComponent("Cpu", ["svg", defaultAttributes, [["rect", {
  x: "4",
  y: "4",
  width: "16",
  height: "16",
  rx: "2",
  ry: "2",
  key: "15ziuw"
}], ["rect", {
  x: "9",
  y: "9",
  width: "6",
  height: "6",
  key: "bgbeli"
}], ["line", {
  x1: "9",
  y1: "1",
  x2: "9",
  y2: "4",
  key: "1qmpe9"
}], ["line", {
  x1: "15",
  y1: "1",
  x2: "15",
  y2: "4",
  key: "3eqc0l"
}], ["line", {
  x1: "9",
  y1: "20",
  x2: "9",
  y2: "23",
  key: "1uyze4"
}], ["line", {
  x1: "15",
  y1: "20",
  x2: "15",
  y2: "23",
  key: "111a3f"
}], ["line", {
  x1: "20",
  y1: "9",
  x2: "23",
  y2: "9",
  key: "1xzigm"
}], ["line", {
  x1: "20",
  y1: "14",
  x2: "23",
  y2: "14",
  key: "1strci"
}], ["line", {
  x1: "1",
  y1: "9",
  x2: "4",
  y2: "9",
  key: "fyl7h1"
}], ["line", {
  x1: "1",
  y1: "14",
  x2: "4",
  y2: "14",
  key: "kued05"
}]]]);

var CreditCard = createReactComponent("CreditCard", ["svg", defaultAttributes, [["rect", {
  x: "1",
  y: "4",
  width: "22",
  height: "16",
  rx: "2",
  ry: "2",
  key: "16xw5y"
}], ["line", {
  x1: "1",
  y1: "10",
  x2: "23",
  y2: "10",
  key: "rh2z0g"
}]]]);

var Crop = createReactComponent("Crop", ["svg", defaultAttributes, [["path", {
  d: "M6.13 1L6 16a2 2 0 002 2h15",
  key: "1gox95"
}], ["path", {
  d: "M1 6.13L16 6a2 2 0 012 2v15",
  key: "1c7eum"
}]]]);

var Cross = createReactComponent("Cross", ["svg", defaultAttributes, [["path", {
  d: "M11 2a2 2 0 00-2 2v5H4a2 2 0 00-2 2v2c0 1.1.9 2 2 2h5v5c0 1.1.9 2 2 2h2a2 2 0 002-2v-5h5a2 2 0 002-2v-2a2 2 0 00-2-2h-5V4a2 2 0 00-2-2h-2z",
  key: "tfkc10"
}]]]);

var Crosshair = createReactComponent("Crosshair", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "22",
  y1: "12",
  x2: "18",
  y2: "12",
  key: "gg5kc9"
}], ["line", {
  x1: "6",
  y1: "12",
  x2: "2",
  y2: "12",
  key: "h7b850"
}], ["line", {
  x1: "12",
  y1: "6",
  x2: "12",
  y2: "2",
  key: "m8s3ro"
}], ["line", {
  x1: "12",
  y1: "22",
  x2: "12",
  y2: "18",
  key: "geem49"
}]]]);

var Crown = createReactComponent("Crown", ["svg", defaultAttributes, [["path", {
  d: "M2 4l3 12h14l3-12-6 7-4-7-4 7-6-7zm3 16h14",
  key: "awn3d0"
}]]]);

var Database = createReactComponent("Database", ["svg", defaultAttributes, [["ellipse", {
  cx: "12",
  cy: "5",
  rx: "9",
  ry: "3",
  key: "oewo14"
}], ["path", {
  d: "M21 12c0 1.66-4 3-9 3s-9-1.34-9-3",
  key: "1wf6s5"
}], ["path", {
  d: "M3 5v14c0 1.66 4 3 9 3s9-1.34 9-3V5",
  key: "1klpzq"
}]]]);

var Delete = createReactComponent("Delete", ["svg", defaultAttributes, [["path", {
  d: "M21 4H8l-7 8 7 8h13a2 2 0 002-2V6a2 2 0 00-2-2z",
  key: "76eywn"
}], ["line", {
  x1: "18",
  y1: "9",
  x2: "12",
  y2: "15",
  key: "1f8sgm"
}], ["line", {
  x1: "12",
  y1: "9",
  x2: "18",
  y2: "15",
  key: "qf4fdz"
}]]]);

var Disc = createReactComponent("Disc", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "3",
  key: "n6u7h6"
}]]]);

var DivideCircle = createReactComponent("DivideCircle", ["svg", defaultAttributes, [["line", {
  x1: "8",
  y1: "12",
  x2: "16",
  y2: "12",
  key: "6hnc9b"
}], ["line", {
  x1: "12",
  y1: "16",
  x2: "12",
  y2: "16",
  key: "1f7hkw"
}], ["line", {
  x1: "12",
  y1: "8",
  x2: "12",
  y2: "8",
  key: "1tsolw"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}]]]);

var DivideSquare = createReactComponent("DivideSquare", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["line", {
  x1: "8",
  y1: "12",
  x2: "16",
  y2: "12",
  key: "6hnc9b"
}], ["line", {
  x1: "12",
  y1: "16",
  x2: "12",
  y2: "16",
  key: "1f7hkw"
}], ["line", {
  x1: "12",
  y1: "8",
  x2: "12",
  y2: "8",
  key: "1tsolw"
}]]]);

var Divide = createReactComponent("Divide", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "6",
  r: "1",
  key: "g09ld9"
}], ["line", {
  x1: "5",
  y1: "12",
  x2: "19",
  y2: "12",
  key: "iqslzh"
}], ["circle", {
  cx: "12",
  cy: "18",
  r: "1",
  key: "1r2cf2"
}]]]);

var DollarSign = createReactComponent("DollarSign", ["svg", defaultAttributes, [["line", {
  x1: "12",
  y1: "1",
  x2: "12",
  y2: "23",
  key: "1gx242"
}], ["path", {
  d: "M17 5H9.5a3.5 3.5 0 000 7h5a3.5 3.5 0 010 7H6",
  key: "3516nr"
}]]]);

var DownloadCloud = createReactComponent("DownloadCloud", ["svg", defaultAttributes, [["polyline", {
  points: "8 17 12 21 16 17",
  key: "o2c958"
}], ["line", {
  x1: "12",
  y1: "12",
  x2: "12",
  y2: "21",
  key: "16to6l"
}], ["path", {
  d: "M20.88 18.09A5 5 0 0018 9h-1.26A8 8 0 103 16.29",
  key: "dsu18w"
}]]]);

var Download = createReactComponent("Download", ["svg", defaultAttributes, [["path", {
  d: "M21 15v4a2 2 0 01-2 2H5a2 2 0 01-2-2v-4",
  key: "x5hunq"
}], ["polyline", {
  points: "7 10 12 15 17 10",
  key: "uq33c5"
}], ["line", {
  x1: "12",
  y1: "15",
  x2: "12",
  y2: "3",
  key: "32l95z"
}]]]);

var Dribbble = createReactComponent("Dribbble", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["path", {
  d: "M8.56 2.75c4.37 6.03 6.02 9.42 8.03 17.72m2.54-15.38c-3.72 4.35-8.94 5.66-16.88 5.85m19.5 1.9c-3.5-.93-6.63-.82-8.94 0-2.58.92-5.01 2.86-7.44 6.32",
  key: "1y16kq"
}]]]);

var Droplet = createReactComponent("Droplet", ["svg", defaultAttributes, [["path", {
  d: "M12 22a7 7 0 007-7c0-2-1-3.9-3-5.5s-3.5-4-4-6.5c-.5 2.5-2 4.9-4 6.5C6 11.1 5 13 5 15a7 7 0 007 7z",
  key: "1qtmz2"
}]]]);

var Edit2 = createReactComponent("Edit2", ["svg", defaultAttributes, [["path", {
  d: "M17 3a2.828 2.828 0 114 4L7.5 20.5 2 22l1.5-5.5L17 3z",
  key: "7yj3lx"
}]]]);

var Edit3 = createReactComponent("Edit3", ["svg", defaultAttributes, [["path", {
  d: "M12 20h9",
  key: "1lfwq3"
}], ["path", {
  d: "M16.5 3.5a2.121 2.121 0 013 3L7 19l-4 1 1-4L16.5 3.5z",
  key: "1ho5ni"
}]]]);

var Edit = createReactComponent("Edit", ["svg", defaultAttributes, [["path", {
  d: "M11 4H4a2 2 0 00-2 2v14a2 2 0 002 2h14a2 2 0 002-2v-7",
  key: "18kr1r"
}], ["path", {
  d: "M18.5 2.5a2.121 2.121 0 013 3L12 15l-4 1 1-4 9.5-9.5z",
  key: "1hedc5"
}]]]);

var EqualNot = createReactComponent("EqualNot", ["svg", defaultAttributes, [["line", {
  x1: "5",
  y1: "9",
  x2: "19",
  y2: "9",
  key: "wuwez1"
}], ["line", {
  x1: "5",
  y1: "15",
  x2: "19",
  y2: "15",
  key: "h3flfx"
}], ["line", {
  x1: "19",
  y1: "5",
  x2: "5",
  y2: "19",
  key: "18lstb"
}]]]);

var Equal = createReactComponent("Equal", ["svg", defaultAttributes, [["line", {
  x1: "5",
  y1: "9",
  x2: "19",
  y2: "9",
  key: "wuwez1"
}], ["line", {
  x1: "5",
  y1: "15",
  x2: "19",
  y2: "15",
  key: "h3flfx"
}]]]);

var ExternalLink = createReactComponent("ExternalLink", ["svg", defaultAttributes, [["path", {
  d: "M18 13v6a2 2 0 01-2 2H5a2 2 0 01-2-2V8a2 2 0 012-2h6",
  key: "1nnhnp"
}], ["polyline", {
  points: "15 3 21 3 21 9",
  key: "8bv4a6"
}], ["line", {
  x1: "10",
  y1: "14",
  x2: "21",
  y2: "3",
  key: "viqih0"
}]]]);

var EyeOff = createReactComponent("EyeOff", ["svg", defaultAttributes, [["path", {
  d: "M17.94 17.94A10.07 10.07 0 0112 20c-7 0-11-8-11-8a18.45 18.45 0 015.06-5.94M9.9 4.24A9.12 9.12 0 0112 4c7 0 11 8 11 8a18.5 18.5 0 01-2.16 3.19m-6.72-1.07a3 3 0 11-4.24-4.24",
  key: "1nxafh"
}], ["line", {
  x1: "1",
  y1: "1",
  x2: "23",
  y2: "23",
  key: "czowfk"
}]]]);

var Eye = createReactComponent("Eye", ["svg", defaultAttributes, [["path", {
  d: "M1 12s4-8 11-8 11 8 11 8-4 8-11 8-11-8-11-8z",
  key: "1t2h7b"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "3",
  key: "n6u7h6"
}]]]);

var Facebook = createReactComponent("Facebook", ["svg", defaultAttributes, [["path", {
  d: "M18 2h-3a5 5 0 00-5 5v3H7v4h3v8h4v-8h3l1-4h-4V7a1 1 0 011-1h3z",
  key: "c3wm0s"
}]]]);

var FastForward = createReactComponent("FastForward", ["svg", defaultAttributes, [["polygon", {
  points: "13 19 22 12 13 5 13 19",
  key: "6wx0cv"
}], ["polygon", {
  points: "2 19 11 12 2 5 2 19",
  key: "1qmptp"
}]]]);

var Feather = createReactComponent("Feather", ["svg", defaultAttributes, [["path", {
  d: "M20.24 12.24a6 6 0 00-8.49-8.49L5 10.5V19h8.5z",
  key: "4k6fqo"
}], ["line", {
  x1: "16",
  y1: "8",
  x2: "2",
  y2: "22",
  key: "zg5inx"
}], ["line", {
  x1: "17.5",
  y1: "15",
  x2: "9",
  y2: "15",
  key: "v1iq8k"
}]]]);

var Figma = createReactComponent("Figma", ["svg", defaultAttributes, [["path", {
  d: "M5 5.5A3.5 3.5 0 018.5 2H12v7H8.5A3.5 3.5 0 015 5.5z",
  key: "1gfupn"
}], ["path", {
  d: "M12 2h3.5a3.5 3.5 0 110 7H12V2z",
  key: "wsu2e6"
}], ["path", {
  d: "M12 12.5a3.5 3.5 0 117 0 3.5 3.5 0 11-7 0z",
  key: "vlgc3l"
}], ["path", {
  d: "M5 19.5A3.5 3.5 0 018.5 16H12v3.5a3.5 3.5 0 11-7 0z",
  key: "yrm84q"
}], ["path", {
  d: "M5 12.5A3.5 3.5 0 018.5 9H12v7H8.5A3.5 3.5 0 015 12.5z",
  key: "dc8psd"
}]]]);

var FileCheck2 = createReactComponent("FileCheck2", ["svg", defaultAttributes, [["path", {
  d: "M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v4",
  key: "19r7vg"
}], ["path", {
  d: "M14 2v6h6",
  key: "1yykud"
}], ["path", {
  d: "M3 15l2 2 4-4",
  key: "10w847"
}]]]);

var FileCheck = createReactComponent("FileCheck", ["svg", defaultAttributes, [["path", {
  d: "M14.5 2H6a2 2 0 00-2 2v16a2 2 0 002 2h12a2 2 0 002-2V7.5L14.5 2z",
  key: "1gzwd3"
}], ["polyline", {
  points: "14 2 14 8 20 8",
  key: "ag3yzn"
}], ["path", {
  d: "M9 15l2 2 4-4",
  key: "1fwnb4"
}]]]);

var FileCode = createReactComponent("FileCode", ["svg", defaultAttributes, [["path", {
  d: "M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v4",
  key: "19r7vg"
}], ["path", {
  d: "M14 2v6h6",
  key: "1yykud"
}], ["path", {
  d: "M9 18l3-3-3-3",
  key: "446dnh"
}], ["path", {
  d: "M5 12l-3 3 3 3",
  key: "tm2zkr"
}]]]);

var FileDigit = createReactComponent("FileDigit", ["svg", defaultAttributes, [["path", {
  d: "M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v4",
  key: "19r7vg"
}], ["path", {
  d: "M14 2v6h6",
  key: "1yykud"
}], ["path", {
  d: "M10 12h2v6",
  key: "hsb8tw"
}], ["rect", {
  x: "2",
  y: "12",
  width: "4",
  height: "6",
  key: "ma9s1h"
}], ["path", {
  d: "M10 18h4",
  key: "139y51"
}]]]);

var FileMinus2 = createReactComponent("FileMinus2", ["svg", defaultAttributes, [["path", {
  d: "M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v4",
  key: "19r7vg"
}], ["path", {
  d: "M14 2v6h6",
  key: "1yykud"
}], ["path", {
  d: "M3 15h6",
  key: "s0iqat"
}]]]);

var FileMinus = createReactComponent("FileMinus", ["svg", defaultAttributes, [["path", {
  d: "M14.5 2H6a2 2 0 00-2 2v16a2 2 0 002 2h12a2 2 0 002-2V7.5L14.5 2z",
  key: "1gzwd3"
}], ["polyline", {
  points: "14 2 14 8 20 8",
  key: "ag3yzn"
}], ["line", {
  x1: "9",
  y1: "15",
  x2: "15",
  y2: "15",
  key: "15lfvp"
}]]]);

var FilePlus2 = createReactComponent("FilePlus2", ["svg", defaultAttributes, [["path", {
  d: "M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v4",
  key: "19r7vg"
}], ["path", {
  d: "M14 2v6h6",
  key: "1yykud"
}], ["path", {
  d: "M3 15h6",
  key: "s0iqat"
}], ["path", {
  d: "M6 12v6",
  key: "1hougi"
}]]]);

var FilePlus = createReactComponent("FilePlus", ["svg", defaultAttributes, [["path", {
  d: "M14.5 2H6a2 2 0 00-2 2v16a2 2 0 002 2h12a2 2 0 002-2V7.5L14.5 2z",
  key: "1gzwd3"
}], ["polyline", {
  points: "14 2 14 8 20 8",
  key: "ag3yzn"
}], ["line", {
  x1: "12",
  y1: "18",
  x2: "12",
  y2: "12",
  key: "1jc1ax"
}], ["line", {
  x1: "9",
  y1: "15",
  x2: "15",
  y2: "15",
  key: "15lfvp"
}]]]);

var FileSearch = createReactComponent("FileSearch", ["svg", defaultAttributes, [["path", {
  d: "M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v3",
  key: "o9xtjo"
}], ["path", {
  d: "M14 2v6h6",
  key: "1yykud"
}], ["path", {
  d: "M5 17a3 3 0 100-6 3 3 0 000 6z",
  key: "1uj6y4"
}], ["path", {
  d: "M9 18l-1.5-1.5",
  key: "114x5l"
}]]]);

var FileText = createReactComponent("FileText", ["svg", defaultAttributes, [["path", {
  d: "M14.5 2H6a2 2 0 00-2 2v16a2 2 0 002 2h12a2 2 0 002-2V7.5L14.5 2z",
  key: "1gzwd3"
}], ["polyline", {
  points: "14 2 14 8 20 8",
  key: "ag3yzn"
}], ["line", {
  x1: "16",
  y1: "13",
  x2: "8",
  y2: "13",
  key: "1ggvzc"
}], ["line", {
  x1: "16",
  y1: "17",
  x2: "8",
  y2: "17",
  key: "3t9q27"
}], ["line", {
  x1: "10",
  y1: "9",
  x2: "8",
  y2: "9",
  key: "clsc21"
}]]]);

var FileX2 = createReactComponent("FileX2", ["svg", defaultAttributes, [["path", {
  d: "M4 22h14a2 2 0 002-2V7.5L14.5 2H6a2 2 0 00-2 2v4",
  key: "19r7vg"
}], ["path", {
  d: "M14 2v6h6",
  key: "1yykud"
}], ["path", {
  d: "M3 12.5l5 5",
  key: "1ctyc8"
}], ["path", {
  d: "M8 12.5l-5 5",
  key: "12t4sq"
}]]]);

var FileX = createReactComponent("FileX", ["svg", defaultAttributes, [["path", {
  d: "M14.5 2H6a2 2 0 00-2 2v16a2 2 0 002 2h12a2 2 0 002-2V7.5L14.5 2z",
  key: "1gzwd3"
}], ["polyline", {
  points: "14 2 14 8 20 8",
  key: "ag3yzn"
}], ["line", {
  x1: "9.5",
  y1: "12.5",
  x2: "14.5",
  y2: "17.5",
  key: "qmand5"
}], ["line", {
  x1: "14.5",
  y1: "12.5",
  x2: "9.5",
  y2: "17.5",
  key: "1s384y"
}]]]);

var File = createReactComponent("File", ["svg", defaultAttributes, [["path", {
  d: "M14.5 2H6a2 2 0 00-2 2v16a2 2 0 002 2h12a2 2 0 002-2V7.5L14.5 2z",
  key: "1gzwd3"
}], ["polyline", {
  points: "14 2 14 8 20 8",
  key: "ag3yzn"
}]]]);

var Files = createReactComponent("Files", ["svg", defaultAttributes, [["path", {
  d: "M15.5 2H8.6c-.4 0-.8.2-1.1.5-.3.3-.5.7-.5 1.1v12.8c0 .4.2.8.5 1.1.3.3.7.5 1.1.5h9.8c.4 0 .8-.2 1.1-.5.3-.3.5-.7.5-1.1V6.5L15.5 2z",
  key: "1r3rih"
}], ["path", {
  d: "M3 7.6v12.8c0 .4.2.8.5 1.1.3.3.7.5 1.1.5h9.8",
  key: "1k9wwo"
}], ["path", {
  d: "M15 2v5h5",
  key: "2ov5c4"
}]]]);

var Film = createReactComponent("Film", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "2",
  width: "20",
  height: "20",
  rx: "2.18",
  ry: "2.18",
  key: "1salsp"
}], ["line", {
  x1: "7",
  y1: "2",
  x2: "7",
  y2: "22",
  key: "18n2oj"
}], ["line", {
  x1: "17",
  y1: "2",
  x2: "17",
  y2: "22",
  key: "1rnemm"
}], ["line", {
  x1: "2",
  y1: "12",
  x2: "22",
  y2: "12",
  key: "1vcgga"
}], ["line", {
  x1: "2",
  y1: "7",
  x2: "7",
  y2: "7",
  key: "d12ql1"
}], ["line", {
  x1: "2",
  y1: "17",
  x2: "7",
  y2: "17",
  key: "1yu0kk"
}], ["line", {
  x1: "17",
  y1: "17",
  x2: "22",
  y2: "17",
  key: "14cddp"
}], ["line", {
  x1: "17",
  y1: "7",
  x2: "22",
  y2: "7",
  key: "m1e22e"
}]]]);

var Filter = createReactComponent("Filter", ["svg", defaultAttributes, [["polygon", {
  points: "22 3 2 3 10 12.46 10 19 14 21 14 12.46 22 3",
  key: "120hn0"
}]]]);

var Flag = createReactComponent("Flag", ["svg", defaultAttributes, [["path", {
  d: "M4 15s1-1 4-1 5 2 8 2 4-1 4-1V3s-1 1-4 1-5-2-8-2-4 1-4 1z",
  key: "wkvj37"
}], ["line", {
  x1: "4",
  y1: "22",
  x2: "4",
  y2: "15",
  key: "c1x1is"
}]]]);

var Flame = createReactComponent("Flame", ["svg", defaultAttributes, [["path", {
  d: "M8.5 14.5A2.5 2.5 0 0011 12c0-1.38-.5-2-1-3-1.072-2.143-.224-4.054 2-6 .5 2.5 2 4.9 4 6.5 2 1.6 3 3.5 3 5.5a7 7 0 11-14 0c0-1.153.433-2.294 1-3a2.5 2.5 0 002.5 2.5z",
  key: "18k39s"
}]]]);

var FlashlightOff = createReactComponent("FlashlightOff", ["svg", defaultAttributes, [["path", {
  d: "M16 16v4a2 2 0 01-2 2h-4a2 2 0 01-2-2V10c0-2-2-2-2-4",
  key: "fa6m7j"
}], ["path", {
  d: "M7 2h11v4c0 2-2 2-2 4v1",
  key: "nri0ub"
}], ["line", {
  x1: "11",
  y1: "6",
  x2: "18",
  y2: "6",
  key: "1bu00n"
}], ["line", {
  x1: "2",
  y1: "2",
  x2: "22",
  y2: "22",
  key: "1ehfaz"
}]]]);

var Flashlight = createReactComponent("Flashlight", ["svg", defaultAttributes, [["path", {
  d: "M18 6c0 2-2 2-2 4v10a2 2 0 01-2 2h-4a2 2 0 01-2-2V10c0-2-2-2-2-4V2h12z",
  key: "tcmgpj"
}], ["line", {
  x1: "6",
  y1: "6",
  x2: "18",
  y2: "6",
  key: "1fa9qb"
}], ["line", {
  x1: "12",
  y1: "12",
  x2: "12",
  y2: "12",
  key: "1uoiww"
}]]]);

var FlaskConical = createReactComponent("FlaskConical", ["svg", defaultAttributes, [["path", {
  d: "M10 2v8L4.72 20.55a1 1 0 00.9 1.45h12.76a1 1 0 00.9-1.45L14 10V2",
  key: "1dlqr6"
}], ["path", {
  d: "M8.5 2h7",
  key: "j9cflu"
}], ["path", {
  d: "M7 16h10",
  key: "1lyvjv"
}]]]);

var FlaskRound = createReactComponent("FlaskRound", ["svg", defaultAttributes, [["path", {
  d: "M10 2v7.31",
  key: "zu6ln6"
}], ["path", {
  d: "M14 9.3V1.99",
  key: "unm02s"
}], ["path", {
  d: "M8.5 2h7",
  key: "j9cflu"
}], ["path", {
  d: "M14 9.3a6.5 6.5 0 11-4 0",
  key: "1g5fut"
}], ["path", {
  d: "M5.58 16.5h12.85",
  key: "1aaksh"
}]]]);

var FolderMinus = createReactComponent("FolderMinus", ["svg", defaultAttributes, [["path", {
  d: "M22 19a2 2 0 01-2 2H4a2 2 0 01-2-2V5a2 2 0 012-2h5l2 3h9a2 2 0 012 2z",
  key: "1bzg3j"
}], ["line", {
  x1: "9",
  y1: "14",
  x2: "15",
  y2: "14",
  key: "1v9vyv"
}]]]);

var FolderPlus = createReactComponent("FolderPlus", ["svg", defaultAttributes, [["path", {
  d: "M22 19a2 2 0 01-2 2H4a2 2 0 01-2-2V5a2 2 0 012-2h5l2 3h9a2 2 0 012 2z",
  key: "1bzg3j"
}], ["line", {
  x1: "12",
  y1: "11",
  x2: "12",
  y2: "17",
  key: "umnme"
}], ["line", {
  x1: "9",
  y1: "14",
  x2: "15",
  y2: "14",
  key: "1v9vyv"
}]]]);

var Folder = createReactComponent("Folder", ["svg", defaultAttributes, [["path", {
  d: "M22 19a2 2 0 01-2 2H4a2 2 0 01-2-2V5a2 2 0 012-2h5l2 3h9a2 2 0 012 2z",
  key: "1bzg3j"
}]]]);

var Framer = createReactComponent("Framer", ["svg", defaultAttributes, [["path", {
  d: "M5 16V9h14V2H5l14 14h-7m-7 0l7 7v-7m-7 0h7",
  key: "urwt6s"
}]]]);

var Frown = createReactComponent("Frown", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["path", {
  d: "M16 16s-1.5-2-4-2-4 2-4 2",
  key: "1kaepz"
}], ["line", {
  x1: "9",
  y1: "9",
  x2: "9.01",
  y2: "9",
  key: "sl5tun"
}], ["line", {
  x1: "15",
  y1: "9",
  x2: "15.01",
  y2: "9",
  key: "136pav"
}]]]);

var Gamepad2 = createReactComponent("Gamepad2", ["svg", defaultAttributes, [["line", {
  x1: "6",
  y1: "11",
  x2: "10",
  y2: "11",
  key: "38l5fr"
}], ["line", {
  x1: "8",
  y1: "9",
  x2: "8",
  y2: "13",
  key: "1o7vfq"
}], ["line", {
  x1: "15",
  y1: "12",
  x2: "15.01",
  y2: "12",
  key: "ior5sv"
}], ["line", {
  x1: "18",
  y1: "10",
  x2: "18.01",
  y2: "10",
  key: "h5vdlr"
}], ["path", {
  d: "M17.32 5H6.68a4 4 0 00-3.978 3.59c-.006.052-.01.101-.017.152C2.604 9.416 2 14.456 2 16a3 3 0 003 3c1 0 1.5-.5 2-1l1.414-1.414A2 2 0 019.828 16h4.344a2 2 0 011.414.586L17 18c.5.5 1 1 2 1a3 3 0 003-3c0-1.545-.604-6.584-.685-7.258-.007-.05-.011-.1-.017-.151A4 4 0 0017.32 5z",
  key: "q8oq0v"
}]]]);

var Gamepad = createReactComponent("Gamepad", ["svg", defaultAttributes, [["line", {
  x1: "6",
  y1: "12",
  x2: "10",
  y2: "12",
  key: "p15dmv"
}], ["line", {
  x1: "8",
  y1: "10",
  x2: "8",
  y2: "14",
  key: "12n05p"
}], ["line", {
  x1: "15",
  y1: "13",
  x2: "15.01",
  y2: "13",
  key: "ar4m67"
}], ["line", {
  x1: "18",
  y1: "11",
  x2: "18.01",
  y2: "11",
  key: "18afha"
}], ["rect", {
  x: "2",
  y: "6",
  width: "20",
  height: "12",
  rx: "2",
  key: "ubz4ml"
}]]]);

var Gauge = createReactComponent("Gauge", ["svg", defaultAttributes, [["path", {
  d: "M12 15l3.5-3.5",
  key: "1404hq"
}], ["path", {
  d: "M20.3 18c.4-1 .7-2.2.7-3.4C21 9.8 17 6 12 6s-9 3.8-9 8.6c0 1.2.3 2.4.7 3.4",
  key: "rohdz3"
}]]]);

var Gavel = createReactComponent("Gavel", ["svg", defaultAttributes, [["path", {
  d: "M14 13l-7.5 7.5c-.83.83-2.17.83-3 0 0 0 0 0 0 0a2.12 2.12 0 010-3L11 10",
  key: "1153wj"
}], ["path", {
  d: "M16 16l6-6",
  key: "tnu50t"
}], ["path", {
  d: "M8 8l6-6",
  key: "cq27m5"
}], ["path", {
  d: "M9 7l8 8",
  key: "nwi5se"
}], ["path", {
  d: "M21 11l-8-8",
  key: "y6coz"
}]]]);

var Gift = createReactComponent("Gift", ["svg", defaultAttributes, [["polyline", {
  points: "20 12 20 22 4 22 4 12",
  key: "1rylk6"
}], ["rect", {
  x: "2",
  y: "7",
  width: "20",
  height: "5",
  key: "zlnt2s"
}], ["line", {
  x1: "12",
  y1: "22",
  x2: "12",
  y2: "7",
  key: "1oebls"
}], ["path", {
  d: "M12 7H7.5a2.5 2.5 0 010-5C11 2 12 7 12 7z",
  key: "2us9pr"
}], ["path", {
  d: "M12 7h4.5a2.5 2.5 0 000-5C13 2 12 7 12 7z",
  key: "17syld"
}]]]);

var GitBranchPlus = createReactComponent("GitBranchPlus", ["svg", defaultAttributes, [["path", {
  d: "M6 3v12",
  key: "13fvxb"
}], ["path", {
  d: "M18 9a3 3 0 100-6 3 3 0 000 6z",
  key: "1bjcfs"
}], ["path", {
  d: "M6 21a3 3 0 100-6 3 3 0 000 6z",
  key: "1t59tx"
}], ["path", {
  d: "M15 6a9 9 0 00-9 9",
  key: "1yzcpc"
}], ["path", {
  d: "M18 15v6",
  key: "p8g77l"
}], ["path", {
  d: "M21 18h-6",
  key: "md2nyt"
}]]]);

var GitBranch = createReactComponent("GitBranch", ["svg", defaultAttributes, [["line", {
  x1: "6",
  y1: "3",
  x2: "6",
  y2: "15",
  key: "1k99od"
}], ["circle", {
  cx: "18",
  cy: "6",
  r: "3",
  key: "1r01jq"
}], ["circle", {
  cx: "6",
  cy: "18",
  r: "3",
  key: "l375l1"
}], ["path", {
  d: "M18 9a9 9 0 01-9 9",
  key: "1yrk5u"
}]]]);

var GitCommit = createReactComponent("GitCommit", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "3",
  key: "n6u7h6"
}], ["line", {
  x1: "3",
  y1: "12",
  x2: "9",
  y2: "12",
  key: "akdq56"
}], ["line", {
  x1: "15",
  y1: "12",
  x2: "21",
  y2: "12",
  key: "gr5rsn"
}]]]);

var GitMerge = createReactComponent("GitMerge", ["svg", defaultAttributes, [["circle", {
  cx: "18",
  cy: "18",
  r: "3",
  key: "1p0y38"
}], ["circle", {
  cx: "6",
  cy: "6",
  r: "3",
  key: "ey9bbu"
}], ["path", {
  d: "M6 21V9a9 9 0 009 9",
  key: "16n2ug"
}]]]);

var GitPullRequest = createReactComponent("GitPullRequest", ["svg", defaultAttributes, [["circle", {
  cx: "18",
  cy: "18",
  r: "3",
  key: "1p0y38"
}], ["circle", {
  cx: "6",
  cy: "6",
  r: "3",
  key: "ey9bbu"
}], ["path", {
  d: "M13 6h3a2 2 0 012 2v7",
  key: "1qlfnd"
}], ["line", {
  x1: "6",
  y1: "9",
  x2: "6",
  y2: "21",
  key: "1b6pko"
}]]]);

var Github = createReactComponent("Github", ["svg", defaultAttributes, [["path", {
  d: "M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 00-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0020 4.77 5.07 5.07 0 0019.91 1S18.73.65 16 2.48a13.38 13.38 0 00-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 005 4.77a5.44 5.44 0 00-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 009 18.13V22",
  key: "w2htu0"
}]]]);

var Gitlab = createReactComponent("Gitlab", ["svg", defaultAttributes, [["path", {
  d: "M22.65 14.39L12 22.13 1.35 14.39a.84.84 0 01-.3-.94l1.22-3.78 2.44-7.51A.42.42 0 014.82 2a.43.43 0 01.58 0 .42.42 0 01.11.18l2.44 7.49h8.1l2.44-7.51A.42.42 0 0118.6 2a.43.43 0 01.58 0 .42.42 0 01.11.18l2.44 7.51L23 13.45a.84.84 0 01-.35.94z",
  key: "1a3khb"
}]]]);

var Globe2 = createReactComponent("Globe2", ["svg", defaultAttributes, [["path", {
  d: "M15 21v-4a2 2 0 012-2h4",
  key: "1e2agi"
}], ["path", {
  d: "M7 4v2a3 3 0 003 2h0a2 2 0 012 2 2 2 0 004 0 2 2 0 012-2h3",
  key: "xwmqo"
}], ["path", {
  d: "M3 11h2a2 2 0 012 2v1a2 2 0 002 2 2 2 0 012 2v4",
  key: "1lp6iw"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}]]]);

var Globe = createReactComponent("Globe", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "2",
  y1: "12",
  x2: "22",
  y2: "12",
  key: "1vcgga"
}], ["path", {
  d: "M12 2a15.3 15.3 0 014 10 15.3 15.3 0 01-4 10 15.3 15.3 0 01-4-10 15.3 15.3 0 014-10z",
  key: "p5qyqu"
}]]]);

var Grab = createReactComponent("Grab", ["svg", defaultAttributes, [["path", {
  d: "M18 11.5V9a2 2 0 00-2-2v0a2 2 0 00-2 2v1.4",
  key: "1oq2lj"
}], ["path", {
  d: "M14 10V8a2 2 0 00-2-2v0a2 2 0 00-2 2v2",
  key: "1ywd9e"
}], ["path", {
  d: "M10 9.9V9a2 2 0 00-2-2v0a2 2 0 00-2 2v5",
  key: "xm2yqs"
}], ["path", {
  d: "M6 14v0a2 2 0 00-2-2v0a2 2 0 00-2 2v0",
  key: "sovcyc"
}], ["path", {
  d: "M18 11v0a2 2 0 114 0v3a8 8 0 01-8 8h-4a8 8 0 01-8-8 2 2 0 114 0",
  key: "16u0ve"
}]]]);

var Grid = createReactComponent("Grid", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["line", {
  x1: "3",
  y1: "9",
  x2: "21",
  y2: "9",
  key: "1chj20"
}], ["line", {
  x1: "3",
  y1: "15",
  x2: "21",
  y2: "15",
  key: "106rj4"
}], ["line", {
  x1: "9",
  y1: "3",
  x2: "9",
  y2: "21",
  key: "5od17k"
}], ["line", {
  x1: "15",
  y1: "3",
  x2: "15",
  y2: "21",
  key: "1j0i69"
}]]]);

var GripHorizontal = createReactComponent("GripHorizontal", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "9",
  r: "1",
  key: "9w5l36"
}], ["circle", {
  cx: "19",
  cy: "9",
  r: "1",
  key: "fp9e2x"
}], ["circle", {
  cx: "5",
  cy: "9",
  r: "1",
  key: "1ov2e7"
}], ["circle", {
  cx: "12",
  cy: "15",
  r: "1",
  key: "wskt3z"
}], ["circle", {
  cx: "19",
  cy: "15",
  r: "1",
  key: "u07zys"
}], ["circle", {
  cx: "5",
  cy: "15",
  r: "1",
  key: "15pq1v"
}]]]);

var GripVertical = createReactComponent("GripVertical", ["svg", defaultAttributes, [["circle", {
  cx: "9",
  cy: "12",
  r: "1",
  key: "150moy"
}], ["circle", {
  cx: "9",
  cy: "5",
  r: "1",
  key: "xcz4l0"
}], ["circle", {
  cx: "9",
  cy: "19",
  r: "1",
  key: "pyqsqx"
}], ["circle", {
  cx: "15",
  cy: "12",
  r: "1",
  key: "4recin"
}], ["circle", {
  cx: "15",
  cy: "5",
  r: "1",
  key: "viq9zd"
}], ["circle", {
  cx: "15",
  cy: "19",
  r: "1",
  key: "7hp0b8"
}]]]);

var Hammer = createReactComponent("Hammer", ["svg", defaultAttributes, [["path", {
  d: "M15 12l-8.5 8.5c-.83.83-2.17.83-3 0 0 0 0 0 0 0a2.12 2.12 0 010-3L12 9",
  key: "5d3s6o"
}], ["path", {
  d: "M17.64 15L22 10.64",
  key: "h99n43"
}], ["path", {
  d: "M20.91 11.7l-1.25-1.25c-.6-.6-.93-1.4-.93-2.25v-.86L16.01 4.6a5.56 5.56 0 00-3.94-1.64H9l.92.82A6.18 6.18 0 0112 8.4v1.56l2 2h2.47l2.26 1.91",
  key: "19j02v"
}]]]);

var Hand = createReactComponent("Hand", ["svg", defaultAttributes, [["path", {
  d: "M18 11V6a2 2 0 00-2-2v0a2 2 0 00-2 2v0",
  key: "1dh6yr"
}], ["path", {
  d: "M14 10V4a2 2 0 00-2-2v0a2 2 0 00-2 2v2",
  key: "1n6pg8"
}], ["path", {
  d: "M10 10.5V6a2 2 0 00-2-2v0a2 2 0 00-2 2v8",
  key: "1c3tix"
}], ["path", {
  d: "M18 8a2 2 0 114 0v6a8 8 0 01-8 8h-2c-2.8 0-4.5-.86-5.99-2.34l-3.6-3.6a2 2 0 012.83-2.82L7 15",
  key: "az17h4"
}]]]);

var HardDrive = createReactComponent("HardDrive", ["svg", defaultAttributes, [["line", {
  x1: "22",
  y1: "12",
  x2: "2",
  y2: "12",
  key: "1kub4m"
}], ["path", {
  d: "M5.45 5.11L2 12v6a2 2 0 002 2h16a2 2 0 002-2v-6l-3.45-6.89A2 2 0 0016.76 4H7.24a2 2 0 00-1.79 1.11z",
  key: "1vlmtz"
}], ["line", {
  x1: "6",
  y1: "16",
  x2: "6.01",
  y2: "16",
  key: "1qihrp"
}], ["line", {
  x1: "10",
  y1: "16",
  x2: "10.01",
  y2: "16",
  key: "1qqomo"
}]]]);

var HardHat = createReactComponent("HardHat", ["svg", defaultAttributes, [["path", {
  d: "M2 18a1 1 0 001 1h18a1 1 0 001-1v-2a1 1 0 00-1-1H3a1 1 0 00-1 1v2z",
  key: "15ptlr"
}], ["path", {
  d: "M10 10V5a1 1 0 011-1h2a1 1 0 011 1v5",
  key: "4cs8ez"
}], ["path", {
  d: "M4 15v-3a6 6 0 016-6h0",
  key: "19utn5"
}], ["path", {
  d: "M14 6h0a6 6 0 016 6v3",
  key: "15jmrv"
}]]]);

var Hash = createReactComponent("Hash", ["svg", defaultAttributes, [["line", {
  x1: "4",
  y1: "9",
  x2: "20",
  y2: "9",
  key: "v4ep6u"
}], ["line", {
  x1: "4",
  y1: "15",
  x2: "20",
  y2: "15",
  key: "1id021"
}], ["line", {
  x1: "10",
  y1: "3",
  x2: "8",
  y2: "21",
  key: "1q2ln8"
}], ["line", {
  x1: "16",
  y1: "3",
  x2: "14",
  y2: "21",
  key: "10y5cg"
}]]]);

var Headphones = createReactComponent("Headphones", ["svg", defaultAttributes, [["path", {
  d: "M3 18v-6a9 9 0 0118 0v6",
  key: "1x0wt5"
}], ["path", {
  d: "M21 19a2 2 0 01-2 2h-1a2 2 0 01-2-2v-3a2 2 0 012-2h3zM3 19a2 2 0 002 2h1a2 2 0 002-2v-3a2 2 0 00-2-2H3z",
  key: "17qwj8"
}]]]);

var Heart = createReactComponent("Heart", ["svg", defaultAttributes, [["path", {
  d: "M20.42 4.58a5.4 5.4 0 00-7.65 0l-.77.78-.77-.78a5.4 5.4 0 00-7.65 0C1.46 6.7 1.33 10.28 4 13l8 8 8-8c2.67-2.72 2.54-6.3.42-8.42z",
  key: "1rhzi3"
}]]]);

var HelpCircle = createReactComponent("HelpCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["path", {
  d: "M9.09 9a3 3 0 015.83 1c0 2-3 3-3 3",
  key: "2vwi0r"
}], ["line", {
  x1: "12",
  y1: "17",
  x2: "12.01",
  y2: "17",
  key: "1fwdlo"
}]]]);

var Hexagon = createReactComponent("Hexagon", ["svg", defaultAttributes, [["path", {
  d: "M21 16V8a2 2 0 00-1-1.73l-7-4a2 2 0 00-2 0l-7 4A2 2 0 003 8v8a2 2 0 001 1.73l7 4a2 2 0 002 0l7-4A2 2 0 0021 16z",
  key: "eor234"
}]]]);

var Highlighter = createReactComponent("Highlighter", ["svg", defaultAttributes, [["path", {
  d: "M9 11l-6 6v3h9l3-3",
  key: "juy2f5"
}], ["path", {
  d: "M22 12l-4.6 4.6a2 2 0 01-2.8 0l-5.2-5.2a2 2 0 010-2.8L14 4",
  key: "xhs0vz"
}]]]);

var History = createReactComponent("History", ["svg", defaultAttributes, [["path", {
  d: "M3 3v5h5",
  key: "wxw82a"
}], ["path", {
  d: "M3.05 13A9 9 0 106 5.3L3 8",
  key: "y0tb33"
}], ["path", {
  d: "M12 7v5l4 2",
  key: "7lmkcx"
}]]]);

var Home = createReactComponent("Home", ["svg", defaultAttributes, [["path", {
  d: "M3 9l9-7 9 7v11a2 2 0 01-2 2H5a2 2 0 01-2-2z",
  key: "s0qlbv"
}], ["polyline", {
  points: "9 22 9 12 15 12 15 22",
  key: "9mql8z"
}]]]);

var ImageMinus = createReactComponent("ImageMinus", ["svg", defaultAttributes, [["path", {
  d: "M21 11v8a2 2 0 01-2 2H5a2 2 0 01-2-2V5a2 2 0 012-2h8",
  key: "16dvto"
}], ["path", {
  d: "M8.5 10a1.5 1.5 0 100-3 1.5 1.5 0 000 3z",
  key: "btlqe1"
}], ["path", {
  d: "M21 15l-5-5L5 21",
  key: "h3p7gd"
}], ["path", {
  d: "M22 5h-6",
  key: "pn198q"
}]]]);

var ImageOff = createReactComponent("ImageOff", ["svg", defaultAttributes, [["path", {
  d: "M2 2l20 20",
  key: "131xdx"
}], ["path", {
  d: "M9 3h10a2 2 0 012 2v10M3.59 3.59A2 2 0 003 5v14c0 1.1.9 2 2 2h14a2 2 0 001.41-.59",
  key: "191sqh"
}], ["path", {
  d: "M9.56 9.56a1.5 1.5 0 01-2.12-2.12",
  key: "11hmqw"
}], ["path", {
  d: "M21 15l-5-5M5 21l8-8",
  key: "1xyrbb"
}]]]);

var ImagePlus = createReactComponent("ImagePlus", ["svg", defaultAttributes, [["path", {
  d: "M21 11v8a2 2 0 01-2 2H5a2 2 0 01-2-2V5a2 2 0 012-2h8",
  key: "16dvto"
}], ["path", {
  d: "M8.5 10a1.5 1.5 0 100-3 1.5 1.5 0 000 3z",
  key: "btlqe1"
}], ["path", {
  d: "M21 15l-5-5L5 21",
  key: "h3p7gd"
}], ["path", {
  d: "M19 2v6",
  key: "hzml92"
}], ["path", {
  d: "M22 5h-6",
  key: "pn198q"
}]]]);

var Image = createReactComponent("Image", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["circle", {
  cx: "8.5",
  cy: "8.5",
  r: "1.5",
  key: "118yen"
}], ["polyline", {
  points: "21 15 16 10 5 21",
  key: "u5mf50"
}]]]);

var Inbox = createReactComponent("Inbox", ["svg", defaultAttributes, [["polyline", {
  points: "22 12 16 12 14 15 10 15 8 12 2 12",
  key: "hfj1a2"
}], ["path", {
  d: "M5.45 5.11L2 12v6a2 2 0 002 2h16a2 2 0 002-2v-6l-3.45-6.89A2 2 0 0016.76 4H7.24a2 2 0 00-1.79 1.11z",
  key: "1vlmtz"
}]]]);

var Indent = createReactComponent("Indent", ["svg", defaultAttributes, [["polyline", {
  points: "3 8 7 12 3 16",
  key: "1y28ex"
}], ["line", {
  x1: "21",
  y1: "12",
  x2: "11",
  y2: "12",
  key: "1sr45m"
}], ["line", {
  x1: "21",
  y1: "6",
  x2: "11",
  y2: "6",
  key: "1w72kw"
}], ["line", {
  x1: "21",
  y1: "18",
  x2: "11",
  y2: "18",
  key: "108ogh"
}]]]);

var Infinity = createReactComponent("Infinity", ["svg", defaultAttributes, [["path", {
  d: "M18.178 8c5.096 0 5.096 8 0 8-5.095 0-7.133-8-12.739-8-4.585 0-4.585 8 0 8 5.606 0 7.644-8 12.74-8z",
  key: "zjqofg"
}]]]);

var Info = createReactComponent("Info", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "12",
  y1: "16",
  x2: "12",
  y2: "12",
  key: "1ts0ao"
}], ["line", {
  x1: "12",
  y1: "8",
  x2: "12.01",
  y2: "8",
  key: "1uipcb"
}]]]);

var Inspect = createReactComponent("Inspect", ["svg", defaultAttributes, [["path", {
  d: "M19 11V4a2 2 0 00-2-2H4a2 2 0 00-2 2v13a2 2 0 002 2h7",
  key: "1chnfv"
}], ["path", {
  d: "M12 12l4.166 10 1.48-4.355L22 16.166 12 12z",
  key: "2kdk4z"
}], ["path", {
  d: "M18 18l3 3",
  key: "o3skm8"
}]]]);

var Instagram = createReactComponent("Instagram", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "2",
  width: "20",
  height: "20",
  rx: "5",
  ry: "5",
  key: "1wj0nt"
}], ["path", {
  d: "M16 11.37A4 4 0 1112.63 8 4 4 0 0116 11.37z",
  key: "6dhq4m"
}], ["line", {
  x1: "17.5",
  y1: "6.5",
  x2: "17.51",
  y2: "6.5",
  key: "1x6ige"
}]]]);

var Italic = createReactComponent("Italic", ["svg", defaultAttributes, [["line", {
  x1: "19",
  y1: "4",
  x2: "10",
  y2: "4",
  key: "1ykz7a"
}], ["line", {
  x1: "14",
  y1: "20",
  x2: "5",
  y2: "20",
  key: "1i8hhh"
}], ["line", {
  x1: "15",
  y1: "4",
  x2: "9",
  y2: "20",
  key: "1fl2dm"
}]]]);

var Key = createReactComponent("Key", ["svg", defaultAttributes, [["path", {
  d: "M21 2l-2 2m-7.61 7.61a5.5 5.5 0 11-7.778 7.778 5.5 5.5 0 017.777-7.777zm0 0L15.5 7.5m0 0l3 3L22 7l-3-3m-3.5 3.5L19 4",
  key: "1kley0"
}]]]);

var Languages = createReactComponent("Languages", ["svg", defaultAttributes, [["path", {
  d: "M5 8l6 6",
  key: "5qrnal"
}], ["path", {
  d: "M4 14l6-6 2-3",
  key: "4glnps"
}], ["path", {
  d: "M2 5h12",
  key: "16zq9b"
}], ["path", {
  d: "M7 2h1",
  key: "1lgm6t"
}], ["path", {
  d: "M22 22l-5-10-5 10",
  key: "tipqjx"
}], ["path", {
  d: "M14 18h6",
  key: "1jewtv"
}]]]);

var Laptop2 = createReactComponent("Laptop2", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "4",
  width: "18",
  height: "12",
  rx: "2",
  ry: "2",
  key: "u7jfu2"
}], ["line", {
  x1: "2",
  y1: "20",
  x2: "22",
  y2: "20",
  key: "9xhvb6"
}]]]);

var Laptop = createReactComponent("Laptop", ["svg", defaultAttributes, [["path", {
  d: "M20 16V7a2 2 0 00-2-2H6a2 2 0 00-2 2v9m16 0H4m16 0l1.28 2.55a1 1 0 01-.9 1.45H3.62a1 1 0 01-.9-1.45L4 16",
  key: "twdetq"
}]]]);

var LassoSelect = createReactComponent("LassoSelect", ["svg", defaultAttributes, [["path", {
  d: "M7 22a5 5 0 01-2-4",
  key: "16rap9"
}], ["path", {
  d: "M7 16.93c.96.43 1.96.74 2.99.91",
  key: "1ljl7n"
}], ["path", {
  d: "M3.34 14A6.8 6.8 0 012 10c0-4.42 4.48-8 10-8s10 3.58 10 8a7.19 7.19 0 01-.33 2",
  key: "119roj"
}], ["path", {
  d: "M5 18a2 2 0 100-4 2 2 0 000 4z",
  key: "1rvba9"
}], ["path", {
  d: "M14.33 22h-.09a.35.35 0 01-.24-.32v-10a.34.34 0 01.33-.34c.08 0 .15.03.21.08l7.34 6a.33.33 0 01-.21.59h-4.49l-2.57 3.85a.35.35 0 01-.28.14v0z",
  key: "vwv8lq"
}]]]);

var Lasso = createReactComponent("Lasso", ["svg", defaultAttributes, [["path", {
  d: "M7 22a5 5 0 01-2-4",
  key: "16rap9"
}], ["path", {
  d: "M3.3 14A6.8 6.8 0 012 10c0-4.4 4.5-8 10-8s10 3.6 10 8-4.5 8-10 8a12 12 0 01-5-1",
  key: "9ulfy8"
}], ["path", {
  d: "M5 18a2 2 0 100-4 2 2 0 000 4z",
  key: "1rvba9"
}]]]);

var Layers = createReactComponent("Layers", ["svg", defaultAttributes, [["polygon", {
  points: "12 2 2 7 12 12 22 7 12 2",
  key: "gzi815"
}], ["polyline", {
  points: "2 17 12 22 22 17",
  key: "j6unwi"
}], ["polyline", {
  points: "2 12 12 17 22 12",
  key: "1akav3"
}]]]);

var LayoutDashboard = createReactComponent("LayoutDashboard", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "7",
  height: "9",
  key: "1l9xge"
}], ["rect", {
  x: "14",
  y: "3",
  width: "7",
  height: "5",
  key: "x7ld02"
}], ["rect", {
  x: "14",
  y: "12",
  width: "7",
  height: "9",
  key: "r7q49a"
}], ["rect", {
  x: "3",
  y: "16",
  width: "7",
  height: "5",
  key: "1rj4p1"
}]]]);

var LayoutGrid = createReactComponent("LayoutGrid", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "7",
  height: "7",
  key: "ikfx8m"
}], ["rect", {
  x: "14",
  y: "3",
  width: "7",
  height: "7",
  key: "1dv5ta"
}], ["rect", {
  x: "14",
  y: "14",
  width: "7",
  height: "7",
  key: "lpkkw6"
}], ["rect", {
  x: "3",
  y: "14",
  width: "7",
  height: "7",
  key: "aud0k0"
}]]]);

var LayoutList = createReactComponent("LayoutList", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "14",
  width: "7",
  height: "7",
  key: "aud0k0"
}], ["rect", {
  x: "3",
  y: "3",
  width: "7",
  height: "7",
  key: "ikfx8m"
}], ["line", {
  x1: "14",
  y1: "4",
  x2: "21",
  y2: "4",
  key: "1m2r3u"
}], ["line", {
  x1: "14",
  y1: "9",
  x2: "21",
  y2: "9",
  key: "1l7vvx"
}], ["line", {
  x1: "14",
  y1: "15",
  x2: "21",
  y2: "15",
  key: "l5vdpy"
}], ["line", {
  x1: "14",
  y1: "20",
  x2: "21",
  y2: "20",
  key: "orewdy"
}]]]);

var Layout = createReactComponent("Layout", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["line", {
  x1: "3",
  y1: "9",
  x2: "21",
  y2: "9",
  key: "1chj20"
}], ["line", {
  x1: "9",
  y1: "21",
  x2: "9",
  y2: "9",
  key: "1es4cq"
}]]]);

var Library = createReactComponent("Library", ["svg", defaultAttributes, [["path", {
  d: "M16 6l4 14",
  key: "1dclo4"
}], ["path", {
  d: "M12 6v14",
  key: "13vody"
}], ["path", {
  d: "M8 8v12",
  key: "flrwk9"
}], ["path", {
  d: "M4 4v16",
  key: "41fst9"
}]]]);

var LifeBuoy = createReactComponent("LifeBuoy", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "4",
  key: "7pwehp"
}], ["line", {
  x1: "4.93",
  y1: "4.93",
  x2: "9.17",
  y2: "9.17",
  key: "ur3s5s"
}], ["line", {
  x1: "14.83",
  y1: "14.83",
  x2: "19.07",
  y2: "19.07",
  key: "1rakqz"
}], ["line", {
  x1: "14.83",
  y1: "9.17",
  x2: "19.07",
  y2: "4.93",
  key: "d8cfsg"
}], ["line", {
  x1: "14.83",
  y1: "9.17",
  x2: "18.36",
  y2: "5.64",
  key: "p16tkq"
}], ["line", {
  x1: "4.93",
  y1: "19.07",
  x2: "9.17",
  y2: "14.83",
  key: "13at7l"
}]]]);

var Lightbulb = createReactComponent("Lightbulb", ["svg", defaultAttributes, [["line", {
  x1: "9",
  y1: "18",
  x2: "15",
  y2: "18",
  key: "1m27rk"
}], ["line", {
  x1: "10",
  y1: "22",
  x2: "14",
  y2: "22",
  key: "h3a5w4"
}], ["path", {
  d: "M15.09 14c.18-.98.65-1.74 1.41-2.5A4.65 4.65 0 0018 8 6 6 0 006 8c0 1 .23 2.23 1.5 3.5A4.61 4.61 0 018.91 14",
  key: "1g26sa"
}]]]);

var Link2Off = createReactComponent("Link2Off", ["svg", defaultAttributes, [["path", {
  d: "M15 7h2a5 5 0 014 8M9 17H7A5 5 0 017 7",
  key: "fm2in8"
}], ["line", {
  x1: "8",
  y1: "12",
  x2: "12",
  y2: "12",
  key: "17phrg"
}], ["line", {
  x1: "2",
  y1: "2",
  x2: "22",
  y2: "22",
  key: "1ehfaz"
}]]]);

var Link2 = createReactComponent("Link2", ["svg", defaultAttributes, [["path", {
  d: "M15 7h2a5 5 0 010 10h-2m-6 0H7A5 5 0 017 7h2",
  key: "2adp1c"
}], ["line", {
  x1: "8",
  y1: "12",
  x2: "16",
  y2: "12",
  key: "6hnc9b"
}]]]);

var Link = createReactComponent("Link", ["svg", defaultAttributes, [["path", {
  d: "M10 13a5 5 0 007.54.54l3-3a5 5 0 00-7.07-7.07l-1.72 1.71",
  key: "swjwyj"
}], ["path", {
  d: "M14 11a5 5 0 00-7.54-.54l-3 3a5 5 0 007.07 7.07l1.71-1.71",
  key: "1aphnl"
}]]]);

var Linkedin = createReactComponent("Linkedin", ["svg", defaultAttributes, [["path", {
  d: "M16 8a6 6 0 016 6v7h-4v-7a2 2 0 00-2-2 2 2 0 00-2 2v7h-4v-7a6 6 0 016-6z",
  key: "dxpxhk"
}], ["rect", {
  x: "2",
  y: "9",
  width: "4",
  height: "12",
  key: "zi0lh6"
}], ["circle", {
  cx: "4",
  cy: "4",
  r: "2",
  key: "nx0fkr"
}]]]);

var ListChecks = createReactComponent("ListChecks", ["svg", defaultAttributes, [["line", {
  x1: "10",
  y1: "6",
  x2: "21",
  y2: "6",
  key: "g3fxpu"
}], ["line", {
  x1: "10",
  y1: "12",
  x2: "21",
  y2: "12",
  key: "dmx5v6"
}], ["line", {
  x1: "10",
  y1: "18",
  x2: "21",
  y2: "18",
  key: "c5zasy"
}], ["polyline", {
  points: "3 6 4 7 6 5",
  key: "1eculw"
}], ["polyline", {
  points: "3 12 4 13 6 11",
  key: "1s1ef5"
}], ["polyline", {
  points: "3 18 4 19 6 17",
  key: "3aj6dh"
}]]]);

var ListOrdered = createReactComponent("ListOrdered", ["svg", defaultAttributes, [["line", {
  x1: "10",
  y1: "6",
  x2: "21",
  y2: "6",
  key: "g3fxpu"
}], ["line", {
  x1: "10",
  y1: "12",
  x2: "21",
  y2: "12",
  key: "dmx5v6"
}], ["line", {
  x1: "10",
  y1: "18",
  x2: "21",
  y2: "18",
  key: "c5zasy"
}], ["path", {
  d: "M4 6h1v4",
  key: "r283p1"
}], ["path", {
  d: "M4 10h2",
  key: "1ioyz2"
}], ["path", {
  d: "M6 18H4c0-1 2-2 2-3s-1-1.5-2-1",
  key: "plto16"
}]]]);

var List = createReactComponent("List", ["svg", defaultAttributes, [["line", {
  x1: "8",
  y1: "6",
  x2: "21",
  y2: "6",
  key: "xpnqsr"
}], ["line", {
  x1: "8",
  y1: "12",
  x2: "21",
  y2: "12",
  key: "7t48vf"
}], ["line", {
  x1: "8",
  y1: "18",
  x2: "21",
  y2: "18",
  key: "1y6o8k"
}], ["line", {
  x1: "3",
  y1: "6",
  x2: "3.01",
  y2: "6",
  key: "udquin"
}], ["line", {
  x1: "3",
  y1: "12",
  x2: "3.01",
  y2: "12",
  key: "1dcyyx"
}], ["line", {
  x1: "3",
  y1: "18",
  x2: "3.01",
  y2: "18",
  key: "16eb49"
}]]]);

var Loader = createReactComponent("Loader", ["svg", defaultAttributes, [["line", {
  x1: "12",
  y1: "2",
  x2: "12",
  y2: "6",
  key: "53aro4"
}], ["line", {
  x1: "12",
  y1: "18",
  x2: "12",
  y2: "22",
  key: "1slx9a"
}], ["line", {
  x1: "4.93",
  y1: "4.93",
  x2: "7.76",
  y2: "7.76",
  key: "91at3k"
}], ["line", {
  x1: "16.24",
  y1: "16.24",
  x2: "19.07",
  y2: "19.07",
  key: "1jjqbg"
}], ["line", {
  x1: "2",
  y1: "12",
  x2: "6",
  y2: "12",
  key: "1o7g50"
}], ["line", {
  x1: "18",
  y1: "12",
  x2: "22",
  y2: "12",
  key: "8ldd2x"
}], ["line", {
  x1: "4.93",
  y1: "19.07",
  x2: "7.76",
  y2: "16.24",
  key: "1k6nnb"
}], ["line", {
  x1: "16.24",
  y1: "7.76",
  x2: "19.07",
  y2: "4.93",
  key: "tii48m"
}]]]);

var LocateFixed = createReactComponent("LocateFixed", ["svg", defaultAttributes, [["line", {
  x1: "2",
  x2: "5",
  y1: "12",
  y2: "12",
  key: "sajh3r"
}], ["line", {
  x1: "19",
  x2: "22",
  y1: "12",
  y2: "12",
  key: "1kevsh"
}], ["line", {
  x1: "12",
  x2: "12",
  y1: "2",
  y2: "5",
  key: "gdek53"
}], ["line", {
  x1: "12",
  x2: "12",
  y1: "19",
  y2: "22",
  key: "on13co"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "7",
  key: "f8eh5q"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "3",
  key: "n6u7h6"
}]]]);

var Locate = createReactComponent("Locate", ["svg", defaultAttributes, [["line", {
  x1: "2",
  x2: "5",
  y1: "12",
  y2: "12",
  key: "sajh3r"
}], ["line", {
  x1: "19",
  x2: "22",
  y1: "12",
  y2: "12",
  key: "1kevsh"
}], ["line", {
  x1: "12",
  x2: "12",
  y1: "2",
  y2: "5",
  key: "gdek53"
}], ["line", {
  x1: "12",
  x2: "12",
  y1: "19",
  y2: "22",
  key: "on13co"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "7",
  key: "f8eh5q"
}]]]);

var Lock = createReactComponent("Lock", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "11",
  width: "18",
  height: "11",
  rx: "2",
  ry: "2",
  key: "1jfkdd"
}], ["path", {
  d: "M7 11V7a5 5 0 0110 0v4",
  key: "3zkwqh"
}]]]);

var LogIn = createReactComponent("LogIn", ["svg", defaultAttributes, [["path", {
  d: "M15 3h4a2 2 0 012 2v14a2 2 0 01-2 2h-4",
  key: "ji49vs"
}], ["polyline", {
  points: "10 17 15 12 10 7",
  key: "15i8xt"
}], ["line", {
  x1: "15",
  y1: "12",
  x2: "3",
  y2: "12",
  key: "wvjmef"
}]]]);

var LogOut = createReactComponent("LogOut", ["svg", defaultAttributes, [["path", {
  d: "M9 21H5a2 2 0 01-2-2V5a2 2 0 012-2h4",
  key: "1wlf0y"
}], ["polyline", {
  points: "16 17 21 12 16 7",
  key: "xjwp0i"
}], ["line", {
  x1: "21",
  y1: "12",
  x2: "9",
  y2: "12",
  key: "1g0nry"
}]]]);

var Mail = createReactComponent("Mail", ["svg", defaultAttributes, [["path", {
  d: "M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z",
  key: "1oqfcz"
}], ["polyline", {
  points: "22,6 12,13 2,6",
  key: "1y3j1y"
}]]]);

var MapPin = createReactComponent("MapPin", ["svg", defaultAttributes, [["path", {
  d: "M21 10c0 7-9 13-9 13s-9-6-9-13a9 9 0 0118 0z",
  key: "gy1kji"
}], ["circle", {
  cx: "12",
  cy: "10",
  r: "3",
  key: "1byug6"
}]]]);

var Map = createReactComponent("Map", ["svg", defaultAttributes, [["polygon", {
  points: "1 6 1 22 8 18 16 22 23 18 23 2 16 6 8 2 1 6",
  key: "yi6pq4"
}], ["line", {
  x1: "8",
  y1: "2",
  x2: "8",
  y2: "18",
  key: "ekzmkr"
}], ["line", {
  x1: "16",
  y1: "6",
  x2: "16",
  y2: "22",
  key: "1y28ex"
}]]]);

var Maximize2 = createReactComponent("Maximize2", ["svg", defaultAttributes, [["polyline", {
  points: "15 3 21 3 21 9",
  key: "8bv4a6"
}], ["polyline", {
  points: "9 21 3 21 3 15",
  key: "qq2vbi"
}], ["line", {
  x1: "21",
  y1: "3",
  x2: "14",
  y2: "10",
  key: "1s3k4x"
}], ["line", {
  x1: "3",
  y1: "21",
  x2: "10",
  y2: "14",
  key: "qskfes"
}]]]);

var Maximize = createReactComponent("Maximize", ["svg", defaultAttributes, [["path", {
  d: "M8 3H5a2 2 0 00-2 2v3m18 0V5a2 2 0 00-2-2h-3m0 18h3a2 2 0 002-2v-3M3 16v3a2 2 0 002 2h3",
  key: "1mfej3"
}]]]);

var Meh = createReactComponent("Meh", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "8",
  y1: "15",
  x2: "16",
  y2: "15",
  key: "1al99h"
}], ["line", {
  x1: "9",
  y1: "9",
  x2: "9.01",
  y2: "9",
  key: "sl5tun"
}], ["line", {
  x1: "15",
  y1: "9",
  x2: "15.01",
  y2: "9",
  key: "136pav"
}]]]);

var Menu = createReactComponent("Menu", ["svg", defaultAttributes, [["line", {
  x1: "4",
  y1: "12",
  x2: "20",
  y2: "12",
  key: "e1pn9i"
}], ["line", {
  x1: "4",
  y1: "6",
  x2: "20",
  y2: "6",
  key: "6zyfom"
}], ["line", {
  x1: "4",
  y1: "18",
  x2: "20",
  y2: "18",
  key: "5e5kzq"
}]]]);

var MessageCircle = createReactComponent("MessageCircle", ["svg", defaultAttributes, [["path", {
  d: "M21 11.5a8.38 8.38 0 01-.9 3.8 8.5 8.5 0 01-7.6 4.7 8.38 8.38 0 01-3.8-.9L3 21l1.9-5.7a8.38 8.38 0 01-.9-3.8 8.5 8.5 0 014.7-7.6 8.38 8.38 0 013.8-.9h.5a8.48 8.48 0 018 8v.5z",
  key: "d9lhbr"
}]]]);

var MessageSquare = createReactComponent("MessageSquare", ["svg", defaultAttributes, [["path", {
  d: "M21 15a2 2 0 01-2 2H7l-4 4V5a2 2 0 012-2h14a2 2 0 012 2z",
  key: "1tpnww"
}]]]);

var MicOff = createReactComponent("MicOff", ["svg", defaultAttributes, [["line", {
  x1: "1",
  y1: "1",
  x2: "23",
  y2: "23",
  key: "czowfk"
}], ["path", {
  d: "M9 9v3a3 3 0 005.12 2.12M15 9.34V4a3 3 0 00-5.94-.6",
  key: "1pfz6f"
}], ["path", {
  d: "M17 16.95A7 7 0 015 12v-2m14 0v2a7 7 0 01-.11 1.23",
  key: "ul691t"
}], ["line", {
  x1: "12",
  y1: "19",
  x2: "12",
  y2: "23",
  key: "2d2mwp"
}], ["line", {
  x1: "8",
  y1: "23",
  x2: "16",
  y2: "23",
  key: "52tlcf"
}]]]);

var Mic = createReactComponent("Mic", ["svg", defaultAttributes, [["path", {
  d: "M12 1a3 3 0 00-3 3v8a3 3 0 006 0V4a3 3 0 00-3-3z",
  key: "6jm2uq"
}], ["path", {
  d: "M19 10v2a7 7 0 01-14 0v-2",
  key: "1dkhyo"
}], ["line", {
  x1: "12",
  y1: "19",
  x2: "12",
  y2: "23",
  key: "2d2mwp"
}], ["line", {
  x1: "8",
  y1: "23",
  x2: "16",
  y2: "23",
  key: "52tlcf"
}]]]);

var Minimize2 = createReactComponent("Minimize2", ["svg", defaultAttributes, [["polyline", {
  points: "4 14 10 14 10 20",
  key: "18b7oj"
}], ["polyline", {
  points: "20 10 14 10 14 4",
  key: "jvrbid"
}], ["line", {
  x1: "14",
  y1: "10",
  x2: "21",
  y2: "3",
  key: "u5avic"
}], ["line", {
  x1: "3",
  y1: "21",
  x2: "10",
  y2: "14",
  key: "qskfes"
}]]]);

var Minimize = createReactComponent("Minimize", ["svg", defaultAttributes, [["path", {
  d: "M8 3v3a2 2 0 01-2 2H3m18 0h-3a2 2 0 01-2-2V3m0 18v-3a2 2 0 012-2h3M3 16h3a2 2 0 012 2v3",
  key: "1v9xpv"
}]]]);

var MinusCircle = createReactComponent("MinusCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "8",
  y1: "12",
  x2: "16",
  y2: "12",
  key: "6hnc9b"
}]]]);

var MinusSquare = createReactComponent("MinusSquare", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["line", {
  x1: "8",
  y1: "12",
  x2: "16",
  y2: "12",
  key: "6hnc9b"
}]]]);

var Minus = createReactComponent("Minus", ["svg", defaultAttributes, [["line", {
  x1: "5",
  y1: "12",
  x2: "19",
  y2: "12",
  key: "iqslzh"
}]]]);

var MonitorOff = createReactComponent("MonitorOff", ["svg", defaultAttributes, [["path", {
  d: "M17 17H4a2 2 0 01-2-2V5c0-1.5 1-2 1-2",
  key: "50l0vr"
}], ["path", {
  d: "M22 15V5a2 2 0 00-2-2H9",
  key: "b9w757"
}], ["path", {
  d: "M8 21h8",
  key: "62csdj"
}], ["path", {
  d: "M12 17v4",
  key: "icnn2j"
}], ["path", {
  d: "M2 2l20 20",
  key: "131xdx"
}]]]);

var Monitor = createReactComponent("Monitor", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "3",
  width: "20",
  height: "14",
  rx: "2",
  ry: "2",
  key: "1krask"
}], ["line", {
  x1: "8",
  y1: "21",
  x2: "16",
  y2: "21",
  key: "10fzyn"
}], ["line", {
  x1: "12",
  y1: "17",
  x2: "12",
  y2: "21",
  key: "78hxbp"
}]]]);

var Moon = createReactComponent("Moon", ["svg", defaultAttributes, [["path", {
  d: "M21 12.79A9 9 0 1111.21 3 7 7 0 0021 12.79z",
  key: "1qo1pg"
}]]]);

var MoreHorizontal = createReactComponent("MoreHorizontal", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "1",
  key: "1cly3i"
}], ["circle", {
  cx: "19",
  cy: "12",
  r: "1",
  key: "14bauv"
}], ["circle", {
  cx: "5",
  cy: "12",
  r: "1",
  key: "11uqdf"
}]]]);

var MoreVertical = createReactComponent("MoreVertical", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "1",
  key: "1cly3i"
}], ["circle", {
  cx: "12",
  cy: "5",
  r: "1",
  key: "1p6u0l"
}], ["circle", {
  cx: "12",
  cy: "19",
  r: "1",
  key: "19tybt"
}]]]);

var MousePointer2 = createReactComponent("MousePointer2", ["svg", defaultAttributes, [["path", {
  d: "M4 4l7.07 17 2.51-7.39L21 11.07z",
  key: "1rpga0"
}]]]);

var MousePointer = createReactComponent("MousePointer", ["svg", defaultAttributes, [["path", {
  d: "M3 3l7.07 16.97 2.51-7.39 7.39-2.51L3 3z",
  key: "13etbm"
}], ["path", {
  d: "M13 13l6 6",
  key: "fs9n40"
}]]]);

var MoveDiagonal2 = createReactComponent("MoveDiagonal2", ["svg", defaultAttributes, [["polyline", {
  points: "5 11 5 5 11 5",
  key: "ibd62b"
}], ["polyline", {
  points: "19 13 19 19 13 19",
  key: "17z3m7"
}], ["line", {
  x1: "5",
  y1: "5",
  x2: "19",
  y2: "19",
  key: "12h6nj"
}]]]);

var MoveDiagonal = createReactComponent("MoveDiagonal", ["svg", defaultAttributes, [["polyline", {
  points: "13 5 19 5 19 11",
  key: "1v5ik2"
}], ["polyline", {
  points: "11 19 5 19 5 13",
  key: "1evf3t"
}], ["line", {
  x1: "19",
  y1: "5",
  x2: "5",
  y2: "19",
  key: "18lstb"
}]]]);

var MoveHorizontal = createReactComponent("MoveHorizontal", ["svg", defaultAttributes, [["polyline", {
  points: "18 8 22 12 18 16",
  key: "1eylrn"
}], ["polyline", {
  points: "6 8 2 12 6 16",
  key: "1dpeq5"
}], ["line", {
  x1: "2",
  y1: "12",
  x2: "22",
  y2: "12",
  key: "1vcgga"
}]]]);

var MoveVertical = createReactComponent("MoveVertical", ["svg", defaultAttributes, [["polyline", {
  points: "8 18 12 22 16 18",
  key: "1s5whx"
}], ["polyline", {
  points: "8 6 12 2 16 6",
  key: "rp6fz1"
}], ["line", {
  x1: "12",
  y1: "2",
  x2: "12",
  y2: "22",
  key: "1hqhh2"
}]]]);

var Move = createReactComponent("Move", ["svg", defaultAttributes, [["polyline", {
  points: "5 9 2 12 5 15",
  key: "1v65fw"
}], ["polyline", {
  points: "9 5 12 2 15 5",
  key: "1aef3b"
}], ["polyline", {
  points: "15 19 12 22 9 19",
  key: "1e6jgb"
}], ["polyline", {
  points: "19 9 22 12 19 15",
  key: "11lfqd"
}], ["line", {
  x1: "2",
  y1: "12",
  x2: "22",
  y2: "12",
  key: "1vcgga"
}], ["line", {
  x1: "12",
  y1: "2",
  x2: "12",
  y2: "22",
  key: "1hqhh2"
}]]]);

var Music = createReactComponent("Music", ["svg", defaultAttributes, [["path", {
  d: "M9 18V5l12-2v13",
  key: "1m5kjg"
}], ["circle", {
  cx: "6",
  cy: "18",
  r: "3",
  key: "l375l1"
}], ["circle", {
  cx: "18",
  cy: "16",
  r: "3",
  key: "4j247o"
}]]]);

var Navigation2 = createReactComponent("Navigation2", ["svg", defaultAttributes, [["polygon", {
  points: "12 2 19 21 12 17 5 21 12 2",
  key: "1ofcjk"
}]]]);

var Navigation = createReactComponent("Navigation", ["svg", defaultAttributes, [["polygon", {
  points: "3 11 22 2 13 21 11 13 3 11",
  key: "v6bc7e"
}]]]);

var Network = createReactComponent("Network", ["svg", defaultAttributes, [["rect", {
  x: "9",
  y: "2",
  width: "6",
  height: "6",
  key: "gjave5"
}], ["rect", {
  x: "16",
  y: "16",
  width: "6",
  height: "6",
  key: "1hz3r1"
}], ["rect", {
  x: "2",
  y: "16",
  width: "6",
  height: "6",
  key: "pc2xv7"
}], ["path", {
  d: "M12 8v4m0 0H5v4m7-4h7v4",
  key: "14mn3q"
}]]]);

var Octagon = createReactComponent("Octagon", ["svg", defaultAttributes, [["polygon", {
  points: "7.86 2 16.14 2 22 7.86 22 16.14 16.14 22 7.86 22 2 16.14 2 7.86 7.86 2",
  key: "1qo9ur"
}]]]);

var Option = createReactComponent("Option", ["svg", defaultAttributes, [["path", {
  d: "M3 3h6l6 18h6",
  key: "1gmagw"
}], ["path", {
  d: "M14 3h7",
  key: "1n0z0g"
}]]]);

var Outdent = createReactComponent("Outdent", ["svg", defaultAttributes, [["polyline", {
  points: "7 8 3 12 7 16",
  key: "19cpua"
}], ["line", {
  x1: "21",
  y1: "12",
  x2: "11",
  y2: "12",
  key: "1sr45m"
}], ["line", {
  x1: "21",
  y1: "6",
  x2: "11",
  y2: "6",
  key: "1w72kw"
}], ["line", {
  x1: "21",
  y1: "18",
  x2: "11",
  y2: "18",
  key: "108ogh"
}]]]);

var Package = createReactComponent("Package", ["svg", defaultAttributes, [["line", {
  x1: "16.5",
  y1: "9.4",
  x2: "7.5",
  y2: "4.21",
  key: "1j3duo"
}], ["path", {
  d: "M21 16V8a2 2 0 00-1-1.73l-7-4a2 2 0 00-2 0l-7 4A2 2 0 003 8v8a2 2 0 001 1.73l7 4a2 2 0 002 0l7-4A2 2 0 0021 16z",
  key: "eor234"
}], ["polyline", {
  points: "3.27 6.96 12 12.01 20.73 6.96",
  key: "5vdlqk"
}], ["line", {
  x1: "12",
  y1: "22.08",
  x2: "12",
  y2: "12",
  key: "n2060l"
}]]]);

var Palette = createReactComponent("Palette", ["svg", defaultAttributes, [["circle", {
  cx: "13.5",
  cy: "6.5",
  r: ".5",
  key: "x804jq"
}], ["circle", {
  cx: "17.5",
  cy: "10.5",
  r: ".5",
  key: "ksltlh"
}], ["circle", {
  cx: "8.5",
  cy: "7.5",
  r: ".5",
  key: "13jnl4"
}], ["circle", {
  cx: "6.5",
  cy: "12.5",
  r: ".5",
  key: "1il44g"
}], ["path", {
  d: "M12 2C6.5 2 2 6.5 2 12s4.5 10 10 10c.926 0 1.648-.746 1.648-1.688 0-.437-.18-.835-.437-1.125-.29-.289-.438-.652-.438-1.125a1.64 1.64 0 011.668-1.668h1.996c3.051 0 5.555-2.503 5.555-5.554C21.965 6.012 17.461 2 12 2z",
  key: "mpc7ar"
}]]]);

var Paperclip = createReactComponent("Paperclip", ["svg", defaultAttributes, [["path", {
  d: "M21.44 11.05l-9.19 9.19a6 6 0 01-8.49-8.49l9.19-9.19a4 4 0 015.66 5.66l-9.2 9.19a2 2 0 01-2.83-2.83l8.49-8.48",
  key: "gebfs1"
}]]]);

var PauseCircle = createReactComponent("PauseCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "10",
  y1: "15",
  x2: "10",
  y2: "9",
  key: "fqtey5"
}], ["line", {
  x1: "14",
  y1: "15",
  x2: "14",
  y2: "9",
  key: "1o0kt7"
}]]]);

var Pause = createReactComponent("Pause", ["svg", defaultAttributes, [["rect", {
  x: "6",
  y: "4",
  width: "4",
  height: "16",
  key: "18bzn1"
}], ["rect", {
  x: "14",
  y: "4",
  width: "4",
  height: "16",
  key: "xo49r8"
}]]]);

var PenTool = createReactComponent("PenTool", ["svg", defaultAttributes, [["path", {
  d: "M12 19l7-7 3 3-7 7-3-3z",
  key: "1as30j"
}], ["path", {
  d: "M18 13l-1.5-7.5L2 2l3.5 14.5L13 18l5-5z",
  key: "yc5qzr"
}], ["path", {
  d: "M2 2l7.586 7.586",
  key: "106w32"
}], ["circle", {
  cx: "11",
  cy: "11",
  r: "2",
  key: "tihcyj"
}]]]);

var Pencil = createReactComponent("Pencil", ["svg", defaultAttributes, [["line", {
  x1: "18",
  y1: "2",
  x2: "22",
  y2: "6",
  key: "x6disd"
}], ["path", {
  d: "M7.5 20.5L19 9l-4-4L3.5 16.5 2 22l5.5-1.5z",
  key: "nsyti5"
}]]]);

var Percent = createReactComponent("Percent", ["svg", defaultAttributes, [["line", {
  x1: "19",
  y1: "5",
  x2: "5",
  y2: "19",
  key: "18lstb"
}], ["circle", {
  cx: "6.5",
  cy: "6.5",
  r: "2.5",
  key: "1r6og4"
}], ["circle", {
  cx: "17.5",
  cy: "17.5",
  r: "2.5",
  key: "1jfk5q"
}]]]);

var PhoneCall = createReactComponent("PhoneCall", ["svg", defaultAttributes, [["path", {
  d: "M14.05 6A5 5 0 0118 10m-3.95-8a9 9 0 018 7.94m0 7v3a2 2 0 01-2 2h-.19a19.79 19.79 0 01-8.63-3.07 19.52 19.52 0 01-6-6 19.82 19.82 0 01-3.11-8.69A2 2 0 013.93 2h3.18a2 2 0 012 1.72 13 13 0 00.7 2.81 2 2 0 01-.45 2.11L8.09 9.91a16 16 0 006 6l1.27-1.27a2 2 0 012.11-.45 13 13 0 002.81.7A2 2 0 0122 16.92z",
  key: "ldscnw"
}]]]);

var PhoneForwarded = createReactComponent("PhoneForwarded", ["svg", defaultAttributes, [["polyline", {
  points: "18 2 22 6 18 10",
  key: "zrhx86"
}], ["line", {
  x1: "14",
  y1: "6",
  x2: "22",
  y2: "6",
  key: "vs4hjp"
}], ["path", {
  d: "M22 16.92v3a2 2 0 01-2.18 2 19.79 19.79 0 01-8.63-3.07 19.5 19.5 0 01-6-6 19.79 19.79 0 01-3.07-8.67A2 2 0 014.11 2h3a2 2 0 012 1.72 12.84 12.84 0 00.7 2.81 2 2 0 01-.45 2.11L8.09 9.91a16 16 0 006 6l1.27-1.27a2 2 0 012.11-.45 12.84 12.84 0 002.81.7A2 2 0 0122 16.92z",
  key: "zm4dm"
}]]]);

var PhoneIncoming = createReactComponent("PhoneIncoming", ["svg", defaultAttributes, [["polyline", {
  points: "16 2 16 8 22 8",
  key: "1eqnog"
}], ["line", {
  x1: "22",
  y1: "2",
  x2: "16",
  y2: "8",
  key: "myku2l"
}], ["path", {
  d: "M22 16.92v3a2 2 0 01-2.18 2 19.79 19.79 0 01-8.63-3.07 19.5 19.5 0 01-6-6 19.79 19.79 0 01-3.07-8.67A2 2 0 014.11 2h3a2 2 0 012 1.72 12.84 12.84 0 00.7 2.81 2 2 0 01-.45 2.11L8.09 9.91a16 16 0 006 6l1.27-1.27a2 2 0 012.11-.45 12.84 12.84 0 002.81.7A2 2 0 0122 16.92z",
  key: "zm4dm"
}]]]);

var PhoneMissed = createReactComponent("PhoneMissed", ["svg", defaultAttributes, [["line", {
  x1: "22",
  y1: "2",
  x2: "16",
  y2: "8",
  key: "myku2l"
}], ["line", {
  x1: "16",
  y1: "2",
  x2: "22",
  y2: "8",
  key: "1r1n8w"
}], ["path", {
  d: "M22 16.92v3a2 2 0 01-2.18 2 19.79 19.79 0 01-8.63-3.07 19.5 19.5 0 01-6-6 19.79 19.79 0 01-3.07-8.67A2 2 0 014.11 2h3a2 2 0 012 1.72 12.84 12.84 0 00.7 2.81 2 2 0 01-.45 2.11L8.09 9.91a16 16 0 006 6l1.27-1.27a2 2 0 012.11-.45 12.84 12.84 0 002.81.7A2 2 0 0122 16.92z",
  key: "zm4dm"
}]]]);

var PhoneOff = createReactComponent("PhoneOff", ["svg", defaultAttributes, [["path", {
  d: "M10.68 13.31a16 16 0 003.41 2.6l1.27-1.27a2 2 0 012.11-.45 12.84 12.84 0 002.81.7 2 2 0 011.72 2v3a2 2 0 01-2.18 2 19.79 19.79 0 01-8.63-3.07 19.42 19.42 0 01-3.33-2.67m-2.67-3.34a19.79 19.79 0 01-3.07-8.63A2 2 0 014.11 2h3a2 2 0 012 1.72 12.84 12.84 0 00.7 2.81 2 2 0 01-.45 2.11L8.09 9.91",
  key: "na38gb"
}], ["line", {
  x1: "22",
  y1: "2",
  x2: "2",
  y2: "22",
  key: "wydbrk"
}]]]);

var PhoneOutgoing = createReactComponent("PhoneOutgoing", ["svg", defaultAttributes, [["polyline", {
  points: "22 8 22 2 16 2",
  key: "1inasa"
}], ["line", {
  x1: "16",
  y1: "8",
  x2: "22",
  y2: "2",
  key: "gs97a5"
}], ["path", {
  d: "M22 16.92v3a2 2 0 01-2.18 2 19.79 19.79 0 01-8.63-3.07 19.5 19.5 0 01-6-6 19.79 19.79 0 01-3.07-8.67A2 2 0 014.11 2h3a2 2 0 012 1.72 12.84 12.84 0 00.7 2.81 2 2 0 01-.45 2.11L8.09 9.91a16 16 0 006 6l1.27-1.27a2 2 0 012.11-.45 12.84 12.84 0 002.81.7A2 2 0 0122 16.92z",
  key: "zm4dm"
}]]]);

var Phone = createReactComponent("Phone", ["svg", defaultAttributes, [["path", {
  d: "M22 16.92v3a2 2 0 01-2.18 2 19.79 19.79 0 01-8.63-3.07 19.5 19.5 0 01-6-6 19.79 19.79 0 01-3.07-8.67A2 2 0 014.11 2h3a2 2 0 012 1.72 12.84 12.84 0 00.7 2.81 2 2 0 01-.45 2.11L8.09 9.91a16 16 0 006 6l1.27-1.27a2 2 0 012.11-.45 12.84 12.84 0 002.81.7A2 2 0 0122 16.92z",
  key: "zm4dm"
}]]]);

var PieChart = createReactComponent("PieChart", ["svg", defaultAttributes, [["path", {
  d: "M21.21 15.89A10 10 0 118 2.83",
  key: "1quz0o"
}], ["path", {
  d: "M22 12A10 10 0 0012 2v10z",
  key: "82go5e"
}]]]);

var Pipette = createReactComponent("Pipette", ["svg", defaultAttributes, [["path", {
  d: "M2 22l2-2h3l7-7",
  key: "j8ihnp"
}], ["path", {
  d: "M4 20v-3l7-7",
  key: "n5m2sz"
}], ["path", {
  d: "M14.29 13.3a1 1 0 001.41 0l.8-.8c.27-.27.27-.72 0-1s-.28-.72 0-1l4.08-4.08a2 2 0 000-2.83l-.17-.17a2 2 0 00-2.83 0L13.5 7.51c-.28.27-.73.27-1 0s-.73-.28-1 0l-.8.79a1 1 0 000 1.41l3.59 3.59z",
  key: "1ackp7"
}]]]);

var Plane = createReactComponent("Plane", ["svg", defaultAttributes, [["path", {
  d: "M17.8 19.2L16 11l3.5-3.5C21 6 21.5 4 21 3c-1-.5-3 0-4.5 1.5L13 8 4.8 6.2c-.5-.1-.9.1-1.1.5l-.3.5c-.2.5-.1 1 .3 1.3L9 12l-2 3H4l-1 1 3 2 2 3 1-1v-3l3-2 3.5 5.3c.3.4.8.5 1.3.3l.5-.2c.4-.3.6-.7.5-1.2z",
  key: "109i22"
}]]]);

var PlayCircle = createReactComponent("PlayCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["polygon", {
  points: "10 8 16 12 10 16 10 8",
  key: "1njv9f"
}]]]);

var Play = createReactComponent("Play", ["svg", defaultAttributes, [["polygon", {
  points: "5 3 19 12 5 21 5 3",
  key: "cqruti"
}]]]);

var PlusCircle = createReactComponent("PlusCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "12",
  y1: "8",
  x2: "12",
  y2: "16",
  key: "74zyzz"
}], ["line", {
  x1: "8",
  y1: "12",
  x2: "16",
  y2: "12",
  key: "6hnc9b"
}]]]);

var PlusSquare = createReactComponent("PlusSquare", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["line", {
  x1: "12",
  y1: "8",
  x2: "12",
  y2: "16",
  key: "74zyzz"
}], ["line", {
  x1: "8",
  y1: "12",
  x2: "16",
  y2: "12",
  key: "6hnc9b"
}]]]);

var Plus = createReactComponent("Plus", ["svg", defaultAttributes, [["line", {
  x1: "12",
  y1: "5",
  x2: "12",
  y2: "19",
  key: "1ebv7v"
}], ["line", {
  x1: "5",
  y1: "12",
  x2: "19",
  y2: "12",
  key: "iqslzh"
}]]]);

var Pocket = createReactComponent("Pocket", ["svg", defaultAttributes, [["path", {
  d: "M4 3h16a2 2 0 012 2v6a10 10 0 01-10 10A10 10 0 012 11V5a2 2 0 012-2z",
  key: "1c9ua7"
}], ["polyline", {
  points: "8 10 12 14 16 10",
  key: "1jatan"
}]]]);

var Podcast = createReactComponent("Podcast", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "11",
  r: "1",
  key: "6ljrkr"
}], ["path", {
  d: "M17.03 18.46a9 9 0 10-10.02.03",
  key: "103srb"
}], ["path", {
  d: "M16.06 13.91a5 5 0 10-7.97.2",
  key: "12jfsu"
}], ["path", {
  d: "M11.11 17a.9.9 0 111.78 0l-.52 4.67a.37.37 0 01-.74 0l-.52-4.68z",
  key: "1ggobr"
}]]]);

var Pointer = createReactComponent("Pointer", ["svg", defaultAttributes, [["path", {
  d: "M22 14a8 8 0 01-8 8",
  key: "yn0cwk"
}], ["path", {
  d: "M18 11v-1a2 2 0 00-2-2v0a2 2 0 00-2 2v0",
  key: "ux5h6q"
}], ["path", {
  d: "M14 10V9a2 2 0 00-2-2v0a2 2 0 00-2 2v1",
  key: "ctsnjf"
}], ["path", {
  d: "M10 9.5V4a2 2 0 00-2-2v0a2 2 0 00-2 2v10",
  key: "czgu6p"
}], ["path", {
  d: "M18 11a2 2 0 114 0v3a8 8 0 01-8 8h-2c-2.8 0-4.5-.86-5.99-2.34l-3.6-3.6a2 2 0 012.83-2.82L7 15",
  key: "1aj9ev"
}]]]);

var Power = createReactComponent("Power", ["svg", defaultAttributes, [["path", {
  d: "M18.36 6.64a9 9 0 11-12.73 0",
  key: "2o6fg1"
}], ["line", {
  x1: "12",
  y1: "2",
  x2: "12",
  y2: "12",
  key: "14lt8r"
}]]]);

var Printer = createReactComponent("Printer", ["svg", defaultAttributes, [["polyline", {
  points: "6 9 6 2 18 2 18 9",
  key: "nud0cz"
}], ["path", {
  d: "M6 18H4a2 2 0 01-2-2v-5a2 2 0 012-2h16a2 2 0 012 2v5a2 2 0 01-2 2h-2",
  key: "vrwuoq"
}], ["rect", {
  x: "6",
  y: "14",
  width: "12",
  height: "8",
  key: "n2kej2"
}]]]);

var QrCode = createReactComponent("QrCode", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "2",
  width: "8",
  height: "8",
  key: "ctxgbq"
}], ["path", {
  d: "M6 6h.01",
  key: "1wio8d"
}], ["rect", {
  x: "14",
  y: "2",
  width: "8",
  height: "8",
  key: "1re0wi"
}], ["path", {
  d: "M18 6h.01",
  key: "qrgi8k"
}], ["rect", {
  x: "2",
  y: "14",
  width: "8",
  height: "8",
  key: "9t1ey9"
}], ["path", {
  d: "M6 18h.01",
  key: "1t0lit"
}], ["path", {
  d: "M14 14h.01",
  key: "h7lxmz"
}], ["path", {
  d: "M18 18h.01",
  key: "191fmy"
}], ["path", {
  d: "M18 22h4v-4",
  key: "1fna9d"
}], ["path", {
  d: "M14 18v4",
  key: "imfp0i"
}], ["path", {
  d: "M22 14h-4",
  key: "fdt8m0"
}]]]);

var RadioReceiver = createReactComponent("RadioReceiver", ["svg", defaultAttributes, [["path", {
  d: "M5 16v2",
  key: "czliuy"
}], ["path", {
  d: "M19 16v2",
  key: "1f2n3c"
}], ["rect", {
  x: "2",
  y: "8",
  width: "20",
  height: "8",
  rx: "2",
  key: "37yfvc"
}], ["path", {
  d: "M18 12h0",
  key: "149gs7"
}]]]);

var Radio = createReactComponent("Radio", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "2",
  key: "1pacaq"
}], ["path", {
  d: "M16.24 7.76a6 6 0 010 8.49m-8.48-.01a6 6 0 010-8.49m11.31-2.82a10 10 0 010 14.14m-14.14 0a10 10 0 010-14.14",
  key: "mmj8hz"
}]]]);

var Redo = createReactComponent("Redo", ["svg", defaultAttributes, [["polyline", {
  points: "22 10 22 16 16 16",
  key: "rjn2t"
}], ["path", {
  d: "M1.994 16c1.813-6.542 9.559-9.482 15.36-4.36l4.64 4.36",
  key: "11ok0f"
}]]]);

var RefreshCcw = createReactComponent("RefreshCcw", ["svg", defaultAttributes, [["path", {
  d: "M3 2v6h6",
  key: "1hhjly"
}], ["path", {
  d: "M21 12A9 9 0 006 5.3L3 8",
  key: "1xl0ej"
}], ["path", {
  d: "M21 22v-6h-6",
  key: "1g0k1o"
}], ["path", {
  d: "M3 12a9 9 0 0015 6.7l3-2.7",
  key: "ixk1tr"
}]]]);

var RefreshCw = createReactComponent("RefreshCw", ["svg", defaultAttributes, [["path", {
  d: "M21 2v6h-6",
  key: "1vq7dv"
}], ["path", {
  d: "M3 12a9 9 0 0115-6.7L21 8",
  key: "48bf8d"
}], ["path", {
  d: "M3 22v-6h6",
  key: "y8mpgs"
}], ["path", {
  d: "M21 12a9 9 0 01-15 6.7L3 16",
  key: "qnxswy"
}]]]);

var Regex = createReactComponent("Regex", ["svg", defaultAttributes, [["path", {
  d: "M17 3v10",
  key: "abuoby"
}], ["path", {
  d: "M12.67 5.5l8.66 5",
  key: "wibli9"
}], ["path", {
  d: "M12.67 10.5l8.66-5",
  key: "aim3x4"
}], ["path", {
  d: "M9 17a2 2 0 00-2-2H5a2 2 0 00-2 2v2a2 2 0 002 2h2a2 2 0 002-2v-2z",
  key: "ttz7pf"
}]]]);

var Repeat1 = createReactComponent("Repeat1", ["svg", defaultAttributes, [["path", {
  d: "M17 2l4 4-4 4",
  key: "ij54qh"
}], ["path", {
  d: "M3 11v-1a4 4 0 014-4h14",
  key: "1ymu1p"
}], ["path", {
  d: "M7 22l-4-4 4-4",
  key: "16pa58"
}], ["path", {
  d: "M21 13v1a4 4 0 01-4 4H3",
  key: "1gwnas"
}], ["path", {
  d: "M11 10h1v4",
  key: "ykpro6"
}]]]);

var Repeat = createReactComponent("Repeat", ["svg", defaultAttributes, [["path", {
  d: "M17 2l4 4-4 4",
  key: "ij54qh"
}], ["path", {
  d: "M3 11v-1a4 4 0 014-4h14",
  key: "1ymu1p"
}], ["path", {
  d: "M7 22l-4-4 4-4",
  key: "16pa58"
}], ["path", {
  d: "M21 13v1a4 4 0 01-4 4H3",
  key: "1gwnas"
}]]]);

var Rewind = createReactComponent("Rewind", ["svg", defaultAttributes, [["polygon", {
  points: "11 19 2 12 11 5 11 19",
  key: "231j3z"
}], ["polygon", {
  points: "22 19 13 12 22 5 22 19",
  key: "1r8x43"
}]]]);

var RotateCcw = createReactComponent("RotateCcw", ["svg", defaultAttributes, [["path", {
  d: "M3 2v6h6",
  key: "1hhjly"
}], ["path", {
  d: "M3 13a9 9 0 103-7.7L3 8",
  key: "1dt8aq"
}]]]);

var RotateCw = createReactComponent("RotateCw", ["svg", defaultAttributes, [["path", {
  d: "M21 2v6h-6",
  key: "1vq7dv"
}], ["path", {
  d: "M21 13a9 9 0 11-3-7.7L21 8",
  key: "v7e9xy"
}]]]);

var Rss = createReactComponent("Rss", ["svg", defaultAttributes, [["path", {
  d: "M4 11a9 9 0 019 9",
  key: "x1s85k"
}], ["path", {
  d: "M4 4a16 16 0 0116 16",
  key: "5pd258"
}], ["circle", {
  cx: "5",
  cy: "19",
  r: "1",
  key: "o81s3p"
}]]]);

var Ruler = createReactComponent("Ruler", ["svg", defaultAttributes, [["path", {
  d: "M16 2l6 6L8 22l-6-6L16 2",
  key: "ltwcqc"
}], ["path", {
  d: "M7.5 10.5l2 2",
  key: "1na9wt"
}], ["path", {
  d: "M10.5 7.5l2 2",
  key: "34qx5i"
}], ["path", {
  d: "M13.5 4.5l2 2",
  key: "1bhj2u"
}], ["path", {
  d: "M4.5 13.5l2 2",
  key: "1ayu9b"
}]]]);

var Save = createReactComponent("Save", ["svg", defaultAttributes, [["path", {
  d: "M19 21H5a2 2 0 01-2-2V5a2 2 0 012-2h11l5 5v11a2 2 0 01-2 2z",
  key: "11cxyx"
}], ["polyline", {
  points: "17 21 17 13 7 13 7 21",
  key: "zvdnyr"
}], ["polyline", {
  points: "7 3 7 8 15 8",
  key: "1pv15q"
}]]]);

var Scissors = createReactComponent("Scissors", ["svg", defaultAttributes, [["circle", {
  cx: "6",
  cy: "6",
  r: "3",
  key: "ey9bbu"
}], ["circle", {
  cx: "6",
  cy: "18",
  r: "3",
  key: "l375l1"
}], ["line", {
  x1: "20",
  y1: "4",
  x2: "8.12",
  y2: "15.88",
  key: "19romf"
}], ["line", {
  x1: "14.47",
  y1: "14.48",
  x2: "20",
  y2: "20",
  key: "16feih"
}], ["line", {
  x1: "8.12",
  y1: "8.12",
  x2: "12",
  y2: "12",
  key: "9wbq4w"
}]]]);

var ScreenShareOff = createReactComponent("ScreenShareOff", ["svg", defaultAttributes, [["path", {
  d: "M13 3H4a2 2 0 00-2 2v10a2 2 0 002 2h16a2 2 0 002-2v-3",
  key: "9s0k9s"
}], ["path", {
  d: "M8 21h8",
  key: "62csdj"
}], ["path", {
  d: "M12 17v4",
  key: "icnn2j"
}], ["path", {
  d: "M22 3l-5 5",
  key: "xzltjy"
}], ["path", {
  d: "M17 3l5 5",
  key: "125xta"
}]]]);

var ScreenShare = createReactComponent("ScreenShare", ["svg", defaultAttributes, [["path", {
  d: "M13 3H4a2 2 0 00-2 2v10a2 2 0 002 2h16a2 2 0 002-2v-3",
  key: "9s0k9s"
}], ["path", {
  d: "M8 21h8",
  key: "62csdj"
}], ["path", {
  d: "M12 17v4",
  key: "icnn2j"
}], ["path", {
  d: "M17 8l5-5",
  key: "1hq5sa"
}], ["path", {
  d: "M17 3h5v5",
  key: "ku4s93"
}]]]);

var Search = createReactComponent("Search", ["svg", defaultAttributes, [["circle", {
  cx: "11",
  cy: "11",
  r: "8",
  key: "18pnwe"
}], ["line", {
  x1: "21",
  y1: "21",
  x2: "16.65",
  y2: "16.65",
  key: "u0ckfk"
}]]]);

var Send = createReactComponent("Send", ["svg", defaultAttributes, [["line", {
  x1: "22",
  y1: "2",
  x2: "11",
  y2: "13",
  key: "1fvfb0"
}], ["polygon", {
  points: "22 2 15 22 11 13 2 9 22 2",
  key: "1ews36"
}]]]);

var SeparatorHorizontal = createReactComponent("SeparatorHorizontal", ["svg", defaultAttributes, [["line", {
  x1: "3",
  y1: "12",
  x2: "21",
  y2: "12",
  key: "thangg"
}], ["polyline", {
  points: "8 8 12 4 16 8",
  key: "2pgq1n"
}], ["polyline", {
  points: "16 16 12 20 8 16",
  key: "12gw48"
}]]]);

var SeparatorVertical = createReactComponent("SeparatorVertical", ["svg", defaultAttributes, [["line", {
  x1: "12",
  y1: "3",
  x2: "12",
  y2: "21",
  key: "1cuoio"
}], ["polyline", {
  points: "8 8 4 12 8 16",
  key: "n464wr"
}], ["polyline", {
  points: "16 16 20 12 16 8",
  key: "11d4sb"
}]]]);

var ServerCrash = createReactComponent("ServerCrash", ["svg", defaultAttributes, [["path", {
  d: "M6 10H4a2 2 0 01-2-2V4a2 2 0 012-2h16a2 2 0 012 2v4a2 2 0 01-2 2h-2",
  key: "16j29y"
}], ["path", {
  d: "M6 14H4a2 2 0 00-2 2v4a2 2 0 002 2h16a2 2 0 002-2v-4a2 2 0 00-2-2h-2",
  key: "1t0co4"
}], ["path", {
  d: "M6 6h.01",
  key: "1wio8d"
}], ["path", {
  d: "M6 18h.01",
  key: "1t0lit"
}], ["path", {
  d: "M13 6l-4 6h6l-4 6",
  key: "160ein"
}]]]);

var ServerOff = createReactComponent("ServerOff", ["svg", defaultAttributes, [["path", {
  d: "M7 2h13a2 2 0 012 2v4a2 2 0 01-2 2h-5",
  key: "106nhj"
}], ["path", {
  d: "M10 10L2.5 2.5C2 2 2 2.5 2 5v3a2 2 0 002 2h6z",
  key: "d79b3j"
}], ["path", {
  d: "M22 17v-1a2 2 0 00-2-2h-1",
  key: "wpj4dh"
}], ["path", {
  d: "M4 14a2 2 0 00-2 2v4a2 2 0 002 2h16.5l1-.5.5.5-8-8H4z",
  key: "1sqkvj"
}], ["path", {
  d: "M6 18h.01",
  key: "1t0lit"
}], ["path", {
  d: "M2 2l20 20",
  key: "131xdx"
}]]]);

var Server = createReactComponent("Server", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "2",
  width: "20",
  height: "8",
  rx: "2",
  ry: "2",
  key: "rx073x"
}], ["rect", {
  x: "2",
  y: "14",
  width: "20",
  height: "8",
  rx: "2",
  ry: "2",
  key: "uf09xm"
}], ["line", {
  x1: "6",
  y1: "6",
  x2: "6.01",
  y2: "6",
  key: "1jphnz"
}], ["line", {
  x1: "6",
  y1: "18",
  x2: "6.01",
  y2: "18",
  key: "1pm8sc"
}]]]);

var Settings = createReactComponent("Settings", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "3",
  key: "n6u7h6"
}], ["path", {
  d: "M19.4 15a1.65 1.65 0 00.33 1.82l.06.06a2 2 0 010 2.83 2 2 0 01-2.83 0l-.06-.06a1.65 1.65 0 00-1.82-.33 1.65 1.65 0 00-1 1.51V21a2 2 0 01-2 2 2 2 0 01-2-2v-.09A1.65 1.65 0 009 19.4a1.65 1.65 0 00-1.82.33l-.06.06a2 2 0 01-2.83 0 2 2 0 010-2.83l.06-.06a1.65 1.65 0 00.33-1.82 1.65 1.65 0 00-1.51-1H3a2 2 0 01-2-2 2 2 0 012-2h.09A1.65 1.65 0 004.6 9a1.65 1.65 0 00-.33-1.82l-.06-.06a2 2 0 010-2.83 2 2 0 012.83 0l.06.06a1.65 1.65 0 001.82.33H9a1.65 1.65 0 001-1.51V3a2 2 0 012-2 2 2 0 012 2v.09a1.65 1.65 0 001 1.51 1.65 1.65 0 001.82-.33l.06-.06a2 2 0 012.83 0 2 2 0 010 2.83l-.06.06a1.65 1.65 0 00-.33 1.82V9a1.65 1.65 0 001.51 1H21a2 2 0 012 2 2 2 0 01-2 2h-.09a1.65 1.65 0 00-1.51 1z",
  key: "mhnzf0"
}]]]);

var Share2 = createReactComponent("Share2", ["svg", defaultAttributes, [["circle", {
  cx: "18",
  cy: "5",
  r: "3",
  key: "80x6fa"
}], ["circle", {
  cx: "6",
  cy: "12",
  r: "3",
  key: "qdtmtr"
}], ["circle", {
  cx: "18",
  cy: "19",
  r: "3",
  key: "17sjzy"
}], ["line", {
  x1: "8.59",
  y1: "13.51",
  x2: "15.42",
  y2: "17.49",
  key: "3elu7f"
}], ["line", {
  x1: "15.41",
  y1: "6.51",
  x2: "8.59",
  y2: "10.49",
  key: "e3sph7"
}]]]);

var Share = createReactComponent("Share", ["svg", defaultAttributes, [["path", {
  d: "M4 12v8a2 2 0 002 2h12a2 2 0 002-2v-8",
  key: "d70k5s"
}], ["polyline", {
  points: "16 6 12 2 8 6",
  key: "yk6rbx"
}], ["line", {
  x1: "12",
  y1: "2",
  x2: "12",
  y2: "15",
  key: "telubq"
}]]]);

var Sheet = createReactComponent("Sheet", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["line", {
  x1: "3",
  y1: "9",
  x2: "21",
  y2: "9",
  key: "1chj20"
}], ["line", {
  x1: "3",
  y1: "15",
  x2: "21",
  y2: "15",
  key: "106rj4"
}], ["line", {
  x1: "9",
  y1: "9",
  x2: "9",
  y2: "21",
  key: "1d1a44"
}], ["line", {
  x1: "15",
  y1: "9",
  x2: "15",
  y2: "21",
  key: "bx1u2"
}]]]);

var ShieldAlert = createReactComponent("ShieldAlert", ["svg", defaultAttributes, [["path", {
  d: "M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z",
  key: "epd26t"
}], ["path", {
  d: "M12 8v4",
  key: "3iljyt"
}], ["path", {
  d: "M12 16h.01",
  key: "1bsi4e"
}]]]);

var ShieldCheck = createReactComponent("ShieldCheck", ["svg", defaultAttributes, [["path", {
  d: "M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z",
  key: "epd26t"
}], ["path", {
  d: "M9 12l2 2 4-4",
  key: "1v3nuv"
}]]]);

var ShieldClose = createReactComponent("ShieldClose", ["svg", defaultAttributes, [["path", {
  d: "M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z",
  key: "epd26t"
}], ["line", {
  x1: "9.5",
  y1: "9",
  x2: "14.5",
  y2: "14",
  key: "1g9bx7"
}], ["line", {
  x1: "14.5",
  y1: "9",
  x2: "9.5",
  y2: "14",
  key: "o3154g"
}]]]);

var ShieldOff = createReactComponent("ShieldOff", ["svg", defaultAttributes, [["path", {
  d: "M19.69 14a6.9 6.9 0 00.31-2V5l-8-3-3.16 1.18",
  key: "1khz6d"
}], ["path", {
  d: "M4.73 4.73L4 5v7c0 6 8 10 8 10a20.29 20.29 0 005.62-4.38",
  key: "evw9xz"
}], ["line", {
  x1: "1",
  y1: "1",
  x2: "23",
  y2: "23",
  key: "czowfk"
}]]]);

var Shield = createReactComponent("Shield", ["svg", defaultAttributes, [["path", {
  d: "M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z",
  key: "epd26t"
}]]]);

var Shirt = createReactComponent("Shirt", ["svg", defaultAttributes, [["path", {
  d: "M20.38 3.46L16 2a4 4 0 01-8 0L3.62 3.46a2 2 0 00-1.34 2.23l.58 3.47a1 1 0 00.99.84H6v10c0 1.1.9 2 2 2h8a2 2 0 002-2V10h2.15a1 1 0 00.99-.84l.58-3.47a2 2 0 00-1.34-2.23z",
  key: "12dxcr"
}]]]);

var ShoppingBag = createReactComponent("ShoppingBag", ["svg", defaultAttributes, [["path", {
  d: "M6 2L3 6v14a2 2 0 002 2h14a2 2 0 002-2V6l-3-4z",
  key: "fxtaoz"
}], ["line", {
  x1: "3",
  y1: "6",
  x2: "21",
  y2: "6",
  key: "1jdu5d"
}], ["path", {
  d: "M16 10a4 4 0 01-8 0",
  key: "1iny4a"
}]]]);

var ShoppingCart = createReactComponent("ShoppingCart", ["svg", defaultAttributes, [["circle", {
  cx: "9",
  cy: "21",
  r: "1",
  key: "n6kcle"
}], ["circle", {
  cx: "20",
  cy: "21",
  r: "1",
  key: "ccb8gp"
}], ["path", {
  d: "M1 1h4l2.68 13.39a2 2 0 002 1.61h9.72a2 2 0 002-1.61L23 6H6",
  key: "16805q"
}]]]);

var Shovel = createReactComponent("Shovel", ["svg", defaultAttributes, [["path", {
  d: "M2 22v-5l5-5 5 5-5 5z",
  key: "19me9h"
}], ["path", {
  d: "M9.5 14.5L16 8",
  key: "fbygw3"
}], ["path", {
  d: "M17 2l5 5-.5.5a3.53 3.53 0 01-5 0s0 0 0 0a3.53 3.53 0 010-5L17 2",
  key: "zp4oky"
}]]]);

var Shuffle = createReactComponent("Shuffle", ["svg", defaultAttributes, [["polyline", {
  points: "16 3 21 3 21 8",
  key: "1pw6d7"
}], ["line", {
  x1: "4",
  y1: "20",
  x2: "21",
  y2: "3",
  key: "8zqxau"
}], ["polyline", {
  points: "21 16 21 21 16 21",
  key: "b8qqkz"
}], ["line", {
  x1: "15",
  y1: "15",
  x2: "21",
  y2: "21",
  key: "lgqqxc"
}], ["line", {
  x1: "4",
  y1: "4",
  x2: "9",
  y2: "9",
  key: "1tsi8"
}]]]);

var Sidebar = createReactComponent("Sidebar", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["line", {
  x1: "9",
  y1: "3",
  x2: "9",
  y2: "21",
  key: "5od17k"
}]]]);

var Sigma = createReactComponent("Sigma", ["svg", defaultAttributes, [["path", {
  d: "M18 7V4H6l6 8-6 8h12v-3",
  key: "2p2eto"
}]]]);

var SkipBack = createReactComponent("SkipBack", ["svg", defaultAttributes, [["polygon", {
  points: "19 20 9 12 19 4 19 20",
  key: "1roool"
}], ["line", {
  x1: "5",
  y1: "19",
  x2: "5",
  y2: "5",
  key: "9j6jpp"
}]]]);

var SkipForward = createReactComponent("SkipForward", ["svg", defaultAttributes, [["polygon", {
  points: "5 4 15 12 5 20 5 4",
  key: "lppfrf"
}], ["line", {
  x1: "19",
  y1: "5",
  x2: "19",
  y2: "19",
  key: "tlov0t"
}]]]);

var Slack = createReactComponent("Slack", ["svg", defaultAttributes, [["path", {
  d: "M14.5 10c-.83 0-1.5-.67-1.5-1.5v-5c0-.83.67-1.5 1.5-1.5s1.5.67 1.5 1.5v5c0 .83-.67 1.5-1.5 1.5z",
  key: "1fse40"
}], ["path", {
  d: "M20.5 10H19V8.5c0-.83.67-1.5 1.5-1.5s1.5.67 1.5 1.5-.67 1.5-1.5 1.5z",
  key: "1j2e2z"
}], ["path", {
  d: "M9.5 14c.83 0 1.5.67 1.5 1.5v5c0 .83-.67 1.5-1.5 1.5S8 21.33 8 20.5v-5c0-.83.67-1.5 1.5-1.5z",
  key: "m9bxs5"
}], ["path", {
  d: "M3.5 14H5v1.5c0 .83-.67 1.5-1.5 1.5S2 16.33 2 15.5 2.67 14 3.5 14z",
  key: "losb5l"
}], ["path", {
  d: "M14 14.5c0-.83.67-1.5 1.5-1.5h5c.83 0 1.5.67 1.5 1.5s-.67 1.5-1.5 1.5h-5c-.83 0-1.5-.67-1.5-1.5z",
  key: "1pk36l"
}], ["path", {
  d: "M15.5 19H14v1.5c0 .83.67 1.5 1.5 1.5s1.5-.67 1.5-1.5-.67-1.5-1.5-1.5z",
  key: "14yt70"
}], ["path", {
  d: "M10 9.5C10 8.67 9.33 8 8.5 8h-5C2.67 8 2 8.67 2 9.5S2.67 11 3.5 11h5c.83 0 1.5-.67 1.5-1.5z",
  key: "k216fz"
}], ["path", {
  d: "M8.5 5H10V3.5C10 2.67 9.33 2 8.5 2S7 2.67 7 3.5 7.67 5 8.5 5z",
  key: "1r25er"
}]]]);

var Slash = createReactComponent("Slash", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "4.93",
  y1: "4.93",
  x2: "19.07",
  y2: "19.07",
  key: "m6hk5s"
}]]]);

var Sliders = createReactComponent("Sliders", ["svg", defaultAttributes, [["line", {
  x1: "4",
  y1: "21",
  x2: "4",
  y2: "14",
  key: "1d0eeh"
}], ["line", {
  x1: "4",
  y1: "10",
  x2: "4",
  y2: "3",
  key: "1lxy4j"
}], ["line", {
  x1: "12",
  y1: "21",
  x2: "12",
  y2: "12",
  key: "lopc1c"
}], ["line", {
  x1: "12",
  y1: "8",
  x2: "12",
  y2: "3",
  key: "6da0nv"
}], ["line", {
  x1: "20",
  y1: "21",
  x2: "20",
  y2: "16",
  key: "1muld8"
}], ["line", {
  x1: "20",
  y1: "12",
  x2: "20",
  y2: "3",
  key: "10tfhs"
}], ["line", {
  x1: "1",
  y1: "14",
  x2: "7",
  y2: "14",
  key: "s98c5y"
}], ["line", {
  x1: "9",
  y1: "8",
  x2: "15",
  y2: "8",
  key: "3q5vsd"
}], ["line", {
  x1: "17",
  y1: "16",
  x2: "23",
  y2: "16",
  key: "1mqi41"
}]]]);

var Smartphone = createReactComponent("Smartphone", ["svg", defaultAttributes, [["rect", {
  x: "5",
  y: "2",
  width: "14",
  height: "20",
  rx: "2",
  ry: "2",
  key: "3bhjl3"
}], ["line", {
  x1: "12",
  y1: "18",
  x2: "12.01",
  y2: "18",
  key: "rbtten"
}]]]);

var Smile = createReactComponent("Smile", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["path", {
  d: "M8 14s1.5 2 4 2 4-2 4-2",
  key: "1qsru9"
}], ["line", {
  x1: "9",
  y1: "9",
  x2: "9.01",
  y2: "9",
  key: "sl5tun"
}], ["line", {
  x1: "15",
  y1: "9",
  x2: "15.01",
  y2: "9",
  key: "136pav"
}]]]);

var Snowflake = createReactComponent("Snowflake", ["svg", defaultAttributes, [["line", {
  x1: "2",
  y1: "12",
  x2: "22",
  y2: "12",
  key: "1vcgga"
}], ["line", {
  x1: "12",
  y1: "2",
  x2: "12",
  y2: "22",
  key: "1hqhh2"
}], ["path", {
  d: "M20 16l-4-4 4-4",
  key: "l0yr60"
}], ["path", {
  d: "M4 8l4 4-4 4",
  key: "y1ko01"
}], ["path", {
  d: "M16 4l-4 4-4-4",
  key: "e29uri"
}], ["path", {
  d: "M8 20l4-4 4 4",
  key: "3figuv"
}]]]);

var SortAsc = createReactComponent("SortAsc", ["svg", defaultAttributes, [["path", {
  d: "M11 5h10",
  key: "1lbhbw"
}], ["path", {
  d: "M11 9h7",
  key: "ktv0sq"
}], ["path", {
  d: "M11 13h4",
  key: "403xaq"
}], ["path", {
  d: "M3 17l3 3 3-3",
  key: "1rwooe"
}], ["path", {
  d: "M6 18V4",
  key: "13cxt6"
}]]]);

var SortDesc = createReactComponent("SortDesc", ["svg", defaultAttributes, [["path", {
  d: "M11 5h4",
  key: "3q0uf9"
}], ["path", {
  d: "M11 9h7",
  key: "ktv0sq"
}], ["path", {
  d: "M11 13h10",
  key: "59ep1j"
}], ["path", {
  d: "M3 17l3 3 3-3",
  key: "1rwooe"
}], ["path", {
  d: "M6 18V4",
  key: "13cxt6"
}]]]);

var Speaker = createReactComponent("Speaker", ["svg", defaultAttributes, [["rect", {
  x: "4",
  y: "2",
  width: "16",
  height: "20",
  rx: "2",
  ry: "2",
  key: "1p0e78"
}], ["circle", {
  cx: "12",
  cy: "14",
  r: "4",
  key: "rrz68r"
}], ["line", {
  x1: "12",
  y1: "6",
  x2: "12.01",
  y2: "6",
  key: "102yhi"
}]]]);

var Sprout = createReactComponent("Sprout", ["svg", defaultAttributes, [["path", {
  d: "M7 20h10",
  key: "g2f20g"
}], ["path", {
  d: "M10 20c5.5-2.5.8-6.4 3-10",
  key: "1kp9so"
}], ["path", {
  d: "M9.5 9.4c1.1.8 1.8 2.2 2.3 3.7-2 .4-3.5.4-4.8-.3-1.2-.6-2.3-1.9-3-4.2 2.8-.5 4.4 0 5.5.8z",
  key: "f9oc52"
}], ["path", {
  d: "M14.1 6a7 7 0 00-1.1 4c1.9-.1 3.3-.6 4.3-1.4 1-1 1.6-2.3 1.7-4.6-2.7.1-4 1-4.9 2z",
  key: "d3vgt9"
}]]]);

var Square = createReactComponent("Square", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}]]]);

var Star = createReactComponent("Star", ["svg", defaultAttributes, [["polygon", {
  points: "12 2 15.09 8.26 22 9.27 17 14.14 18.18 21.02 12 17.77 5.82 21.02 7 14.14 2 9.27 8.91 8.26 12 2",
  key: "lf1jo1"
}]]]);

var StopCircle = createReactComponent("StopCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["rect", {
  x: "9",
  y: "9",
  width: "6",
  height: "6",
  key: "bgbeli"
}]]]);

var Strikethrough = createReactComponent("Strikethrough", ["svg", defaultAttributes, [["path", {
  d: "M16 4H9a3 3 0 00-2.83 4",
  key: "1xkz4h"
}], ["path", {
  d: "M14 12a4 4 0 010 8H6",
  key: "1f88cb"
}], ["line", {
  x1: "4",
  y1: "12",
  x2: "20",
  y2: "12",
  key: "e1pn9i"
}]]]);

var Sun = createReactComponent("Sun", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "5",
  key: "14nid7"
}], ["line", {
  x1: "12",
  y1: "1",
  x2: "12",
  y2: "3",
  key: "1ag9n0"
}], ["line", {
  x1: "12",
  y1: "21",
  x2: "12",
  y2: "23",
  key: "t04rde"
}], ["line", {
  x1: "4.22",
  y1: "4.22",
  x2: "5.64",
  y2: "5.64",
  key: "tbec0"
}], ["line", {
  x1: "18.36",
  y1: "18.36",
  x2: "19.78",
  y2: "19.78",
  key: "lbkhcg"
}], ["line", {
  x1: "1",
  y1: "12",
  x2: "3",
  y2: "12",
  key: "mbswk2"
}], ["line", {
  x1: "21",
  y1: "12",
  x2: "23",
  y2: "12",
  key: "an60ya"
}], ["line", {
  x1: "4.22",
  y1: "19.78",
  x2: "5.64",
  y2: "18.36",
  key: "1c679v"
}], ["line", {
  x1: "18.36",
  y1: "5.64",
  x2: "19.78",
  y2: "4.22",
  key: "co19ko"
}]]]);

var Sunrise = createReactComponent("Sunrise", ["svg", defaultAttributes, [["path", {
  d: "M17 18a5 5 0 00-10 0",
  key: "1tb2kb"
}], ["line", {
  x1: "12",
  y1: "2",
  x2: "12",
  y2: "9",
  key: "mq3w9n"
}], ["line", {
  x1: "4.22",
  y1: "10.22",
  x2: "5.64",
  y2: "11.64",
  key: "q2r4og"
}], ["line", {
  x1: "1",
  y1: "18",
  x2: "3",
  y2: "18",
  key: "1r97dh"
}], ["line", {
  x1: "21",
  y1: "18",
  x2: "23",
  y2: "18",
  key: "18g1a6"
}], ["line", {
  x1: "18.36",
  y1: "11.64",
  x2: "19.78",
  y2: "10.22",
  key: "4h1pi0"
}], ["line", {
  x1: "23",
  y1: "22",
  x2: "1",
  y2: "22",
  key: "5ycleo"
}], ["polyline", {
  points: "8 6 12 2 16 6",
  key: "rp6fz1"
}]]]);

var Sunset = createReactComponent("Sunset", ["svg", defaultAttributes, [["path", {
  d: "M17 18a5 5 0 00-10 0",
  key: "1tb2kb"
}], ["line", {
  x1: "12",
  y1: "9",
  x2: "12",
  y2: "2",
  key: "1ya76y"
}], ["line", {
  x1: "4.22",
  y1: "10.22",
  x2: "5.64",
  y2: "11.64",
  key: "q2r4og"
}], ["line", {
  x1: "1",
  y1: "18",
  x2: "3",
  y2: "18",
  key: "1r97dh"
}], ["line", {
  x1: "21",
  y1: "18",
  x2: "23",
  y2: "18",
  key: "18g1a6"
}], ["line", {
  x1: "18.36",
  y1: "11.64",
  x2: "19.78",
  y2: "10.22",
  key: "4h1pi0"
}], ["line", {
  x1: "23",
  y1: "22",
  x2: "1",
  y2: "22",
  key: "5ycleo"
}], ["polyline", {
  points: "16 5 12 9 8 5",
  key: "jpnq8m"
}]]]);

var SwitchCamera = createReactComponent("SwitchCamera", ["svg", defaultAttributes, [["path", {
  d: "M11 19H4a2 2 0 01-2-2V7a2 2 0 012-2h5",
  key: "7pmll5"
}], ["path", {
  d: "M13 5h7a2 2 0 012 2v10a2 2 0 01-2 2h-5",
  key: "ire0dq"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "3",
  key: "n6u7h6"
}], ["path", {
  d: "M18 22l-3-3 3-3",
  key: "1gmfgo"
}], ["path", {
  d: "M6 2l3 3-3 3",
  key: "yyfpu1"
}]]]);

var Table = createReactComponent("Table", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["line", {
  x1: "3",
  y1: "9",
  x2: "21",
  y2: "9",
  key: "1chj20"
}], ["line", {
  x1: "3",
  y1: "15",
  x2: "21",
  y2: "15",
  key: "106rj4"
}], ["line", {
  x1: "12",
  y1: "3",
  x2: "12",
  y2: "21",
  key: "1cuoio"
}]]]);

var Tablet = createReactComponent("Tablet", ["svg", defaultAttributes, [["rect", {
  x: "4",
  y: "2",
  width: "16",
  height: "20",
  rx: "2",
  ry: "2",
  key: "1p0e78"
}], ["line", {
  x1: "12",
  y1: "18",
  x2: "12.01",
  y2: "18",
  key: "rbtten"
}]]]);

var Tag = createReactComponent("Tag", ["svg", defaultAttributes, [["path", {
  d: "M20.59 13.41l-7.17 7.17a2 2 0 01-2.83 0L2 12V2h10l8.59 8.59a2 2 0 010 2.82z",
  key: "55nqjy"
}], ["line", {
  x1: "7",
  y1: "7",
  x2: "7.01",
  y2: "7",
  key: "xebodb"
}]]]);

var Target = createReactComponent("Target", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "6",
  key: "1hbwkf"
}], ["circle", {
  cx: "12",
  cy: "12",
  r: "2",
  key: "1pacaq"
}]]]);

var Terminal = createReactComponent("Terminal", ["svg", defaultAttributes, [["polyline", {
  points: "4 17 10 11 4 5",
  key: "1utwfb"
}], ["line", {
  x1: "12",
  y1: "19",
  x2: "20",
  y2: "19",
  key: "dp87g1"
}]]]);

var Thermometer = createReactComponent("Thermometer", ["svg", defaultAttributes, [["path", {
  d: "M14 14.76V3.5a2.5 2.5 0 00-5 0v11.26a4.5 4.5 0 105 0z",
  key: "1secgz"
}]]]);

var ThumbsDown = createReactComponent("ThumbsDown", ["svg", defaultAttributes, [["path", {
  d: "M10 15v4a3 3 0 003 3l4-9V2H5.72a2 2 0 00-2 1.7l-1.38 9a2 2 0 002 2.3zm7-13h2.67A2.31 2.31 0 0122 4v7a2.31 2.31 0 01-2.33 2H17",
  key: "1mnleo"
}]]]);

var ThumbsUp = createReactComponent("ThumbsUp", ["svg", defaultAttributes, [["path", {
  d: "M14 9V5a3 3 0 00-3-3l-4 9v11h11.28a2 2 0 002-1.7l1.38-9a2 2 0 00-2-2.3zM7 22H4a2 2 0 01-2-2v-7a2 2 0 012-2h3",
  key: "6powyc"
}]]]);

var Timer = createReactComponent("Timer", ["svg", defaultAttributes, [["line", {
  x1: "10",
  x2: "14",
  y1: "2",
  y2: "2",
  key: "ama88k"
}], ["line", {
  x1: "12",
  x2: "15",
  y1: "14",
  y2: "11",
  key: "111sbd"
}], ["circle", {
  cx: "12",
  cy: "14",
  r: "8",
  key: "1s0y89"
}]]]);

var ToggleLeft = createReactComponent("ToggleLeft", ["svg", defaultAttributes, [["rect", {
  x: "1",
  y: "5",
  width: "22",
  height: "14",
  rx: "7",
  ry: "7",
  key: "vu0vnq"
}], ["circle", {
  cx: "8",
  cy: "12",
  r: "3",
  key: "1edke8"
}]]]);

var ToggleRight = createReactComponent("ToggleRight", ["svg", defaultAttributes, [["rect", {
  x: "1",
  y: "5",
  width: "22",
  height: "14",
  rx: "7",
  ry: "7",
  key: "vu0vnq"
}], ["circle", {
  cx: "16",
  cy: "12",
  r: "3",
  key: "1h4jq7"
}]]]);

var Trash2 = createReactComponent("Trash2", ["svg", defaultAttributes, [["polyline", {
  points: "3 6 5 6 21 6",
  key: "1xemgo"
}], ["path", {
  d: "M19 6v14a2 2 0 01-2 2H7a2 2 0 01-2-2V6m3 0V4a2 2 0 012-2h4a2 2 0 012 2v2",
  key: "a5em8a"
}], ["line", {
  x1: "10",
  y1: "11",
  x2: "10",
  y2: "17",
  key: "15guvf"
}], ["line", {
  x1: "14",
  y1: "11",
  x2: "14",
  y2: "17",
  key: "1thc97"
}]]]);

var Trash = createReactComponent("Trash", ["svg", defaultAttributes, [["polyline", {
  points: "3 6 5 6 21 6",
  key: "1xemgo"
}], ["path", {
  d: "M19 6v14a2 2 0 01-2 2H7a2 2 0 01-2-2V6m3 0V4a2 2 0 012-2h4a2 2 0 012 2v2",
  key: "a5em8a"
}]]]);

var Trello = createReactComponent("Trello", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["rect", {
  x: "7",
  y: "7",
  width: "3",
  height: "9",
  key: "1hysax"
}], ["rect", {
  x: "14",
  y: "7",
  width: "3",
  height: "5",
  key: "4yzqj6"
}]]]);

var TrendingDown = createReactComponent("TrendingDown", ["svg", defaultAttributes, [["polyline", {
  points: "23 18 13.5 8.5 8.5 13.5 1 6",
  key: "1gaxlq"
}], ["polyline", {
  points: "17 18 23 18 23 12",
  key: "vfsgt2"
}]]]);

var TrendingUp = createReactComponent("TrendingUp", ["svg", defaultAttributes, [["polyline", {
  points: "23 6 13.5 15.5 8.5 10.5 1 18",
  key: "1vbzdo"
}], ["polyline", {
  points: "17 6 23 6 23 12",
  key: "94s3au"
}]]]);

var Triangle = createReactComponent("Triangle", ["svg", defaultAttributes, [["path", {
  d: "M10.29 3.86L1.82 18a2 2 0 001.71 3h16.94a2 2 0 001.71-3L13.71 3.86a2 2 0 00-3.42 0z",
  key: "16e7ti"
}]]]);

var Truck = createReactComponent("Truck", ["svg", defaultAttributes, [["path", {
  d: "M10 17h4V5H2v12h3m15 0h2v-3.34a4 4 0 00-1.17-2.83L19 9h-5v8h1",
  key: "fy9sjr"
}], ["circle", {
  cx: "7.5",
  cy: "17.5",
  r: "2.5",
  key: "dg6ogh"
}], ["circle", {
  cx: "17.5",
  cy: "17.5",
  r: "2.5",
  key: "1jfk5q"
}]]]);

var Tv2 = createReactComponent("Tv2", ["svg", defaultAttributes, [["path", {
  d: "M7 21h10",
  key: "1sd173"
}], ["rect", {
  x: "2",
  y: "3",
  width: "20",
  height: "14",
  rx: "2",
  key: "9rbjce"
}]]]);

var Tv = createReactComponent("Tv", ["svg", defaultAttributes, [["rect", {
  x: "2",
  y: "7",
  width: "20",
  height: "15",
  rx: "2",
  ry: "2",
  key: "1yx64j"
}], ["polyline", {
  points: "17 2 12 7 7 2",
  key: "1dznhz"
}]]]);

var Twitch = createReactComponent("Twitch", ["svg", defaultAttributes, [["path", {
  d: "M21 2H3v16h5v4l4-4h5l4-4V2zm-10 9V7m5 4V7",
  key: "1bgeff"
}]]]);

var Twitter = createReactComponent("Twitter", ["svg", defaultAttributes, [["path", {
  d: "M23 3a10.9 10.9 0 01-3.14 1.53 4.48 4.48 0 00-7.86 3v1A10.66 10.66 0 013 4s-4 9 5 13a11.64 11.64 0 01-7 2c9 5 20 0 20-11.5a4.5 4.5 0 00-.08-.83A7.72 7.72 0 0023 3z",
  key: "78xogl"
}]]]);

var Type = createReactComponent("Type", ["svg", defaultAttributes, [["polyline", {
  points: "4 7 4 4 20 4 20 7",
  key: "1r3p86"
}], ["line", {
  x1: "9",
  y1: "20",
  x2: "15",
  y2: "20",
  key: "1su926"
}], ["line", {
  x1: "12",
  y1: "4",
  x2: "12",
  y2: "20",
  key: "3nxra"
}]]]);

var Umbrella = createReactComponent("Umbrella", ["svg", defaultAttributes, [["path", {
  d: "M23 12a11.05 11.05 0 00-22 0zm-5 7a3 3 0 01-6 0v-7",
  key: "1q6erb"
}]]]);

var Underline = createReactComponent("Underline", ["svg", defaultAttributes, [["path", {
  d: "M6 4v6a6 6 0 0012 0V4",
  key: "1ktmln"
}], ["line", {
  x1: "4",
  y1: "20",
  x2: "20",
  y2: "20",
  key: "1iyt8l"
}]]]);

var Undo = createReactComponent("Undo", ["svg", defaultAttributes, [["polyline", {
  points: "2 10 2 16 8 16",
  key: "1rz5o0"
}], ["path", {
  d: "M22 16c-1.814-6.542-9.56-9.482-15.36-4.36L2 16",
  key: "dfj7rp"
}]]]);

var Unlink2 = createReactComponent("Unlink2", ["svg", defaultAttributes, [["path", {
  d: "M15 7h2a5 5 0 010 10h-2m-6 0H7A5 5 0 017 7h2",
  key: "2adp1c"
}]]]);

var Unlink = createReactComponent("Unlink", ["svg", defaultAttributes, [["path", {
  d: "M18.84 12.25l1.72-1.71h-.02a5.004 5.004 0 00-.12-7.07 5.006 5.006 0 00-6.95 0l-1.72 1.71",
  key: "f4vrq7"
}], ["path", {
  d: "M5.17 11.75l-1.71 1.71a5.004 5.004 0 00.12 7.07 5.006 5.006 0 006.95 0l1.71-1.71",
  key: "pdo7xn"
}], ["line", {
  x1: "8",
  y1: "2",
  x2: "8",
  y2: "5",
  key: "nhhe7r"
}], ["line", {
  x1: "2",
  y1: "8",
  x2: "5",
  y2: "8",
  key: "8azsrb"
}], ["line", {
  x1: "16",
  y1: "19",
  x2: "16",
  y2: "22",
  key: "qm2ym0"
}], ["line", {
  x1: "19",
  y1: "16",
  x2: "22",
  y2: "16",
  key: "15ewtx"
}]]]);

var Unlock = createReactComponent("Unlock", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "11",
  width: "18",
  height: "11",
  rx: "2",
  ry: "2",
  key: "1jfkdd"
}], ["path", {
  d: "M7 11V7a5 5 0 019.9-1",
  key: "6fqo8o"
}]]]);

var UploadCloud = createReactComponent("UploadCloud", ["svg", defaultAttributes, [["polyline", {
  points: "16 16 12 12 8 16",
  key: "8cfiuk"
}], ["line", {
  x1: "12",
  y1: "12",
  x2: "12",
  y2: "21",
  key: "16to6l"
}], ["path", {
  d: "M20.39 18.39A5 5 0 0018 9h-1.26A8 8 0 103 16.3",
  key: "xftpgh"
}], ["polyline", {
  points: "16 16 12 12 8 16",
  key: "8cfiuk"
}]]]);

var Upload = createReactComponent("Upload", ["svg", defaultAttributes, [["path", {
  d: "M21 15v4a2 2 0 01-2 2H5a2 2 0 01-2-2v-4",
  key: "x5hunq"
}], ["polyline", {
  points: "17 8 12 3 7 8",
  key: "1gh315"
}], ["line", {
  x1: "12",
  y1: "3",
  x2: "12",
  y2: "15",
  key: "1xf98b"
}]]]);

var UserCheck = createReactComponent("UserCheck", ["svg", defaultAttributes, [["path", {
  d: "M16 21v-2a4 4 0 00-4-4H5a4 4 0 00-4 4v2",
  key: "1octem"
}], ["circle", {
  cx: "8.5",
  cy: "7",
  r: "4",
  key: "195wvc"
}], ["polyline", {
  points: "17 11 19 13 23 9",
  key: "1x6o07"
}]]]);

var UserMinus = createReactComponent("UserMinus", ["svg", defaultAttributes, [["path", {
  d: "M16 21v-2a4 4 0 00-4-4H5a4 4 0 00-4 4v2",
  key: "1octem"
}], ["circle", {
  cx: "8.5",
  cy: "7",
  r: "4",
  key: "195wvc"
}], ["line", {
  x1: "23",
  y1: "11",
  x2: "17",
  y2: "11",
  key: "1450kj"
}]]]);

var UserPlus = createReactComponent("UserPlus", ["svg", defaultAttributes, [["path", {
  d: "M16 21v-2a4 4 0 00-4-4H5a4 4 0 00-4 4v2",
  key: "1octem"
}], ["circle", {
  cx: "8.5",
  cy: "7",
  r: "4",
  key: "195wvc"
}], ["line", {
  x1: "20",
  y1: "8",
  x2: "20",
  y2: "14",
  key: "1pmqam"
}], ["line", {
  x1: "23",
  y1: "11",
  x2: "17",
  y2: "11",
  key: "1450kj"
}]]]);

var UserX = createReactComponent("UserX", ["svg", defaultAttributes, [["path", {
  d: "M16 21v-2a4 4 0 00-4-4H5a4 4 0 00-4 4v2",
  key: "1octem"
}], ["circle", {
  cx: "8.5",
  cy: "7",
  r: "4",
  key: "195wvc"
}], ["line", {
  x1: "18",
  y1: "8",
  x2: "23",
  y2: "13",
  key: "hlqvaa"
}], ["line", {
  x1: "23",
  y1: "8",
  x2: "18",
  y2: "13",
  key: "qh6602"
}]]]);

var User = createReactComponent("User", ["svg", defaultAttributes, [["path", {
  d: "M20 21v-2a4 4 0 00-4-4H8a4 4 0 00-4 4v2",
  key: "kk2qn1"
}], ["circle", {
  cx: "12",
  cy: "7",
  r: "4",
  key: "1p8uz1"
}]]]);

var Users = createReactComponent("Users", ["svg", defaultAttributes, [["path", {
  d: "M17 21v-2a4 4 0 00-4-4H5a4 4 0 00-4 4v2",
  key: "1lcsvf"
}], ["circle", {
  cx: "9",
  cy: "7",
  r: "4",
  key: "5wextf"
}], ["path", {
  d: "M23 21v-2a4 4 0 00-3-3.87",
  key: "125eqd"
}], ["path", {
  d: "M16 3.13a4 4 0 010 7.75",
  key: "1y0zii"
}]]]);

var VideoOff = createReactComponent("VideoOff", ["svg", defaultAttributes, [["path", {
  d: "M16 16v1a2 2 0 01-2 2H3a2 2 0 01-2-2V7a2 2 0 012-2h2m5.66 0H14a2 2 0 012 2v3.34l1 1L23 7v10",
  key: "oq19pp"
}], ["line", {
  x1: "1",
  y1: "1",
  x2: "23",
  y2: "23",
  key: "czowfk"
}]]]);

var Video = createReactComponent("Video", ["svg", defaultAttributes, [["polygon", {
  points: "23 7 16 12 23 17 23 7",
  key: "1b60zo"
}], ["rect", {
  x: "1",
  y: "5",
  width: "15",
  height: "14",
  rx: "2",
  ry: "2",
  key: "cvl3lu"
}]]]);

var View = createReactComponent("View", ["svg", defaultAttributes, [["path", {
  d: "M5 12s2.545-5 7-5c4.454 0 7 5 7 5s-2.546 5-7 5c-4.455 0-7-5-7-5z",
  key: "1i69hl"
}], ["path", {
  d: "M12 13a1 1 0 100-2 1 1 0 000 2z",
  key: "1b4zes"
}], ["path", {
  d: "M21 8V5a2 2 0 00-2-2H5a2 2 0 00-2 2v3m18 8v3a2 2 0 01-2 2H5a2 2 0 01-2-2v-3",
  key: "y1dmu8"
}]]]);

var Voicemail = createReactComponent("Voicemail", ["svg", defaultAttributes, [["circle", {
  cx: "5.5",
  cy: "11.5",
  r: "4.5",
  key: "1dtq6q"
}], ["circle", {
  cx: "18.5",
  cy: "11.5",
  r: "4.5",
  key: "1ymvvr"
}], ["line", {
  x1: "5.5",
  y1: "16",
  x2: "18.5",
  y2: "16",
  key: "1dc5s6"
}]]]);

var Volume1 = createReactComponent("Volume1", ["svg", defaultAttributes, [["polygon", {
  points: "11 5 6 9 2 9 2 15 6 15 11 19 11 5",
  key: "m3abqr"
}], ["path", {
  d: "M15.54 8.46a5 5 0 010 7.07",
  key: "az8w0d"
}]]]);

var Volume2 = createReactComponent("Volume2", ["svg", defaultAttributes, [["polygon", {
  points: "11 5 6 9 2 9 2 15 6 15 11 19 11 5",
  key: "m3abqr"
}], ["path", {
  d: "M19.07 4.93a10 10 0 010 14.14M15.54 8.46a5 5 0 010 7.07",
  key: "73qc8v"
}]]]);

var VolumeX = createReactComponent("VolumeX", ["svg", defaultAttributes, [["polygon", {
  points: "11 5 6 9 2 9 2 15 6 15 11 19 11 5",
  key: "m3abqr"
}], ["line", {
  x1: "23",
  y1: "9",
  x2: "17",
  y2: "15",
  key: "94dqui"
}], ["line", {
  x1: "17",
  y1: "9",
  x2: "23",
  y2: "15",
  key: "1er0hc"
}]]]);

var Volume = createReactComponent("Volume", ["svg", defaultAttributes, [["polygon", {
  points: "11 5 6 9 2 9 2 15 6 15 11 19 11 5",
  key: "m3abqr"
}]]]);

var Wallet = createReactComponent("Wallet", ["svg", defaultAttributes, [["path", {
  d: "M20 12V8H6a2 2 0 01-2-2c0-1.1.9-2 2-2h12v4",
  key: "ua1bbl"
}], ["path", {
  d: "M4 6v12c0 1.1.9 2 2 2h14v-4",
  key: "wbjgsh"
}], ["path", {
  d: "M18 12a2 2 0 00-2 2c0 1.1.9 2 2 2h4v-4h-4z",
  key: "axqwre"
}]]]);

var Watch = createReactComponent("Watch", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "7",
  key: "f8eh5q"
}], ["polyline", {
  points: "12 9 12 12 13.5 13.5",
  key: "x5elnt"
}], ["path", {
  d: "M16.51 17.35l-.35 3.83a2 2 0 01-2 1.82H9.83a2 2 0 01-2-1.82l-.35-3.83m.01-10.7l.35-3.83A2 2 0 019.83 1h4.35a2 2 0 012 1.82l.35 3.83",
  key: "vbew3j"
}]]]);

var Webcam = createReactComponent("Webcam", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "10",
  r: "8",
  key: "11tx6q"
}], ["circle", {
  cx: "12",
  cy: "10",
  r: "3",
  key: "1byug6"
}], ["path", {
  d: "M12 22v-4m-5 4h5-5zm10 0h-5 5z",
  key: "ki1bkt"
}]]]);

var WifiOff = createReactComponent("WifiOff", ["svg", defaultAttributes, [["line", {
  x1: "2",
  y1: "2",
  x2: "22",
  y2: "22",
  key: "1ehfaz"
}], ["path", {
  d: "M8.5 16.5a5 5 0 017 0",
  key: "gv8qp0"
}], ["path", {
  d: "M2 8.82a15 15 0 014.17-2.65",
  key: "1k0sjq"
}], ["path", {
  d: "M10.66 5c4.01-.36 8.14.9 11.34 3.76",
  key: "8na309"
}], ["path", {
  d: "M16.85 11.25a10 10 0 012.22 1.68",
  key: "8jhg8s"
}], ["path", {
  d: "M5 13a10 10 0 015.24-2.76",
  key: "1he7n8"
}], ["line", {
  x1: "12",
  y1: "20",
  x2: "12.01",
  y2: "20",
  key: "5gs3sv"
}]]]);

var Wifi = createReactComponent("Wifi", ["svg", defaultAttributes, [["path", {
  d: "M5 13a10 10 0 0114 0",
  key: "h8a88u"
}], ["path", {
  d: "M8.5 16.5a5 5 0 017 0",
  key: "gv8qp0"
}], ["path", {
  d: "M2 8.82a15 15 0 0120 0",
  key: "iokqm8"
}], ["line", {
  x1: "12",
  y1: "20",
  x2: "12.01",
  y2: "20",
  key: "5gs3sv"
}]]]);

var Wind = createReactComponent("Wind", ["svg", defaultAttributes, [["path", {
  d: "M9.59 4.59A2 2 0 1111 8H2m10.59 11.41A2 2 0 1014 16H2m15.73-8.27A2.5 2.5 0 1119.5 12H2",
  key: "3ff5yx"
}]]]);

var Wrench = createReactComponent("Wrench", ["svg", defaultAttributes, [["path", {
  d: "M14.7 6.3a1 1 0 000 1.4l1.6 1.6a1 1 0 001.4 0l3.77-3.77a6 6 0 01-7.94 7.94l-6.91 6.91a2.12 2.12 0 01-3-3l6.91-6.91a6 6 0 017.94-7.94l-3.76 3.76z",
  key: "zrsdfh"
}]]]);

var XCircle = createReactComponent("XCircle", ["svg", defaultAttributes, [["circle", {
  cx: "12",
  cy: "12",
  r: "10",
  key: "14id60"
}], ["line", {
  x1: "15",
  y1: "9",
  x2: "9",
  y2: "15",
  key: "jmj1s"
}], ["line", {
  x1: "9",
  y1: "9",
  x2: "15",
  y2: "15",
  key: "bhcchc"
}]]]);

var XOctagon = createReactComponent("XOctagon", ["svg", defaultAttributes, [["polygon", {
  points: "7.86 2 16.14 2 22 7.86 22 16.14 16.14 22 7.86 22 2 16.14 2 7.86 7.86 2",
  key: "1qo9ur"
}], ["line", {
  x1: "15",
  y1: "9",
  x2: "9",
  y2: "15",
  key: "jmj1s"
}], ["line", {
  x1: "9",
  y1: "9",
  x2: "15",
  y2: "15",
  key: "bhcchc"
}]]]);

var XSquare = createReactComponent("XSquare", ["svg", defaultAttributes, [["rect", {
  x: "3",
  y: "3",
  width: "18",
  height: "18",
  rx: "2",
  ry: "2",
  key: "1sylwi"
}], ["line", {
  x1: "9",
  y1: "9",
  x2: "15",
  y2: "15",
  key: "bhcchc"
}], ["line", {
  x1: "15",
  y1: "9",
  x2: "9",
  y2: "15",
  key: "jmj1s"
}]]]);

var X = createReactComponent("X", ["svg", defaultAttributes, [["line", {
  x1: "18",
  y1: "6",
  x2: "6",
  y2: "18",
  key: "1sacsb"
}], ["line", {
  x1: "6",
  y1: "6",
  x2: "18",
  y2: "18",
  key: "1o5kmt"
}]]]);

var Youtube = createReactComponent("Youtube", ["svg", defaultAttributes, [["path", {
  d: "M22.54 6.42a2.78 2.78 0 00-1.94-2C18.88 4 12 4 12 4s-6.88 0-8.6.46a2.78 2.78 0 00-1.94 2A29 29 0 001 11.75a29 29 0 00.46 5.33A2.78 2.78 0 003.4 19c1.72.46 8.6.46 8.6.46s6.88 0 8.6-.46a2.78 2.78 0 001.94-2 29 29 0 00.46-5.25 29 29 0 00-.46-5.33z",
  key: "1vjxvm"
}], ["polygon", {
  points: "9.75 15.02 15.5 11.75 9.75 8.48 9.75 15.02",
  key: "1tptg8"
}]]]);

var ZapOff = createReactComponent("ZapOff", ["svg", defaultAttributes, [["polyline", {
  points: "12.41 6.75 13 2 10.57 4.92",
  key: "11g4b1"
}], ["polyline", {
  points: "18.57 12.91 21 10 15.66 10",
  key: "l0gmze"
}], ["polyline", {
  points: "8 8 3 14 12 14 11 22 16 16",
  key: "1sa4vm"
}], ["line", {
  x1: "1",
  y1: "1",
  x2: "23",
  y2: "23",
  key: "czowfk"
}]]]);

var Zap = createReactComponent("Zap", ["svg", defaultAttributes, [["polygon", {
  points: "13 2 3 14 12 14 11 22 21 10 12 10 13 2",
  key: "157nmd"
}]]]);

var ZoomIn = createReactComponent("ZoomIn", ["svg", defaultAttributes, [["circle", {
  cx: "11",
  cy: "11",
  r: "8",
  key: "18pnwe"
}], ["line", {
  x1: "21",
  y1: "21",
  x2: "16.65",
  y2: "16.65",
  key: "u0ckfk"
}], ["line", {
  x1: "11",
  y1: "8",
  x2: "11",
  y2: "14",
  key: "1n5115"
}], ["line", {
  x1: "8",
  y1: "11",
  x2: "14",
  y2: "11",
  key: "1u5rbm"
}]]]);

var ZoomOut = createReactComponent("ZoomOut", ["svg", defaultAttributes, [["circle", {
  cx: "11",
  cy: "11",
  r: "8",
  key: "18pnwe"
}], ["line", {
  x1: "21",
  y1: "21",
  x2: "16.65",
  y2: "16.65",
  key: "u0ckfk"
}], ["line", {
  x1: "8",
  y1: "11",
  x2: "14",
  y2: "11",
  key: "1u5rbm"
}]]]);

exports.Activity = Activity;
exports.Airplay = Airplay;
exports.AlarmCheck = AlarmCheck;
exports.AlarmClock = AlarmClock;
exports.AlarmMinus = AlarmMinus;
exports.AlarmPlus = AlarmPlus;
exports.Album = Album;
exports.AlertCircle = AlertCircle;
exports.AlertOctagon = AlertOctagon;
exports.AlertTriangle = AlertTriangle;
exports.AlignCenter = AlignCenter;
exports.AlignJustify = AlignJustify;
exports.AlignLeft = AlignLeft;
exports.AlignRight = AlignRight;
exports.Anchor = Anchor;
exports.Aperture = Aperture;
exports.Archive = Archive;
exports.ArrowBigDown = ArrowBigDown;
exports.ArrowBigLeft = ArrowBigLeft;
exports.ArrowBigRight = ArrowBigRight;
exports.ArrowBigUp = ArrowBigUp;
exports.ArrowDown = ArrowDown;
exports.ArrowDownCircle = ArrowDownCircle;
exports.ArrowDownLeft = ArrowDownLeft;
exports.ArrowDownRight = ArrowDownRight;
exports.ArrowLeft = ArrowLeft;
exports.ArrowLeftCircle = ArrowLeftCircle;
exports.ArrowRight = ArrowRight;
exports.ArrowRightCircle = ArrowRightCircle;
exports.ArrowUp = ArrowUp;
exports.ArrowUpCircle = ArrowUpCircle;
exports.ArrowUpLeft = ArrowUpLeft;
exports.ArrowUpRight = ArrowUpRight;
exports.Asterisk = Asterisk;
exports.AtSign = AtSign;
exports.Award = Award;
exports.Axe = Axe;
exports.BarChart = BarChart;
exports.BarChart2 = BarChart2;
exports.Battery = Battery;
exports.BatteryCharging = BatteryCharging;
exports.BatteryFull = BatteryFull;
exports.BatteryLow = BatteryLow;
exports.BatteryMedium = BatteryMedium;
exports.Beaker = Beaker;
exports.Bell = Bell;
exports.BellMinus = BellMinus;
exports.BellOff = BellOff;
exports.BellPlus = BellPlus;
exports.Bike = Bike;
exports.Binary = Binary;
exports.Bluetooth = Bluetooth;
exports.BluetoothConnected = BluetoothConnected;
exports.BluetoothOff = BluetoothOff;
exports.BluetoothSearching = BluetoothSearching;
exports.Bold = Bold;
exports.Book = Book;
exports.BookOpen = BookOpen;
exports.Bookmark = Bookmark;
exports.Bot = Bot;
exports.Box = Box;
exports.BoxSelect = BoxSelect;
exports.Briefcase = Briefcase;
exports.Brush = Brush;
exports.Building = Building;
exports.Bus = Bus;
exports.Calculator = Calculator;
exports.Calendar = Calendar;
exports.Camera = Camera;
exports.CameraOff = CameraOff;
exports.Car = Car;
exports.Cast = Cast;
exports.Check = Check;
exports.CheckCircle = CheckCircle;
exports.CheckSquare = CheckSquare;
exports.ChevronDown = ChevronDown;
exports.ChevronLeft = ChevronLeft;
exports.ChevronRight = ChevronRight;
exports.ChevronUp = ChevronUp;
exports.ChevronsDown = ChevronsDown;
exports.ChevronsLeft = ChevronsLeft;
exports.ChevronsRight = ChevronsRight;
exports.ChevronsUp = ChevronsUp;
exports.Chrome = Chrome;
exports.Circle = Circle;
exports.Clipboard = Clipboard;
exports.Clock = Clock;
exports.Cloud = Cloud;
exports.CloudDrizzle = CloudDrizzle;
exports.CloudLightning = CloudLightning;
exports.CloudOff = CloudOff;
exports.CloudRain = CloudRain;
exports.CloudSnow = CloudSnow;
exports.Clover = Clover;
exports.Code = Code;
exports.Codepen = Codepen;
exports.Codesandbox = Codesandbox;
exports.Coffee = Coffee;
exports.Coins = Coins;
exports.Columns = Columns;
exports.Command = Command;
exports.Compass = Compass;
exports.Contact = Contact;
exports.Contrast = Contrast;
exports.Copy = Copy;
exports.Copyleft = Copyleft;
exports.Copyright = Copyright;
exports.CornerDownLeft = CornerDownLeft;
exports.CornerDownRight = CornerDownRight;
exports.CornerLeftDown = CornerLeftDown;
exports.CornerLeftUp = CornerLeftUp;
exports.CornerRightDown = CornerRightDown;
exports.CornerRightUp = CornerRightUp;
exports.CornerUpLeft = CornerUpLeft;
exports.CornerUpRight = CornerUpRight;
exports.Cpu = Cpu;
exports.CreditCard = CreditCard;
exports.Crop = Crop;
exports.Cross = Cross;
exports.Crosshair = Crosshair;
exports.Crown = Crown;
exports.Database = Database;
exports.Delete = Delete;
exports.Disc = Disc;
exports.Divide = Divide;
exports.DivideCircle = DivideCircle;
exports.DivideSquare = DivideSquare;
exports.DollarSign = DollarSign;
exports.Download = Download;
exports.DownloadCloud = DownloadCloud;
exports.Dribbble = Dribbble;
exports.Droplet = Droplet;
exports.Edit = Edit;
exports.Edit2 = Edit2;
exports.Edit3 = Edit3;
exports.Equal = Equal;
exports.EqualNot = EqualNot;
exports.ExternalLink = ExternalLink;
exports.Eye = Eye;
exports.EyeOff = EyeOff;
exports.Facebook = Facebook;
exports.FastForward = FastForward;
exports.Feather = Feather;
exports.Figma = Figma;
exports.File = File;
exports.FileCheck = FileCheck;
exports.FileCheck2 = FileCheck2;
exports.FileCode = FileCode;
exports.FileDigit = FileDigit;
exports.FileMinus = FileMinus;
exports.FileMinus2 = FileMinus2;
exports.FilePlus = FilePlus;
exports.FilePlus2 = FilePlus2;
exports.FileSearch = FileSearch;
exports.FileText = FileText;
exports.FileX = FileX;
exports.FileX2 = FileX2;
exports.Files = Files;
exports.Film = Film;
exports.Filter = Filter;
exports.Flag = Flag;
exports.Flame = Flame;
exports.Flashlight = Flashlight;
exports.FlashlightOff = FlashlightOff;
exports.FlaskConical = FlaskConical;
exports.FlaskRound = FlaskRound;
exports.Folder = Folder;
exports.FolderMinus = FolderMinus;
exports.FolderPlus = FolderPlus;
exports.Framer = Framer;
exports.Frown = Frown;
exports.Gamepad = Gamepad;
exports.Gamepad2 = Gamepad2;
exports.Gauge = Gauge;
exports.Gavel = Gavel;
exports.Gift = Gift;
exports.GitBranch = GitBranch;
exports.GitBranchPlus = GitBranchPlus;
exports.GitCommit = GitCommit;
exports.GitMerge = GitMerge;
exports.GitPullRequest = GitPullRequest;
exports.Github = Github;
exports.Gitlab = Gitlab;
exports.Globe = Globe;
exports.Globe2 = Globe2;
exports.Grab = Grab;
exports.Grid = Grid;
exports.GripHorizontal = GripHorizontal;
exports.GripVertical = GripVertical;
exports.Hammer = Hammer;
exports.Hand = Hand;
exports.HardDrive = HardDrive;
exports.HardHat = HardHat;
exports.Hash = Hash;
exports.Headphones = Headphones;
exports.Heart = Heart;
exports.HelpCircle = HelpCircle;
exports.Hexagon = Hexagon;
exports.Highlighter = Highlighter;
exports.History = History;
exports.Home = Home;
exports.Image = Image;
exports.ImageMinus = ImageMinus;
exports.ImageOff = ImageOff;
exports.ImagePlus = ImagePlus;
exports.Inbox = Inbox;
exports.Indent = Indent;
exports.Infinity = Infinity;
exports.Info = Info;
exports.Inspect = Inspect;
exports.Instagram = Instagram;
exports.Italic = Italic;
exports.Key = Key;
exports.Languages = Languages;
exports.Laptop = Laptop;
exports.Laptop2 = Laptop2;
exports.Lasso = Lasso;
exports.LassoSelect = LassoSelect;
exports.Layers = Layers;
exports.Layout = Layout;
exports.LayoutDashboard = LayoutDashboard;
exports.LayoutGrid = LayoutGrid;
exports.LayoutList = LayoutList;
exports.Library = Library;
exports.LifeBuoy = LifeBuoy;
exports.Lightbulb = Lightbulb;
exports.Link = Link;
exports.Link2 = Link2;
exports.Link2Off = Link2Off;
exports.Linkedin = Linkedin;
exports.List = List;
exports.ListChecks = ListChecks;
exports.ListOrdered = ListOrdered;
exports.Loader = Loader;
exports.Locate = Locate;
exports.LocateFixed = LocateFixed;
exports.Lock = Lock;
exports.LogIn = LogIn;
exports.LogOut = LogOut;
exports.Mail = Mail;
exports.Map = Map;
exports.MapPin = MapPin;
exports.Maximize = Maximize;
exports.Maximize2 = Maximize2;
exports.Meh = Meh;
exports.Menu = Menu;
exports.MessageCircle = MessageCircle;
exports.MessageSquare = MessageSquare;
exports.Mic = Mic;
exports.MicOff = MicOff;
exports.Minimize = Minimize;
exports.Minimize2 = Minimize2;
exports.Minus = Minus;
exports.MinusCircle = MinusCircle;
exports.MinusSquare = MinusSquare;
exports.Monitor = Monitor;
exports.MonitorOff = MonitorOff;
exports.Moon = Moon;
exports.MoreHorizontal = MoreHorizontal;
exports.MoreVertical = MoreVertical;
exports.MousePointer = MousePointer;
exports.MousePointer2 = MousePointer2;
exports.Move = Move;
exports.MoveDiagonal = MoveDiagonal;
exports.MoveDiagonal2 = MoveDiagonal2;
exports.MoveHorizontal = MoveHorizontal;
exports.MoveVertical = MoveVertical;
exports.Music = Music;
exports.Navigation = Navigation;
exports.Navigation2 = Navigation2;
exports.Network = Network;
exports.Octagon = Octagon;
exports.Option = Option;
exports.Outdent = Outdent;
exports.Package = Package;
exports.Palette = Palette;
exports.Paperclip = Paperclip;
exports.Pause = Pause;
exports.PauseCircle = PauseCircle;
exports.PenTool = PenTool;
exports.Pencil = Pencil;
exports.Percent = Percent;
exports.Phone = Phone;
exports.PhoneCall = PhoneCall;
exports.PhoneForwarded = PhoneForwarded;
exports.PhoneIncoming = PhoneIncoming;
exports.PhoneMissed = PhoneMissed;
exports.PhoneOff = PhoneOff;
exports.PhoneOutgoing = PhoneOutgoing;
exports.PieChart = PieChart;
exports.Pipette = Pipette;
exports.Plane = Plane;
exports.Play = Play;
exports.PlayCircle = PlayCircle;
exports.Plus = Plus;
exports.PlusCircle = PlusCircle;
exports.PlusSquare = PlusSquare;
exports.Pocket = Pocket;
exports.Podcast = Podcast;
exports.Pointer = Pointer;
exports.Power = Power;
exports.Printer = Printer;
exports.QrCode = QrCode;
exports.Radio = Radio;
exports.RadioReceiver = RadioReceiver;
exports.Redo = Redo;
exports.RefreshCcw = RefreshCcw;
exports.RefreshCw = RefreshCw;
exports.Regex = Regex;
exports.Repeat = Repeat;
exports.Repeat1 = Repeat1;
exports.Rewind = Rewind;
exports.RotateCcw = RotateCcw;
exports.RotateCw = RotateCw;
exports.Rss = Rss;
exports.Ruler = Ruler;
exports.Save = Save;
exports.Scissors = Scissors;
exports.ScreenShare = ScreenShare;
exports.ScreenShareOff = ScreenShareOff;
exports.Search = Search;
exports.Send = Send;
exports.SeparatorHorizontal = SeparatorHorizontal;
exports.SeparatorVertical = SeparatorVertical;
exports.Server = Server;
exports.ServerCrash = ServerCrash;
exports.ServerOff = ServerOff;
exports.Settings = Settings;
exports.Share = Share;
exports.Share2 = Share2;
exports.Sheet = Sheet;
exports.Shield = Shield;
exports.ShieldAlert = ShieldAlert;
exports.ShieldCheck = ShieldCheck;
exports.ShieldClose = ShieldClose;
exports.ShieldOff = ShieldOff;
exports.Shirt = Shirt;
exports.ShoppingBag = ShoppingBag;
exports.ShoppingCart = ShoppingCart;
exports.Shovel = Shovel;
exports.Shuffle = Shuffle;
exports.Sidebar = Sidebar;
exports.Sigma = Sigma;
exports.SkipBack = SkipBack;
exports.SkipForward = SkipForward;
exports.Slack = Slack;
exports.Slash = Slash;
exports.Sliders = Sliders;
exports.Smartphone = Smartphone;
exports.Smile = Smile;
exports.Snowflake = Snowflake;
exports.SortAsc = SortAsc;
exports.SortDesc = SortDesc;
exports.Speaker = Speaker;
exports.Sprout = Sprout;
exports.Square = Square;
exports.Star = Star;
exports.StopCircle = StopCircle;
exports.Strikethrough = Strikethrough;
exports.Sun = Sun;
exports.Sunrise = Sunrise;
exports.Sunset = Sunset;
exports.SwitchCamera = SwitchCamera;
exports.Table = Table;
exports.Tablet = Tablet;
exports.Tag = Tag;
exports.Target = Target;
exports.Terminal = Terminal;
exports.Thermometer = Thermometer;
exports.ThumbsDown = ThumbsDown;
exports.ThumbsUp = ThumbsUp;
exports.Timer = Timer;
exports.ToggleLeft = ToggleLeft;
exports.ToggleRight = ToggleRight;
exports.Trash = Trash;
exports.Trash2 = Trash2;
exports.Trello = Trello;
exports.TrendingDown = TrendingDown;
exports.TrendingUp = TrendingUp;
exports.Triangle = Triangle;
exports.Truck = Truck;
exports.Tv = Tv;
exports.Tv2 = Tv2;
exports.Twitch = Twitch;
exports.Twitter = Twitter;
exports.Type = Type;
exports.Umbrella = Umbrella;
exports.Underline = Underline;
exports.Undo = Undo;
exports.Unlink = Unlink;
exports.Unlink2 = Unlink2;
exports.Unlock = Unlock;
exports.Upload = Upload;
exports.UploadCloud = UploadCloud;
exports.User = User;
exports.UserCheck = UserCheck;
exports.UserMinus = UserMinus;
exports.UserPlus = UserPlus;
exports.UserX = UserX;
exports.Users = Users;
exports.Video = Video;
exports.VideoOff = VideoOff;
exports.View = View;
exports.Voicemail = Voicemail;
exports.Volume = Volume;
exports.Volume1 = Volume1;
exports.Volume2 = Volume2;
exports.VolumeX = VolumeX;
exports.Wallet = Wallet;
exports.Watch = Watch;
exports.Webcam = Webcam;
exports.Wifi = Wifi;
exports.WifiOff = WifiOff;
exports.Wind = Wind;
exports.Wrench = Wrench;
exports.X = X;
exports.XCircle = XCircle;
exports.XOctagon = XOctagon;
exports.XSquare = XSquare;
exports.Youtube = Youtube;
exports.Zap = Zap;
exports.ZapOff = ZapOff;
exports.ZoomIn = ZoomIn;
exports.ZoomOut = ZoomOut;
